<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Delivery Mode and the Meconium Microbiome • Reviews of published signatures based on bugsigdb.org</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.7.1/jquery.min.js" integrity="sha512-v2CJ7UaYy4JwqLDIrZUI/4hqeoQieOmAZNXBeQyjo21dadnwR+8ZaIJVT8EE2iyI61OV8e6M8PP2/4hpQINQ/g==" crossorigin="anonymous" referrerpolicy="no-referrer"></script><!-- Bootstrap --><link href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.4.0/flatly/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="Delivery Mode and the Meconium Microbiome">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">


    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">Reviews of published signatures based on bugsigdb.org</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">0.99.5</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Articles

    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/Bearkland%20capstone%20DED%2012.2.2024.html">Differential Abundance in the ocular microbiome in DED</a>
    </li>
    <li>
      <a href="../articles/c-section_meconium_shaimaa.html">Delivery Mode and the Meconium Microbiome</a>
    </li>
    <li>
      <a href="../articles/capstoneanalysis_clare.html">COVID-19 associated microbiome</a>
    </li>
    <li>
      <a href="../articles/capstoneanalysis_fatima.html">An analysis of Most frequent taxa in Major Depression and Bipolar Disorder</a>
    </li>
    <li>
      <a href="../articles/capstoneanalysis_ifeanyi.html">Distinguishing socioeconomic and racial/ethnic variation of the human gut microbiome by re-analysis of studies from around the world</a>
    </li>
    <li>
      <a href="../articles/capstoneanalysis_kweku.html">The Irritable Bowel Syndrome-associated Microbiome</a>
    </li>
    <li>
      <a href="../articles/fieldworkanalysis_samara.html">The Endometriosis-associated Microbiome</a>
    </li>
    <li>
      <a href="../articles/healthysig.html">Analysis of prevalent stool genera in bugsigdb.org</a>
    </li>
  </ul>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/waldronlab/bugSigSimple" class="external-link">
    <span class="fa fa-github"></span>

  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->



      </header><script src="c-section_meconium_shaimaa_files/kePrint-0.0.1/kePrint.js"></script><link href="c-section_meconium_shaimaa_files/lightable-0.0.1/lightable.css" rel="stylesheet">
<div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>Delivery Mode and the Meconium Microbiome</h1>
            
            <h4 data-toc-skip class="date">17 March, 2025</h4>
      

      <div class="hidden name"><code>c-section_meconium_shaimaa.Rmd</code></div>

    </div>

    
    
<div class="section level2">
<h2 id="analysis-objectives">Analysis objectives<a class="anchor" aria-label="anchor" href="#analysis-objectives"></a>
</h2>
<ol style="list-style-type: decimal">
<li>Import, recode, and subset data from bugsigdb.org<br>
</li>
<li>Create a table of studies</li>
<li>Calculate the frequency of appearance of each taxa in independent
signatures and identify the most frequently reported taxa</li>
</ol>
</div>
<div class="section level2">
<h2 id="making-sure-packages-are-installed">Making sure packages are installed<a class="anchor" aria-label="anchor" href="#making-sure-packages-are-installed"></a>
</h2>
<p>Not evaluated in vignette:</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw">if</span> <span class="op">(</span><span class="op">!</span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">require</a></span><span class="op">(</span><span class="st"><a href="https://bioconductor.github.io/BiocManager/" class="external-link">"BiocManager"</a></span>, quietly <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span><span class="op">)</span></span>
<span>    <span class="fu"><a href="https://rdrr.io/r/utils/install.packages.html" class="external-link">install.packages</a></span><span class="op">(</span><span class="st">"BiocManager"</span><span class="op">)</span></span>
<span><span class="fu">BiocManager</span><span class="fu">::</span><span class="fu"><a href="https://bioconductor.github.io/BiocManager/reference/install.html" class="external-link">install</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"devtools"</span>, <span class="st">"tidyverse"</span>, <span class="st">"kableExtra"</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="fu">BiocManager</span><span class="fu">::</span><span class="fu"><a href="https://bioconductor.github.io/BiocManager/reference/install.html" class="external-link">install</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"waldronlab/bugSigSimple"</span>, <span class="st">"waldronlab/BugSigDBStats"</span>, <span class="st">"waldronlab/bugsigdbr"</span><span class="op">)</span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level2">
<h2 id="load-and-subset-data">Load and subset data<a class="anchor" aria-label="anchor" href="#load-and-subset-data"></a>
</h2>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/base/message.html" class="external-link">suppressPackageStartupMessages</a></span><span class="op">(</span><span class="op">{</span></span>
<span>  <span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://waldronlab.io/bugSigSimple" class="external-link">bugSigSimple</a></span><span class="op">)</span></span>
<span>  <span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/waldronlab/BugSigDBStats" class="external-link">BugSigDBStats</a></span><span class="op">)</span></span>
<span>  <span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/waldronlab/bugsigdbr" class="external-link">bugsigdbr</a></span><span class="op">)</span></span>
<span>  <span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://tidyverse.tidyverse.org" class="external-link">tidyverse</a></span><span class="op">)</span></span>
<span>  <span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://stringr.tidyverse.org" class="external-link">stringr</a></span><span class="op">)</span></span>
<span>  <span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="http://haozhu233.github.io/kableExtra/" class="external-link">kableExtra</a></span><span class="op">)</span></span>
<span>  <span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://dplyr.tidyverse.org" class="external-link">dplyr</a></span><span class="op">)</span></span>
<span><span class="op">}</span><span class="op">)</span></span></code></pre></div>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># use version="devel" and cache = FALSE to take the latest version from bugsigdb.org</span></span>
<span><span class="va">dat</span> <span class="op">&lt;-</span> <span class="fu">bugsigdbr</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/bugsigdbr/man/importBugSigDB.html" class="external-link">importBugSigDB</a></span><span class="op">(</span>version <span class="op">=</span> <span class="st">"devel"</span>, cache <span class="op">=</span> <span class="cn">FALSE</span><span class="op">)</span> </span>
<span><span class="fu"><a href="https://rdrr.io/r/base/dim.html" class="external-link">dim</a></span><span class="op">(</span><span class="va">dat</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## [1] 7343   50</span></span></code></pre>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/base/names.html" class="external-link">names</a></span><span class="op">(</span><span class="va">dat</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">##  [1] "BSDB ID"                    "Study"                     </span></span>
<span><span class="co">##  [3] "Study design"               "PMID"                      </span></span>
<span><span class="co">##  [5] "DOI"                        "URL"                       </span></span>
<span><span class="co">##  [7] "Authors list"               "Title"                     </span></span>
<span><span class="co">##  [9] "Journal"                    "Year"                      </span></span>
<span><span class="co">## [11] "Keywords"                   "Experiment"                </span></span>
<span><span class="co">## [13] "Location of subjects"       "Host species"              </span></span>
<span><span class="co">## [15] "Body site"                  "UBERON ID"                 </span></span>
<span><span class="co">## [17] "Condition"                  "EFO ID"                    </span></span>
<span><span class="co">## [19] "Group 0 name"               "Group 1 name"              </span></span>
<span><span class="co">## [21] "Group 1 definition"         "Group 0 sample size"       </span></span>
<span><span class="co">## [23] "Group 1 sample size"        "Antibiotics exclusion"     </span></span>
<span><span class="co">## [25] "Sequencing type"            "16S variable region"       </span></span>
<span><span class="co">## [27] "Sequencing platform"        "Statistical test"          </span></span>
<span><span class="co">## [29] "Significance threshold"     "MHT correction"            </span></span>
<span><span class="co">## [31] "LDA Score above"            "Matched on"                </span></span>
<span><span class="co">## [33] "Confounders controlled for" "Pielou"                    </span></span>
<span><span class="co">## [35] "Shannon"                    "Chao1"                     </span></span>
<span><span class="co">## [37] "Simpson"                    "Inverse Simpson"           </span></span>
<span><span class="co">## [39] "Richness"                   "Signature page name"       </span></span>
<span><span class="co">## [41] "Source"                     "Curated date"              </span></span>
<span><span class="co">## [43] "Curator"                    "Revision editor"           </span></span>
<span><span class="co">## [45] "Description"                "Abundance in Group 1"      </span></span>
<span><span class="co">## [47] "MetaPhlAn taxon names"      "NCBI Taxonomy IDs"         </span></span>
<span><span class="co">## [49] "State"                      "Reviewer"</span></span></code></pre>
</div>
<div class="section level2">
<h2 id="subsetting">Subsetting<a class="anchor" aria-label="anchor" href="#subsetting"></a>
</h2>
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">included.pmid</span> <span class="op">&lt;-</span></span>
<span>  <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span></span>
<span>    <span class="fl">28018325</span>,</span>
<span>    <span class="fl">24614698</span>,</span>
<span>    <span class="fl">29207565</span>,</span>
<span>    <span class="fl">29459704</span>,</span>
<span>    <span class="fl">29538354</span>,</span>
<span>    <span class="fl">32012716</span>,</span>
<span>    <span class="fl">20566857</span>,</span>
<span>    <span class="fl">28512451</span>,</span>
<span>    <span class="fl">28112736</span>,</span>
<span>    <span class="fl">27362264</span></span>
<span>  <span class="op">)</span></span>
<span><span class="va">subset.dat</span> <span class="op">&lt;-</span></span>
<span>  <span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">dat</span>, <span class="va">PMID</span> <span class="op"><a href="https://rdrr.io/r/base/match.html" class="external-link">%in%</a></span> <span class="va">included.pmid</span><span class="op">)</span> </span></code></pre></div>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/base/unique.html" class="external-link">unique</a></span><span class="op">(</span><span class="va">subset.dat</span><span class="op">$</span><span class="va">`Group 0 name`</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">##  [1] "vaginal delivery (Va)"                        </span></span>
<span><span class="co">##  [2] "vaginal delivery"                             </span></span>
<span><span class="co">##  [3] "&gt;/=33 weeks"                                  </span></span>
<span><span class="co">##  [4] "vaginal delivery after 7 days of delivery"    </span></span>
<span><span class="co">##  [5] "vaginal delivery after 3 months of delivery"  </span></span>
<span><span class="co">##  [6] "vaginal delivery after 6 months of delivery"  </span></span>
<span><span class="co">##  [7] "Maternal samples taken at delivery."          </span></span>
<span><span class="co">##  [8] "Maternal samples taken at 6  weeks postpartum"</span></span>
<span><span class="co">##  [9] "vaginal delivery at day 3"                    </span></span>
<span><span class="co">## [10] "vaginal delivery at day 5"                    </span></span>
<span><span class="co">## [11] "vaginal delivery at day 28"                   </span></span>
<span><span class="co">## [12] "vaginal delivery at day 150"                  </span></span>
<span><span class="co">## [13] "vaginal delivery at day 365"                  </span></span>
<span><span class="co">## [14] "full-term delivery &gt;39 weeks at day 3"        </span></span>
<span><span class="co">## [15] "mecomium in vaginal delivery"                 </span></span>
<span><span class="co">## [16] "mecomium in C-section delivery"               </span></span>
<span><span class="co">## [17] "Antimicrobials use during delivery (no)"      </span></span>
<span><span class="co">## [18] "Maternal consumption of probiotics (NO)"      </span></span>
<span><span class="co">## [19] "Furry pets at home (NO)"</span></span></code></pre>
<div class="sourceCode" id="cb10"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">included.group0</span> <span class="op">&lt;-</span> <span class="st">"vaginal delivery"</span></span></code></pre></div>
<div class="sourceCode" id="cb11"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/base/unique.html" class="external-link">unique</a></span><span class="op">(</span><span class="va">subset.dat</span><span class="op">$</span><span class="va">`Group 1 name`</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">##  [1] "C-section (Cesarean section)"                   </span></span>
<span><span class="co">##  [2] "C-section"                                      </span></span>
<span><span class="co">##  [3] "infants &lt;33 weeks gestational age"              </span></span>
<span><span class="co">##  [4] "C-section delivery"                             </span></span>
<span><span class="co">##  [5] "Neonatal samples taken at delivery"             </span></span>
<span><span class="co">##  [6] "Infant samples taken at 6 weeks postpartum"     </span></span>
<span><span class="co">##  [7] "c-section"                                      </span></span>
<span><span class="co">##  [8] "late preterm 34-36 weeks"                       </span></span>
<span><span class="co">##  [9] "overall time point infants samples of c-section"</span></span>
<span><span class="co">## [10] "transitional stool"                             </span></span>
<span><span class="co">## [11] "Antimicrobials use during delivery (yes)"       </span></span>
<span><span class="co">## [12] "Maternal consumption of probiotics (YES)"       </span></span>
<span><span class="co">## [13] "Furry pets at home (YES)"</span></span></code></pre>
<div class="sourceCode" id="cb13"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">included.group1</span> <span class="op">&lt;-</span> <span class="st">"C-section"</span></span></code></pre></div>
<div class="sourceCode" id="cb14"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">subset.final</span> <span class="op">&lt;-</span></span>
<span>  <span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">subset.dat</span>, <span class="va">`Group 0 name`</span> <span class="op"><a href="https://rdrr.io/r/base/match.html" class="external-link">%in%</a></span> <span class="va">included.group0</span> <span class="op">&amp;</span> <span class="va">`Group 1 name`</span> <span class="op"><a href="https://rdrr.io/r/base/match.html" class="external-link">%in%</a></span> <span class="va">included.group1</span><span class="op">)</span> <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html" class="external-link">%&gt;%</a></span></span>
<span>  <span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">`Body site`</span> <span class="op">==</span> <span class="st">"Meconium"</span><span class="op">)</span> <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html" class="external-link">%&gt;%</a></span></span>
<span>  <span class="fu"><a href="https://dplyr.tidyverse.org/reference/arrange.html" class="external-link">arrange</a></span><span class="op">(</span><span class="va">PMID</span><span class="op">)</span></span></code></pre></div>
<p>Show key characteristics of the included signatures:</p>
<div class="sourceCode" id="cb15"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/detach.html" class="external-link">detach</a></span><span class="op">(</span><span class="st">"package:dplyr"</span>, unload <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## Warning: 'dplyr' namespace cannot be unloaded:</span></span>
<span><span class="co">##   namespace 'dplyr' is imported by 'BiocFileCache', 'bugSigSimple', 'tidyr', 'dbplyr' so cannot be unloaded</span></span></code></pre>
<div class="sourceCode" id="cb17"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://dplyr.tidyverse.org" class="external-link">dplyr</a></span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## </span></span>
<span><span class="co">## Attaching package: 'dplyr'</span></span></code></pre>
<pre><code><span><span class="co">## The following object is masked from 'package:kableExtra':</span></span>
<span><span class="co">## </span></span>
<span><span class="co">##     group_rows</span></span></code></pre>
<pre><code><span><span class="co">## The following objects are masked from 'package:stats':</span></span>
<span><span class="co">## </span></span>
<span><span class="co">##     filter, lag</span></span></code></pre>
<pre><code><span><span class="co">## The following objects are masked from 'package:base':</span></span>
<span><span class="co">## </span></span>
<span><span class="co">##     intersect, setdiff, setequal, union</span></span></code></pre>
<div class="sourceCode" id="cb22"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/select.html" class="external-link">select</a></span><span class="op">(</span><span class="va">subset.final</span>, <span class="st">"PMID"</span>, <span class="st">"Source"</span>, <span class="st">"Group 0 name"</span>, <span class="st">"Group 1 name"</span>, <span class="st">"Abundance in Group 1"</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">##       PMID              Source     Group 0 name Group 1 name</span></span>
<span><span class="co">## 1 24614698 Table S6 in File S1 vaginal delivery    C-section</span></span>
<span><span class="co">## 2 27362264            Figure 1 vaginal delivery    C-section</span></span>
<span><span class="co">## 3 28512451                &lt;NA&gt; vaginal delivery    C-section</span></span>
<span><span class="co">## 4 29459704            Figure 1 vaginal delivery    C-section</span></span>
<span><span class="co">## 5 29459704            Figure 1 vaginal delivery    C-section</span></span>
<span><span class="co">## 6 29538354             Table 3 vaginal delivery    C-section</span></span>
<span><span class="co">## 7 32012716     Figure 4 &amp; text vaginal delivery    C-section</span></span>
<span><span class="co">## 8 32012716     Figure 4 &amp; text vaginal delivery    C-section</span></span>
<span><span class="co">##   Abundance in Group 1</span></span>
<span><span class="co">## 1            increased</span></span>
<span><span class="co">## 2            decreased</span></span>
<span><span class="co">## 3                 &lt;NA&gt;</span></span>
<span><span class="co">## 4            increased</span></span>
<span><span class="co">## 5            decreased</span></span>
<span><span class="co">## 6            decreased</span></span>
<span><span class="co">## 7            increased</span></span>
<span><span class="co">## 8            decreased</span></span></code></pre>
<p>Are any studies missing?</p>
<div class="sourceCode" id="cb24"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/base/sort.html" class="external-link">sort</a></span><span class="op">(</span><span class="fu"><a href="https://generics.r-lib.org/reference/setops.html" class="external-link">setdiff</a></span><span class="op">(</span><span class="va">included.pmid</span>, <span class="va">subset.dat</span><span class="op">$</span><span class="va">PMID</span><span class="op">)</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## numeric(0)</span></span></code></pre>
<div class="sourceCode" id="cb26"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/base/sort.html" class="external-link">sort</a></span><span class="op">(</span><span class="fu"><a href="https://generics.r-lib.org/reference/setops.html" class="external-link">setdiff</a></span><span class="op">(</span><span class="va">included.pmid</span>, <span class="va">subset.final</span><span class="op">$</span><span class="va">PMID</span><span class="op">)</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## [1] 20566857 28018325 28112736 29207565</span></span></code></pre>
<div class="sourceCode" id="cb28"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/base/sort.html" class="external-link">sort</a></span><span class="op">(</span><span class="fu"><a href="https://generics.r-lib.org/reference/setops.html" class="external-link">setdiff</a></span><span class="op">(</span><span class="va">subset.dat</span><span class="op">$</span><span class="va">PMID</span>, <span class="va">subset.final</span><span class="op">$</span><span class="va">PMID</span><span class="op">)</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## [1] 20566857 28018325 28112736 29207565</span></span></code></pre>
</div>
<div class="section level2">
<h2 id="table-of-studies">Table of studies<a class="anchor" aria-label="anchor" href="#table-of-studies"></a>
</h2>
<p>These are the studies included in the review:</p>
<div class="sourceCode" id="cb30"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu">bugSigSimple</span><span class="fu">::</span><span class="fu"><a href="../reference/createStudyTable.html">createStudyTable</a></span><span class="op">(</span><span class="va">subset.final</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## <span style="color: #949494;"># A tibble: 6 × 5</span></span></span>
<span><span class="co">##   Study          Condition        Cases Controls `Study Design`                 </span></span>
<span><span class="co">##   <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>          <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>            <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span>    <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span> <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>                          </span></span>
<span><span class="co">## <span style="color: #BCBCBC;">1</span> Ardissone 2014 Cesarean section    33       19 cross-sectional observational,…</span></span>
<span><span class="co">## <span style="color: #BCBCBC;">2</span> Martin 2016    Cesarean section    28       80 cross-sectional observational,…</span></span>
<span><span class="co">## <span style="color: #BCBCBC;">3</span> Shi 2018       Cesarean section    10        8 cross-sectional observational,…</span></span>
<span><span class="co">## <span style="color: #BCBCBC;">4</span> Tapiainen 2018 Cesarean section    40      172 cross-sectional observational,…</span></span>
<span><span class="co">## <span style="color: #BCBCBC;">5</span> Wampach 2017   Cesarean section     6        4 time series / longitudinal obs…</span></span>
<span><span class="co">## <span style="color: #BCBCBC;">6</span> Wong 2020      Cesarean section    43       62 prospective cohort</span></span></code></pre>
</div>
<div class="section level2">
<h2 id="summary-of-taxa-reported">Summary of taxa reported<a class="anchor" aria-label="anchor" href="#summary-of-taxa-reported"></a>
</h2>
<p>This table summarizes the results for the top <code>n</code> most
frequently identified taxa.</p>
<div class="sourceCode" id="cb32"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/pkg/kableExtra/man/kable_styling.html" class="external-link">kable_styling</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/pkg/kableExtra/man/kbl.html" class="external-link">kbl</a></span><span class="op">(</span><span class="fu">bugSigSimple</span><span class="fu">::</span><span class="fu"><a href="../reference/createTaxonTable.html">createTaxonTable</a></span><span class="op">(</span><span class="va">subset.final</span>, n <span class="op">=</span> <span class="fl">20</span><span class="op">)</span><span class="op">)</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## Warning: Expected 7 pieces. Additional pieces discarded in 5 rows [6,</span></span>
<span><span class="co">## 14, 15, 16, 19].</span></span></code></pre>
<table class="table table" style="margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;">
Taxon Name
</th>
<th style="text-align:left;">
Taxonomic Level
</th>
<th style="text-align:right;">
total_signatures
</th>
<th style="text-align:right;">
increased_signatures
</th>
<th style="text-align:right;">
decreased_signatures
</th>
<th style="text-align:right;">
Binomial Test pval
</th>
<th style="text-align:left;">
kingdom
</th>
<th style="text-align:left;">
phylum
</th>
<th style="text-align:left;">
class
</th>
<th style="text-align:left;">
order
</th>
<th style="text-align:left;">
family
</th>
<th style="text-align:left;">
genus
</th>
<th style="text-align:left;">
species
</th>
<th style="text-align:right;">
n_signatures
</th>
<th style="text-align:left;">
metaphlan_name
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
Staphylococcus
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1.0
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Bacilli
</td>
<td style="text-align:left;">
Bacillales
</td>
<td style="text-align:left;">
Staphylococcaceae
</td>
<td style="text-align:left;">
Staphylococcus
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Bacillota|c__Bacilli|o__Bacillales|f__Staphylococcaceae|g__Staphylococcus
</td>
</tr>
<tr>
<td style="text-align:left;">
Corynebacterium
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.0
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
Actinomycetes
</td>
<td style="text-align:left;">
Mycobacteriales
</td>
<td style="text-align:left;">
Corynebacteriaceae
</td>
<td style="text-align:left;">
Corynebacterium
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Actinomycetota|c__Actinomycetes|o__Mycobacteriales|f__Corynebacteriaceae|g__Corynebacterium
</td>
</tr>
<tr>
<td style="text-align:left;">
Propionibacterium
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.0
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
Actinomycetes
</td>
<td style="text-align:left;">
Propionibacteriales
</td>
<td style="text-align:left;">
Propionibacteriaceae
</td>
<td style="text-align:left;">
Propionibacterium
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Actinomycetota|c__Actinomycetes|o__Propionibacteriales|f__Propionibacteriaceae|g__Propionibacterium
</td>
</tr>
<tr>
<td style="text-align:left;">
Enterococcus
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
0.5
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Bacilli
</td>
<td style="text-align:left;">
Lactobacillales
</td>
<td style="text-align:left;">
Enterococcaceae
</td>
<td style="text-align:left;">
Enterococcus
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Bacillota|c__Bacilli|o__Lactobacillales|f__Enterococcaceae|g__Enterococcus
</td>
</tr>
<tr>
<td style="text-align:left;">
Streptococcus
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.0
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Bacilli
</td>
<td style="text-align:left;">
Lactobacillales
</td>
<td style="text-align:left;">
Streptococcaceae
</td>
<td style="text-align:left;">
Streptococcus
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Bacillota|c__Bacilli|o__Lactobacillales|f__Streptococcaceae|g__Streptococcus
</td>
</tr>
<tr>
<td style="text-align:left;">
Phocaeicola vulgatus
</td>
<td style="text-align:left;">
species
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.0
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Pseudomonadati
</td>
<td style="text-align:left;">
Bacteroidota
</td>
<td style="text-align:left;">
Bacteroidia
</td>
<td style="text-align:left;">
Bacteroidales
</td>
<td style="text-align:left;">
Bacteroidaceae
</td>
<td style="text-align:left;">
Phocaeicola
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
k__Bacteria|k__Pseudomonadati|p__Bacteroidota|c__Bacteroidia|o__Bacteroidales|f__Bacteroidaceae|g__Phocaeicola|s__Phocaeicola
vulgatus
</td>
</tr>
<tr>
<td style="text-align:left;">
Comamonas
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.0
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Pseudomonadati
</td>
<td style="text-align:left;">
Pseudomonadota
</td>
<td style="text-align:left;">
Betaproteobacteria
</td>
<td style="text-align:left;">
Burkholderiales
</td>
<td style="text-align:left;">
Comamonadaceae
</td>
<td style="text-align:left;">
Comamonas
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
k__Bacteria|k__Pseudomonadati|p__Pseudomonadota|c__Betaproteobacteria|o__Burkholderiales|f__Comamonadaceae|g__Comamonas
</td>
</tr>
<tr>
<td style="text-align:left;">
Citrobacter
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.0
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Pseudomonadati
</td>
<td style="text-align:left;">
Pseudomonadota
</td>
<td style="text-align:left;">
Gammaproteobacteria
</td>
<td style="text-align:left;">
Enterobacterales
</td>
<td style="text-align:left;">
Enterobacteriaceae
</td>
<td style="text-align:left;">
Citrobacter
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
k__Bacteria|k__Pseudomonadati|p__Pseudomonadota|c__Gammaproteobacteria|o__Enterobacterales|f__Enterobacteriaceae|g__Citrobacter
</td>
</tr>
<tr>
<td style="text-align:left;">
Stenotrophomonas
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.5
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Pseudomonadati
</td>
<td style="text-align:left;">
Pseudomonadota
</td>
<td style="text-align:left;">
Gammaproteobacteria
</td>
<td style="text-align:left;">
Lysobacterales
</td>
<td style="text-align:left;">
Lysobacteraceae
</td>
<td style="text-align:left;">
Stenotrophomonas
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
k__Bacteria|k__Pseudomonadati|p__Pseudomonadota|c__Gammaproteobacteria|o__Lysobacterales|f__Lysobacteraceae|g__Stenotrophomonas
</td>
</tr>
<tr>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
phylum
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.0
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Actinomycetota
</td>
</tr>
<tr>
<td style="text-align:left;">
Actinomycetes
</td>
<td style="text-align:left;">
class
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.0
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
Actinomycetes
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Actinomycetota|c__Actinomycetes
</td>
</tr>
<tr>
<td style="text-align:left;">
Actinomycetales
</td>
<td style="text-align:left;">
order
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.0
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
Actinomycetes
</td>
<td style="text-align:left;">
Actinomycetales
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Actinomycetota|c__Actinomycetes|o__Actinomycetales
</td>
</tr>
<tr>
<td style="text-align:left;">
Bifidobacterium
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.0
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
Actinomycetes
</td>
<td style="text-align:left;">
Bifidobacteriales
</td>
<td style="text-align:left;">
Bifidobacteriaceae
</td>
<td style="text-align:left;">
Bifidobacterium
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Actinomycetota|c__Actinomycetes|o__Bifidobacteriales|f__Bifidobacteriaceae|g__Bifidobacterium
</td>
</tr>
<tr>
<td style="text-align:left;">
Bifidobacterium bifidum
</td>
<td style="text-align:left;">
species
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.0
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
Actinomycetes
</td>
<td style="text-align:left;">
Bifidobacteriales
</td>
<td style="text-align:left;">
Bifidobacteriaceae
</td>
<td style="text-align:left;">
Bifidobacterium
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Actinomycetota|c__Actinomycetes|o__Bifidobacteriales|f__Bifidobacteriaceae|g__Bifidobacterium|s__Bifidobacterium
bifidum
</td>
</tr>
<tr>
<td style="text-align:left;">
Bifidobacterium catenulatum
</td>
<td style="text-align:left;">
species
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.0
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
Actinomycetes
</td>
<td style="text-align:left;">
Bifidobacteriales
</td>
<td style="text-align:left;">
Bifidobacteriaceae
</td>
<td style="text-align:left;">
Bifidobacterium
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Actinomycetota|c__Actinomycetes|o__Bifidobacteriales|f__Bifidobacteriaceae|g__Bifidobacterium|s__Bifidobacterium
catenulatum
</td>
</tr>
<tr>
<td style="text-align:left;">
Bifidobacterium longum subsp. longum
</td>
<td style="text-align:left;">
species
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.0
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
Actinomycetes
</td>
<td style="text-align:left;">
Bifidobacteriales
</td>
<td style="text-align:left;">
Bifidobacteriaceae
</td>
<td style="text-align:left;">
Bifidobacterium
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Actinomycetota|c__Actinomycetes|o__Bifidobacteriales|f__Bifidobacteriaceae|g__Bifidobacterium|s__Bifidobacterium
longum|s__Bifidobacterium longum subsp. longum
</td>
</tr>
<tr>
<td style="text-align:left;">
Pseudoscardovia
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
1.0
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
Actinomycetes
</td>
<td style="text-align:left;">
Bifidobacteriales
</td>
<td style="text-align:left;">
Bifidobacteriaceae
</td>
<td style="text-align:left;">
Pseudoscardovia
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Actinomycetota|c__Actinomycetes|o__Bifidobacteriales|f__Bifidobacteriaceae|g__Pseudoscardovia
</td>
</tr>
<tr>
<td style="text-align:left;">
Propionibacteriaceae
</td>
<td style="text-align:left;">
family
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.0
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
Actinomycetes
</td>
<td style="text-align:left;">
Propionibacteriales
</td>
<td style="text-align:left;">
Propionibacteriaceae
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Actinomycetota|c__Actinomycetes|o__Propionibacteriales|f__Propionibacteriaceae
</td>
</tr>
<tr>
<td style="text-align:left;">
Cutibacterium acnes
</td>
<td style="text-align:left;">
species
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.0
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
Actinomycetes
</td>
<td style="text-align:left;">
Propionibacteriales
</td>
<td style="text-align:left;">
Propionibacteriaceae
</td>
<td style="text-align:left;">
Cutibacterium
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Actinomycetota|c__Actinomycetes|o__Propionibacteriales|f__Propionibacteriaceae|g__Cutibacterium|s__Cutibacterium
acnes
</td>
</tr>
<tr>
<td style="text-align:left;">
Pseudonocardiaceae
</td>
<td style="text-align:left;">
family
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.0
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
Actinomycetes
</td>
<td style="text-align:left;">
Pseudonocardiales
</td>
<td style="text-align:left;">
Pseudonocardiaceae
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Actinomycetota|c__Actinomycetes|o__Pseudonocardiales|f__Pseudonocardiaceae
</td>
</tr>
</tbody>
</table>
</div>
<div class="section level2">
<h2 id="long-list-of-most-frequently-identified-taxa">Long list of most frequently identified taxa<a class="anchor" aria-label="anchor" href="#long-list-of-most-frequently-identified-taxa"></a>
</h2>
<p>These are not needed because of the taxon table above, but they list
a larger number of taxa.</p>
<div class="sourceCode" id="cb34"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="../reference/getMostFrequentTaxa.html">getMostFrequentTaxa</a></span><span class="op">(</span><span class="va">subset.final</span>, n <span class="op">=</span> <span class="fl">50</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">##    1279    1301    1350    1716    1743     283   40323     544     821  114248 </span></span>
<span><span class="co">##       3       2       2       2       2       2       2       2       2       1 </span></span>
<span><span class="co">##  117563  118964    1236    1239    1243   12916    1297    1298 1302778  135614 </span></span>
<span><span class="co">##       1       1       1       1       1       1       1       1       1       1 </span></span>
<span><span class="co">##    1357    1380    1385    1386    1390    1402    1485    1502  150247    1578 </span></span>
<span><span class="co">##       1       1       1       1       1       1       1       1       1       1 </span></span>
<span><span class="co">##  158851    1596    1598    1678    1679    1681    1686    1730    1747    1760 </span></span>
<span><span class="co">##       1       1       1       1       1       1       1       1       1       1 </span></span>
<span><span class="co">##  183710  186806  186817  188787  201174    2037    2070  216851     222    2737 </span></span>
<span><span class="co">##       1       1       1       1       1       1       1       1       1       1</span></span></code></pre>
<div class="sourceCode" id="cb36"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="../reference/getMostFrequentTaxa.html">getMostFrequentTaxa</a></span><span class="op">(</span><span class="va">subset.final</span>, direction<span class="op">=</span><span class="st">"UP"</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">##  40323 114248 117563 118964   1239   1243   1279  12916   1297   1298 </span></span>
<span><span class="co">##      2      1      1      1      1      1      1      1      1      1</span></span></code></pre>
<div class="sourceCode" id="cb38"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="../reference/getMostFrequentTaxa.html">getMostFrequentTaxa</a></span><span class="op">(</span><span class="va">subset.final</span>, direction<span class="op">=</span><span class="st">"DOWN"</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">##   1279   1350   1236   1301   1380   1485   1502 158851   1596   1598 </span></span>
<span><span class="co">##      2      2      1      1      1      1      1      1      1      1</span></span></code></pre>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

      </div>

</div>



      <footer><div class="copyright">
  <p></p>
<p>Developed by Ludwig Geistlinger, Chloe Mirzayi, Fatima Zohra, Levi Waldron.</p>
</div>

<div class="pkgdown">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.1.1.</p>
</div>

      </footer>
</div>






  </body>
</html>
