<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Distinguishing socioeconomic and racial/ethnic variation of the human gut microbiome by re-analysis of studies from around the world • Reviews of published signatures based on bugsigdb.org</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.7.1/jquery.min.js" integrity="sha512-v2CJ7UaYy4JwqLDIrZUI/4hqeoQieOmAZNXBeQyjo21dadnwR+8ZaIJVT8EE2iyI61OV8e6M8PP2/4hpQINQ/g==" crossorigin="anonymous" referrerpolicy="no-referrer"></script><!-- Bootstrap --><link href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.4.0/flatly/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="Distinguishing socioeconomic and racial/ethnic variation of the human gut microbiome by re-analysis of studies from around the world">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">


    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">Reviews of published signatures based on bugsigdb.org</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">0.99.5</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Articles

    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/Bearkland%20capstone%20DED%2012.2.2024.html">Differential Abundance in the ocular microbiome in DED</a>
    </li>
    <li>
      <a href="../articles/c-section_meconium_shaimaa.html">Delivery Mode and the Meconium Microbiome</a>
    </li>
    <li>
      <a href="../articles/capstoneanalysis_clare.html">COVID-19 associated microbiome</a>
    </li>
    <li>
      <a href="../articles/capstoneanalysis_fatima.html">An analysis of Most frequent taxa in Major Depression and Bipolar Disorder</a>
    </li>
    <li>
      <a href="../articles/capstoneanalysis_ifeanyi.html">Distinguishing socioeconomic and racial/ethnic variation of the human gut microbiome by re-analysis of studies from around the world</a>
    </li>
    <li>
      <a href="../articles/capstoneanalysis_kweku.html">The Irritable Bowel Syndrome-associated Microbiome</a>
    </li>
    <li>
      <a href="../articles/fieldworkanalysis_samara.html">The Endometriosis-associated Microbiome</a>
    </li>
    <li>
      <a href="../articles/healthysig.html">Analysis of prevalent stool genera in bugsigdb.org</a>
    </li>
  </ul>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/waldronlab/bugSigSimple" class="external-link">
    <span class="fa fa-github"></span>

  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->



      </header><script src="capstoneanalysis_ifeanyi_files/kePrint-0.0.1/kePrint.js"></script><link href="capstoneanalysis_ifeanyi_files/lightable-0.0.1/lightable.css" rel="stylesheet">
<div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>Distinguishing socioeconomic and racial/ethnic
variation of the human gut microbiome by re-analysis of studies from
around the world</h1>
                        <h4 data-toc-skip class="author">Ifeanyi
Kalu</h4>
            
      

      <div class="hidden name"><code>capstoneanalysis_ifeanyi.Rmd</code></div>

    </div>

    
    
<div class="section level2">
<h2 id="objectives-of-the-study">Objectives of the study<a class="anchor" aria-label="anchor" href="#objectives-of-the-study"></a>
</h2>
<p>To assess commonality or heterogeneity in socioeconomic and ethnic
patterning of the fecal microbiome in studies undertaken in various
parts of the world, through cluster analysis of results on
differentially abundant taxa.</p>
</div>
<div class="section level2">
<h2 id="data-analysis-plan">Data Analysis Plan<a class="anchor" aria-label="anchor" href="#data-analysis-plan"></a>
</h2>
<ol style="list-style-type: decimal">
<li><p>creating frequency tables of how many signatures each microbial
taxon (species, genus, or other rank) has been reported – by
race/ethnicity separately, by SES separately, and for all studies
combined.</p></li>
<li><p>cluster analysis of the signatures. Similarity between all pairs
of signatures will be calculated using Jaccard Distance, to create a
distance matrix. This distance matrix will be used for hierarchical
clustering, to identify subgroups of similar signatures. This analysis
will be performed separately for signatures of increased abundance and
of decreased abundance (in low SES and non-white/Caucasian ethnicity)
repeated using Semantic Similarity as an alternative measure of
signature similarity, in sensitivity analysis.</p></li>
<li><p>Common and distinct effects of SES and race/ethnicity on the gut
microbiome will be identified using meta-regression for the most
identified taxa. Specifically, the signature will be the unit of
analysis, with the presence or absence of the taxon in each study coded
as a 0/1 binary variable in a regression analysis.</p></li>
</ol>
</div>
<div class="section level2">
<h2 id="install-packages">Install Packages<a class="anchor" aria-label="anchor" href="#install-packages"></a>
</h2>
<p>This is not evaluated, run manually if needed.</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw">if</span> <span class="op">(</span><span class="op">!</span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">require</a></span><span class="op">(</span><span class="st"><a href="https://bioconductor.github.io/BiocManager/" class="external-link">"BiocManager"</a></span>, quietly <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span><span class="op">)</span></span>
<span>    <span class="fu"><a href="https://rdrr.io/r/utils/install.packages.html" class="external-link">install.packages</a></span><span class="op">(</span><span class="st">"BiocManager"</span><span class="op">)</span></span>
<span><span class="fu">BiocManager</span><span class="fu">::</span><span class="fu"><a href="https://bioconductor.github.io/BiocManager/reference/install.html" class="external-link">install</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="fu">BiocManager</span><span class="fu">::</span><span class="fu"><a href="https://bioconductor.github.io/BiocManager/reference/version.html" class="external-link">version</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="fu">BiocManager</span><span class="fu">::</span><span class="fu"><a href="https://bioconductor.github.io/BiocManager/reference/install.html" class="external-link">install</a></span><span class="op">(</span><span class="st">"remotes"</span>, dependencies <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span>
<span><span class="fu">BiocManager</span><span class="fu">::</span><span class="fu"><a href="https://bioconductor.github.io/BiocManager/reference/install.html" class="external-link">install</a></span><span class="op">(</span><span class="st">"waldronlab/bugSigSimple"</span><span class="op">)</span></span>
<span><span class="fu">BiocManager</span><span class="fu">::</span><span class="fu"><a href="https://bioconductor.github.io/BiocManager/reference/install.html" class="external-link">install</a></span><span class="op">(</span><span class="st">"curatedMetagenomicData"</span><span class="op">)</span></span>
<span><span class="fu">devtools</span><span class="fu">::</span><span class="fu"><a href="https://remotes.r-lib.org/reference/install_github.html" class="external-link">install_github</a></span><span class="op">(</span><span class="st">"waldronlab/BugSigDBStats"</span><span class="op">)</span></span>
<span><span class="fu">devtools</span><span class="fu">::</span><span class="fu"><a href="https://remotes.r-lib.org/reference/install_github.html" class="external-link">install_github</a></span><span class="op">(</span><span class="st">"waldronlab/bugsigdbr"</span><span class="op">)</span></span></code></pre></div>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/base/message.html" class="external-link">suppressPackageStartupMessages</a></span><span class="op">(</span><span class="op">{</span></span>
<span>  <span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://waldronlab.io/bugSigSimple" class="external-link">bugSigSimple</a></span><span class="op">)</span></span>
<span>  <span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/waldronlab/BugSigDBStats" class="external-link">BugSigDBStats</a></span><span class="op">)</span></span>
<span>  <span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/waldronlab/bugsigdbr" class="external-link">bugsigdbr</a></span><span class="op">)</span></span>
<span>  <span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://tidyverse.tidyverse.org" class="external-link">tidyverse</a></span><span class="op">)</span></span>
<span>  <span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://stringr.tidyverse.org" class="external-link">stringr</a></span><span class="op">)</span></span>
<span>  <span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="http://haozhu233.github.io/kableExtra/" class="external-link">kableExtra</a></span><span class="op">)</span></span>
<span>  <span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://dplyr.tidyverse.org" class="external-link">dplyr</a></span><span class="op">)</span></span>
<span>  <span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/jokergoo/ComplexHeatmap" class="external-link">ComplexHeatmap</a></span><span class="op">)</span></span>
<span><span class="op">}</span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level2">
<h2 id="import-data-from-bugsigdb-database">Import data from Bugsigdb database<a class="anchor" aria-label="anchor" href="#import-data-from-bugsigdb-database"></a>
</h2>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">dat</span> <span class="op">&lt;-</span> <span class="fu">bugsigdbr</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/bugsigdbr/man/importBugSigDB.html" class="external-link">importBugSigDB</a></span><span class="op">(</span>version <span class="op">=</span> <span class="st">'devel'</span>, cache <span class="op">=</span> <span class="cn">FALSE</span><span class="op">)</span> <span class="co">#11/27/2022</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/dim.html" class="external-link">dim</a></span><span class="op">(</span><span class="va">dat</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## [1] 7349   50</span></span></code></pre>
</div>
<div class="section level2">
<h2 id="subset-signatures-by-curator-and-conditions">Subset Signatures by Curator and Conditions<a class="anchor" aria-label="anchor" href="#subset-signatures-by-curator-and-conditions"></a>
</h2>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">dat.sub</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/subsetByCurator.html">subsetByCurator</a></span><span class="op">(</span><span class="va">dat</span>, <span class="st">"Kaluifeanyi101"</span><span class="op">)</span> <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html" class="external-link">%&gt;%</a></span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">`Body site`</span> <span class="op">==</span> <span class="st">"Feces"</span><span class="op">)</span> <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html" class="external-link">%&gt;%</a></span></span>
<span>  <span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate.html" class="external-link">mutate</a></span><span class="op">(</span>`Body site` <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/chartr.html" class="external-link">tolower</a></span><span class="op">(</span><span class="va">`Body site`</span><span class="op">)</span><span class="op">)</span> <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html" class="external-link">%&gt;%</a></span></span>
<span>  <span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate.html" class="external-link">mutate</a></span><span class="op">(</span>`Condition` <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/chartr.html" class="external-link">tolower</a></span><span class="op">(</span><span class="va">`Condition`</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/table.html" class="external-link">table</a></span><span class="op">(</span><span class="va">dat.sub</span><span class="op">[</span>,<span class="st">"Condition"</span><span class="op">]</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## </span></span>
<span><span class="co">##                                ethnic group </span></span>
<span><span class="co">##                                          31 </span></span>
<span><span class="co">## place of residence measurement,ethnic group </span></span>
<span><span class="co">##                                           2 </span></span>
<span><span class="co">##                                 risk factor </span></span>
<span><span class="co">##                                           2 </span></span>
<span><span class="co">##                               sampling time </span></span>
<span><span class="co">##                                           2 </span></span>
<span><span class="co">##                        socioeconomic status </span></span>
<span><span class="co">##                                          25 </span></span>
<span><span class="co">##           socioeconomic status,ethnic group </span></span>
<span><span class="co">##                                           2</span></span></code></pre>
</div>
<div class="section level2">
<h2 id="table-of-studies">Table of studies<a class="anchor" aria-label="anchor" href="#table-of-studies"></a>
</h2>
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu">bugSigSimple</span><span class="fu">::</span><span class="fu"><a href="../reference/createStudyTable.html">createStudyTable</a></span><span class="op">(</span><span class="va">dat.sub</span><span class="op">)</span> <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html" class="external-link">%&gt;%</a></span></span>
<span>  <span class="fu"><a href="https://rdrr.io/pkg/kableExtra/man/kbl.html" class="external-link">kbl</a></span><span class="op">(</span><span class="op">)</span> <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html" class="external-link">%&gt;%</a></span></span>
<span>  <span class="fu"><a href="https://rdrr.io/pkg/kableExtra/man/kable_styling.html" class="external-link">kable_styling</a></span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
<table class="table table" style="margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;">
Study code
</th>
<th style="text-align:right;">
MaxCases
</th>
<th style="text-align:right;">
MaxControls
</th>
<th style="text-align:left;">
Study design
</th>
<th style="text-align:left;">
Condition
</th>
<th style="text-align:right;">
N_signatures
</th>
<th style="text-align:left;">
PMID
</th>
<th style="text-align:left;">
DOI
</th>
<th style="text-align:left;">
URL
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
AmaruddinAI_2020
</td>
<td style="text-align:right;">
74
</td>
<td style="text-align:right;">
66
</td>
<td style="text-align:left;">
cross-sectional observational, not case-control
</td>
<td style="text-align:left;">
socioeconomic status
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
32604882
</td>
<td style="text-align:left;">
<a href="https://doi.org/10.3390/microorganisms8060961" class="external-link uri">https://doi.org/10.3390/microorganisms8060961</a>
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
AngQY_2021
</td>
<td style="text-align:right;">
22
</td>
<td style="text-align:right;">
24
</td>
<td style="text-align:left;">
cross-sectional observational, not case-control
</td>
<td style="text-align:left;">
ethnic group
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
34617511
</td>
<td style="text-align:left;">
<a href="https://doi.org/10.7554/eLife.70349" class="external-link uri">https://doi.org/10.7554/eLife.70349</a>
</td>
<td style="text-align:left;">
<a href="https://pubmed.ncbi.nlm.nih.gov/34617511" class="external-link uri">https://pubmed.ncbi.nlm.nih.gov/34617511</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
BalakrishnanB_2021
</td>
<td style="text-align:right;">
30
</td>
<td style="text-align:right;">
30
</td>
<td style="text-align:left;">
cross-sectional observational, not case-control
</td>
<td style="text-align:left;">
ethnic group; socioeconomic status
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
33596768
</td>
<td style="text-align:left;">
<a href="https://doi.org/10.1080/19490976.2021.1882926" class="external-link uri">https://doi.org/10.1080/19490976.2021.1882926</a>
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
BowyerRCE_2019
</td>
<td style="text-align:right;">
342
</td>
<td style="text-align:right;">
457
</td>
<td style="text-align:left;">
cross-sectional observational, not case-control
</td>
<td style="text-align:left;">
socioeconomic status
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
30641975
</td>
<td style="text-align:left;">
<a href="https://doi.org/10.3390/microorganisms7010017" class="external-link uri">https://doi.org/10.3390/microorganisms7010017</a>
</td>
<td style="text-align:left;">
<a href="https://pubmed.ncbi.nlm.nih.gov/30641975/" class="external-link uri">https://pubmed.ncbi.nlm.nih.gov/30641975/</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
BrooksAW_2018
</td>
<td style="text-align:right;">
88
</td>
<td style="text-align:right;">
1237
</td>
<td style="text-align:left;">
cross-sectional observational, not case-control
</td>
<td style="text-align:left;">
ethnic group
</td>
<td style="text-align:right;">
12
</td>
<td style="text-align:left;">
30513082
</td>
<td style="text-align:left;">
<a href="https://doi.org/10.1371/journal.pbio.2006842" class="external-link uri">https://doi.org/10.1371/journal.pbio.2006842</a>
</td>
<td style="text-align:left;">
<a href="https://pubmed.ncbi.nlm.nih.gov/30513082/" class="external-link uri">https://pubmed.ncbi.nlm.nih.gov/30513082/</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
CarsonTL_2018
</td>
<td style="text-align:right;">
47
</td>
<td style="text-align:right;">
33
</td>
<td style="text-align:left;">
cross-sectional observational, not case-control
</td>
<td style="text-align:left;">
ethnic group
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
29901485
</td>
<td style="text-align:left;">
<a href="https://doi.org/10.1097/PSY.0000000000000614" class="external-link uri">https://doi.org/10.1097/PSY.0000000000000614</a>.
</td>
<td style="text-align:left;">
<a href="https://pubmed.ncbi.nlm.nih.gov/29901485/" class="external-link uri">https://pubmed.ncbi.nlm.nih.gov/29901485/</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
ChongCW_2015
</td>
<td style="text-align:right;">
20
</td>
<td style="text-align:right;">
24
</td>
<td style="text-align:left;">
cross-sectional observational, not case-control
</td>
<td style="text-align:left;">
socioeconomic status
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
26290472
</td>
<td style="text-align:left;">
<a href="https://doi.org/10.1038/srep13338" class="external-link uri">https://doi.org/10.1038/srep13338</a>
</td>
<td style="text-align:left;">
<a href="https://pubmed.ncbi.nlm.nih.gov/26290472/" class="external-link uri">https://pubmed.ncbi.nlm.nih.gov/26290472/</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
DeFilippoC_2010
</td>
<td style="text-align:right;">
14
</td>
<td style="text-align:right;">
15
</td>
<td style="text-align:left;">
cross-sectional observational, not case-control
</td>
<td style="text-align:left;">
place of residence measurement,ethnic group
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
20679230
</td>
<td style="text-align:left;">
<a href="https://doi.org/10.1073/pnas.1005963107" class="external-link uri">https://doi.org/10.1073/pnas.1005963107</a>
</td>
<td style="text-align:left;">
<a href="https://pubmed.ncbi.nlm.nih.gov/20679230/" class="external-link uri">https://pubmed.ncbi.nlm.nih.gov/20679230/</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
DwiyantoJ_2021
</td>
<td style="text-align:right;">
54
</td>
<td style="text-align:right;">
65
</td>
<td style="text-align:left;">
cross-sectional observational, not case-control
</td>
<td style="text-align:left;">
ethnic group
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:left;">
33514807
</td>
<td style="text-align:left;">
<a href="https://doi.org/10.1038/s41598-021-82311-3" class="external-link uri">https://doi.org/10.1038/s41598-021-82311-3</a>
</td>
<td style="text-align:left;">
<a href="https://pubmed.ncbi.nlm.nih.gov/33514807/" class="external-link uri">https://pubmed.ncbi.nlm.nih.gov/33514807/</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
EscobarJS_2014
</td>
<td style="text-align:right;">
30
</td>
<td style="text-align:right;">
54
</td>
<td style="text-align:left;">
cross-sectional observational, not case-control
</td>
<td style="text-align:left;">
ethnic group
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
25495462
</td>
<td style="text-align:left;">
<a href="https://doi.org/10.1186/s12866-014-0311-6" class="external-link uri">https://doi.org/10.1186/s12866-014-0311-6</a>
</td>
<td style="text-align:left;">
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4275940/" class="external-link uri">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4275940/</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
GreenhillAR_2015
</td>
<td style="text-align:right;">
86
</td>
<td style="text-align:right;">
29
</td>
<td style="text-align:left;">
cross-sectional observational, not case-control
</td>
<td style="text-align:left;">
socioeconomic status
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
25658868
</td>
<td style="text-align:left;">
<a href="https://doi.org/10.1371/journal.pone.0117427" class="external-link uri">https://doi.org/10.1371/journal.pone.0117427</a>
</td>
<td style="text-align:left;">
<a href="https://pubmed.ncbi.nlm.nih.gov/25658868/" class="external-link uri">https://pubmed.ncbi.nlm.nih.gov/25658868/</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
GrześkowiakŁ_2012
</td>
<td style="text-align:right;">
44
</td>
<td style="text-align:right;">
31
</td>
<td style="text-align:left;">
cross-sectional observational, not case-control
</td>
<td style="text-align:left;">
socioeconomic status
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
22228076
</td>
<td style="text-align:left;">
<a href="https://doi.org/10.1097/MPG.0b013e318249039c" class="external-link uri">https://doi.org/10.1097/MPG.0b013e318249039c</a>.
</td>
<td style="text-align:left;">
<a href="https://pubmed.ncbi.nlm.nih.gov/22228076/" class="external-link uri">https://pubmed.ncbi.nlm.nih.gov/22228076/</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
HeY_2018
</td>
<td style="text-align:right;">
7009
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:left;">
cross-sectional observational, not case-control
</td>
<td style="text-align:left;">
socioeconomic status
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
30249275
</td>
<td style="text-align:left;">
<a href="https://doi.org/10.1186/s40168-018-0557-6" class="external-link uri">https://doi.org/10.1186/s40168-018-0557-6</a>.
</td>
<td style="text-align:left;">
<a href="https://pubmed.ncbi.nlm.nih.gov/30249275/" class="external-link uri">https://pubmed.ncbi.nlm.nih.gov/30249275/</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
KortekangasE_2020
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:left;">
prospective cohort
</td>
<td style="text-align:left;">
sampling time
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
32011017
</td>
<td style="text-align:left;">
<a href="https://doi.org/10.1111/ppe.12623" class="external-link uri">https://doi.org/10.1111/ppe.12623</a>
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
LapidotY_2021
</td>
<td style="text-align:right;">
70
</td>
<td style="text-align:right;">
69
</td>
<td style="text-align:left;">
cross-sectional observational, not case-control
</td>
<td style="text-align:left;">
socioeconomic status
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
34444813
</td>
<td style="text-align:left;">
<a href="https://doi.org/10.3390/nu13082645" class="external-link uri">https://doi.org/10.3390/nu13082645</a>
</td>
<td style="text-align:left;">
<a href="https://pubmed.ncbi.nlm.nih.gov/34444813/" class="external-link uri">https://pubmed.ncbi.nlm.nih.gov/34444813/</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
LevinAM_2016
</td>
<td style="text-align:right;">
168
</td>
<td style="text-align:right;">
130
</td>
<td style="text-align:left;">
prospective cohort
</td>
<td style="text-align:left;">
risk factor
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
27558272
</td>
<td style="text-align:left;">
<a href="https://doi.org/10.1038/srep31775" class="external-link uri">https://doi.org/10.1038/srep31775</a>
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
LewisCR_2021
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:left;">
cross-sectional observational, not case-control
</td>
<td style="text-align:left;">
socioeconomic status
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
34442687
</td>
<td style="text-align:left;">
<a href="https://doi.org/10.3390/microorganisms9081608" class="external-link uri">https://doi.org/10.3390/microorganisms9081608</a>
</td>
<td style="text-align:left;">
<a href="https://pubmed.ncbi.nlm.nih.gov/34442687/" class="external-link uri">https://pubmed.ncbi.nlm.nih.gov/34442687/</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
LinA_2013
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
cross-sectional observational, not case-control
</td>
<td style="text-align:left;">
socioeconomic status,ethnic group
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
23349750
</td>
<td style="text-align:left;">
<a href="https://doi.org/10.1371/journal.pone.0053838" class="external-link uri">https://doi.org/10.1371/journal.pone.0053838</a>
</td>
<td style="text-align:left;">
<a href="https://pubmed.ncbi.nlm.nih.gov/23349750/" class="external-link uri">https://pubmed.ncbi.nlm.nih.gov/23349750/</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
MelloCS_2016
</td>
<td style="text-align:right;">
100
</td>
<td style="text-align:right;">
30
</td>
<td style="text-align:left;">
cross-sectional observational, not case-control
</td>
<td style="text-align:left;">
socioeconomic status
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
26982745
</td>
<td style="text-align:left;">
<a href="https://doi.org/10.1097/MPG.0000000000001186" class="external-link uri">https://doi.org/10.1097/MPG.0000000000001186</a>
</td>
<td style="text-align:left;">
<a href="https://pubmed.ncbi.nlm.nih.gov/26982745/" class="external-link uri">https://pubmed.ncbi.nlm.nih.gov/26982745/</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
MillerGE_2016
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
44
</td>
<td style="text-align:left;">
cross-sectional observational, not case-control
</td>
<td style="text-align:left;">
socioeconomic status
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
26859894
</td>
<td style="text-align:left;">
<a href="https://doi.org/10.1371/journal.pone.0148952" class="external-link uri">https://doi.org/10.1371/journal.pone.0148952</a>
</td>
<td style="text-align:left;">
<a href="https://pubmed.ncbi.nlm.nih.gov/26859894/" class="external-link uri">https://pubmed.ncbi.nlm.nih.gov/26859894/</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
Moran-RamosS_2020
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:left;">
cross-sectional observational, not case-control
</td>
<td style="text-align:left;">
socioeconomic status
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
31973685
</td>
<td style="text-align:left;">
<a href="https://doi.org/10.1080/19490976.2020.1712985" class="external-link uri">https://doi.org/10.1080/19490976.2020.1712985</a>
</td>
<td style="text-align:left;">
<a href="https://pubmed.ncbi.nlm.nih.gov/31973685/" class="external-link uri">https://pubmed.ncbi.nlm.nih.gov/31973685/</a>
</td>
</tr>
</tbody>
</table>
</div>
<div class="section level2">
<h2 id="taxon-frequency-table-by-ses">Taxon Frequency Table by SES<a class="anchor" aria-label="anchor" href="#taxon-frequency-table-by-ses"></a>
</h2>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">dat.sub_SES</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">dat.sub</span>, <span class="va">dat.sub</span><span class="op">$</span><span class="va">Condition</span> <span class="op">==</span> <span class="st">"socioeconomic status"</span><span class="op">)</span> </span>
<span><span class="fu"><a href="https://rdrr.io/r/base/dim.html" class="external-link">dim</a></span><span class="op">(</span><span class="va">dat.sub_SES</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## [1] 25 50</span></span></code></pre>
<div class="sourceCode" id="cb10"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu">bugSigSimple</span><span class="fu">::</span><span class="fu"><a href="../reference/createTaxonTable.html">createTaxonTable</a></span><span class="op">(</span><span class="va">dat.sub_SES</span><span class="op">)</span>  <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html" class="external-link">%&gt;%</a></span></span>
<span>  <span class="fu"><a href="https://rdrr.io/pkg/kableExtra/man/kbl.html" class="external-link">kbl</a></span><span class="op">(</span><span class="op">)</span> <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html" class="external-link">%&gt;%</a></span></span>
<span>  <span class="fu"><a href="https://rdrr.io/pkg/kableExtra/man/kable_styling.html" class="external-link">kable_styling</a></span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
<table class="table table" style="margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;">
Taxon Name
</th>
<th style="text-align:left;">
Taxonomic Level
</th>
<th style="text-align:right;">
total_signatures
</th>
<th style="text-align:right;">
increased_signatures
</th>
<th style="text-align:right;">
decreased_signatures
</th>
<th style="text-align:right;">
Binomial Test pval
</th>
<th style="text-align:left;">
kingdom
</th>
<th style="text-align:left;">
phylum
</th>
<th style="text-align:left;">
class
</th>
<th style="text-align:left;">
order
</th>
<th style="text-align:left;">
family
</th>
<th style="text-align:left;">
genus
</th>
<th style="text-align:left;">
species
</th>
<th style="text-align:right;">
n_signatures
</th>
<th style="text-align:left;">
metaphlan_name
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
Prevotella
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
0.37
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Pseudomonadati
</td>
<td style="text-align:left;">
Bacteroidota
</td>
<td style="text-align:left;">
Bacteroidia
</td>
<td style="text-align:left;">
Bacteroidales
</td>
<td style="text-align:left;">
Prevotellaceae
</td>
<td style="text-align:left;">
Prevotella
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
k__Bacteria|k__Pseudomonadati|p__Bacteroidota|c__Bacteroidia|o__Bacteroidales|f__Prevotellaceae|g__Prevotella
</td>
</tr>
<tr>
<td style="text-align:left;">
Eubacterium
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.63
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Eubacteriales
</td>
<td style="text-align:left;">
Eubacteriaceae
</td>
<td style="text-align:left;">
Eubacterium
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Bacillota|c__Clostridia|o__Eubacteriales|f__Eubacteriaceae|g__Eubacterium
</td>
</tr>
<tr>
<td style="text-align:left;">
Ruminococcus
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.63
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Eubacteriales
</td>
<td style="text-align:left;">
Oscillospiraceae
</td>
<td style="text-align:left;">
Ruminococcus
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Bacillota|c__Clostridia|o__Eubacteriales|f__Oscillospiraceae|g__Ruminococcus
</td>
</tr>
<tr>
<td style="text-align:left;">
Bacteroidota
</td>
<td style="text-align:left;">
phylum
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.12
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Pseudomonadati
</td>
<td style="text-align:left;">
Bacteroidota
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
17
</td>
<td style="text-align:left;">
k__Bacteria|k__Pseudomonadati|p__Bacteroidota
</td>
</tr>
<tr>
<td style="text-align:left;">
Rikenellaceae
</td>
<td style="text-align:left;">
family
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.63
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Pseudomonadati
</td>
<td style="text-align:left;">
Bacteroidota
</td>
<td style="text-align:left;">
Bacteroidia
</td>
<td style="text-align:left;">
Bacteroidales
</td>
<td style="text-align:left;">
Rikenellaceae
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
k__Bacteria|k__Pseudomonadati|p__Bacteroidota|c__Bacteroidia|o__Bacteroidales|f__Rikenellaceae
</td>
</tr>
<tr>
<td style="text-align:left;">
Lachnospiraceae
</td>
<td style="text-align:left;">
family
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.00
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Lachnospirales
</td>
<td style="text-align:left;">
Lachnospiraceae
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Bacillota|c__Clostridia|o__Lachnospirales|f__Lachnospiraceae
</td>
</tr>
<tr>
<td style="text-align:left;">
Bacteroides
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1.00
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Pseudomonadati
</td>
<td style="text-align:left;">
Bacteroidota
</td>
<td style="text-align:left;">
Bacteroidia
</td>
<td style="text-align:left;">
Bacteroidales
</td>
<td style="text-align:left;">
Bacteroidaceae
</td>
<td style="text-align:left;">
Bacteroides
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
k__Bacteria|k__Pseudomonadati|p__Bacteroidota|c__Bacteroidia|o__Bacteroidales|f__Bacteroidaceae|g__Bacteroides
</td>
</tr>
<tr>
<td style="text-align:left;">
Deltaproteobacteria
</td>
<td style="text-align:left;">
class
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.50
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Deltaproteobacteria
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
k__Bacteria|c__Deltaproteobacteria
</td>
</tr>
<tr>
<td style="text-align:left;">
Bifidobacterium
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.50
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
Actinomycetes
</td>
<td style="text-align:left;">
Bifidobacteriales
</td>
<td style="text-align:left;">
Bifidobacteriaceae
</td>
<td style="text-align:left;">
Bifidobacterium
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Actinomycetota|c__Actinomycetes|o__Bifidobacteriales|f__Bifidobacteriaceae|g__Bifidobacterium
</td>
</tr>
<tr>
<td style="text-align:left;">
Atopobiaceae
</td>
<td style="text-align:left;">
family
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.00
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
Coriobacteriia
</td>
<td style="text-align:left;">
Coriobacteriales
</td>
<td style="text-align:left;">
Atopobiaceae
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Actinomycetota|c__Coriobacteriia|o__Coriobacteriales|f__Atopobiaceae
</td>
</tr>
</tbody>
</table>
</div>
<div class="section level2">
<h2 id="taxon-frequency-table-by-race-ethnicity">Taxon Frequency Table by Race-ethnicity<a class="anchor" aria-label="anchor" href="#taxon-frequency-table-by-race-ethnicity"></a>
</h2>
<div class="sourceCode" id="cb11"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">dat.sub_ethnic</span> <span class="op">&lt;-</span> <span class="va">dat.sub</span> <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html" class="external-link">%&gt;%</a></span> <span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">dat.sub</span><span class="op">$</span><span class="va">Condition</span> <span class="op">==</span> <span class="st">"ethnic group"</span><span class="op">)</span> </span>
<span><span class="fu"><a href="https://rdrr.io/r/base/dim.html" class="external-link">dim</a></span><span class="op">(</span><span class="va">dat.sub_ethnic</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## [1] 31 50</span></span></code></pre>
<div class="sourceCode" id="cb13"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu">bugSigSimple</span><span class="fu">::</span><span class="fu"><a href="../reference/createTaxonTable.html">createTaxonTable</a></span><span class="op">(</span><span class="va">dat.sub_ethnic</span><span class="op">)</span>  <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html" class="external-link">%&gt;%</a></span></span>
<span>  <span class="fu"><a href="https://rdrr.io/pkg/kableExtra/man/kbl.html" class="external-link">kbl</a></span><span class="op">(</span><span class="op">)</span> <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html" class="external-link">%&gt;%</a></span></span>
<span>  <span class="fu"><a href="https://rdrr.io/pkg/kableExtra/man/kable_styling.html" class="external-link">kable_styling</a></span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
<table class="table table" style="margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;">
Taxon Name
</th>
<th style="text-align:left;">
Taxonomic Level
</th>
<th style="text-align:right;">
total_signatures
</th>
<th style="text-align:right;">
increased_signatures
</th>
<th style="text-align:right;">
decreased_signatures
</th>
<th style="text-align:right;">
Binomial Test pval
</th>
<th style="text-align:left;">
kingdom
</th>
<th style="text-align:left;">
phylum
</th>
<th style="text-align:left;">
class
</th>
<th style="text-align:left;">
order
</th>
<th style="text-align:left;">
family
</th>
<th style="text-align:left;">
genus
</th>
<th style="text-align:left;">
species
</th>
<th style="text-align:right;">
n_signatures
</th>
<th style="text-align:left;">
metaphlan_name
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
Eubacteriales
</td>
<td style="text-align:left;">
order
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
0.450
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Eubacteriales
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Bacillota|c__Clostridia|o__Eubacteriales
</td>
</tr>
<tr>
<td style="text-align:left;">
Coriobacteriaceae
</td>
<td style="text-align:left;">
family
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
0.690
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
Coriobacteriia
</td>
<td style="text-align:left;">
Coriobacteriales
</td>
<td style="text-align:left;">
Coriobacteriaceae
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Actinomycetota|c__Coriobacteriia|o__Coriobacteriales|f__Coriobacteriaceae
</td>
</tr>
<tr>
<td style="text-align:left;">
Christensenellaceae
</td>
<td style="text-align:left;">
family
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
0.220
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Christensenellales
</td>
<td style="text-align:left;">
Christensenellaceae
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Bacillota|c__Clostridia|o__Christensenellales|f__Christensenellaceae
</td>
</tr>
<tr>
<td style="text-align:left;">
Peptococcaceae
</td>
<td style="text-align:left;">
family
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1.000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Eubacteriales
</td>
<td style="text-align:left;">
Peptococcaceae
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Bacillota|c__Clostridia|o__Eubacteriales|f__Peptococcaceae
</td>
</tr>
<tr>
<td style="text-align:left;">
Veillonella
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.031
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Negativicutes
</td>
<td style="text-align:left;">
Veillonellales
</td>
<td style="text-align:left;">
Veillonellaceae
</td>
<td style="text-align:left;">
Veillonella
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Bacillota|c__Negativicutes|o__Veillonellales|f__Veillonellaceae|g__Veillonella
</td>
</tr>
<tr>
<td style="text-align:left;">
Odoribacteraceae
</td>
<td style="text-align:left;">
family
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1.000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Pseudomonadati
</td>
<td style="text-align:left;">
Bacteroidota
</td>
<td style="text-align:left;">
Bacteroidia
</td>
<td style="text-align:left;">
Bacteroidales
</td>
<td style="text-align:left;">
Odoribacteraceae
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
k__Bacteria|k__Pseudomonadati|p__Bacteroidota|c__Bacteroidia|o__Bacteroidales|f__Odoribacteraceae
</td>
</tr>
<tr>
<td style="text-align:left;">
Odoribacter
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1.000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Pseudomonadati
</td>
<td style="text-align:left;">
Bacteroidota
</td>
<td style="text-align:left;">
Bacteroidia
</td>
<td style="text-align:left;">
Bacteroidales
</td>
<td style="text-align:left;">
Odoribacteraceae
</td>
<td style="text-align:left;">
Odoribacter
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
k__Bacteria|k__Pseudomonadati|p__Bacteroidota|c__Bacteroidia|o__Bacteroidales|f__Odoribacteraceae|g__Odoribacter
</td>
</tr>
<tr>
<td style="text-align:left;">
Rikenellaceae
</td>
<td style="text-align:left;">
family
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
0.690
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Pseudomonadati
</td>
<td style="text-align:left;">
Bacteroidota
</td>
<td style="text-align:left;">
Bacteroidia
</td>
<td style="text-align:left;">
Bacteroidales
</td>
<td style="text-align:left;">
Rikenellaceae
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
k__Bacteria|k__Pseudomonadati|p__Bacteroidota|c__Bacteroidia|o__Bacteroidales|f__Rikenellaceae
</td>
</tr>
<tr>
<td style="text-align:left;">
Victivallaceae
</td>
<td style="text-align:left;">
family
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
0.690
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Pseudomonadati
</td>
<td style="text-align:left;">
Lentisphaerota
</td>
<td style="text-align:left;">
Lentisphaeria
</td>
<td style="text-align:left;">
Victivallales
</td>
<td style="text-align:left;">
Victivallaceae
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
k__Bacteria|k__Pseudomonadati|p__Lentisphaerota|c__Lentisphaeria|o__Victivallales|f__Victivallaceae
</td>
</tr>
<tr>
<td style="text-align:left;">
Verrucomicrobiaceae
</td>
<td style="text-align:left;">
family
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
0.031
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Pseudomonadati
</td>
<td style="text-align:left;">
Verrucomicrobiota
</td>
<td style="text-align:left;">
Verrucomicrobiia
</td>
<td style="text-align:left;">
Verrucomicrobiales
</td>
<td style="text-align:left;">
Verrucomicrobiaceae
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
k__Bacteria|k__Pseudomonadati|p__Verrucomicrobiota|c__Verrucomicrobiia|o__Verrucomicrobiales|f__Verrucomicrobiaceae
</td>
</tr>
</tbody>
</table>
</div>
<div class="section level2">
<h2 id="taxon-frequency-table-by-all-studies-ses-and-ethnicity-combined-">Taxon Frequency Table by all studies (SES and Ethnicity)
combined.<a class="anchor" aria-label="anchor" href="#taxon-frequency-table-by-all-studies-ses-and-ethnicity-combined-"></a>
</h2>
<div class="sourceCode" id="cb14"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu">bugSigSimple</span><span class="fu">::</span><span class="fu"><a href="../reference/createTaxonTable.html">createTaxonTable</a></span><span class="op">(</span><span class="va">dat.sub</span>, n <span class="op">=</span> <span class="fl">50</span><span class="op">)</span> <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html" class="external-link">%&gt;%</a></span></span>
<span>  <span class="fu"><a href="https://rdrr.io/pkg/kableExtra/man/kbl.html" class="external-link">kbl</a></span><span class="op">(</span><span class="op">)</span> <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html" class="external-link">%&gt;%</a></span></span>
<span>  <span class="fu"><a href="https://rdrr.io/pkg/kableExtra/man/kable_styling.html" class="external-link">kable_styling</a></span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## Warning: Expected 7 pieces. Additional pieces discarded in 11 rows [19, 22, 23, 24, 28,</span></span>
<span><span class="co">## 29, 34, 35, 36, 37, 44].</span></span></code></pre>
<table class="table table" style="margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;">
Taxon Name
</th>
<th style="text-align:left;">
Taxonomic Level
</th>
<th style="text-align:right;">
total_signatures
</th>
<th style="text-align:right;">
increased_signatures
</th>
<th style="text-align:right;">
decreased_signatures
</th>
<th style="text-align:right;">
Binomial Test pval
</th>
<th style="text-align:left;">
kingdom
</th>
<th style="text-align:left;">
phylum
</th>
<th style="text-align:left;">
class
</th>
<th style="text-align:left;">
order
</th>
<th style="text-align:left;">
family
</th>
<th style="text-align:left;">
genus
</th>
<th style="text-align:left;">
species
</th>
<th style="text-align:right;">
n_signatures
</th>
<th style="text-align:left;">
metaphlan_name
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
Prevotella
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
12
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
0.770
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Pseudomonadati
</td>
<td style="text-align:left;">
Bacteroidota
</td>
<td style="text-align:left;">
Bacteroidia
</td>
<td style="text-align:left;">
Bacteroidales
</td>
<td style="text-align:left;">
Prevotellaceae
</td>
<td style="text-align:left;">
Prevotella
</td>
<td style="text-align:right;">
13
</td>
<td style="text-align:left;">
k__Bacteria|k__Pseudomonadati|p__Bacteroidota|c__Bacteroidia|o__Bacteroidales|f__Prevotellaceae|g__Prevotella
</td>
</tr>
<tr>
<td style="text-align:left;">
Eubacteriales
</td>
<td style="text-align:left;">
order
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
0.750
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Eubacteriales
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
32
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Bacillota|c__Clostridia|o__Eubacteriales
</td>
</tr>
<tr>
<td style="text-align:left;">
Rikenellaceae
</td>
<td style="text-align:left;">
family
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
1.000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Pseudomonadati
</td>
<td style="text-align:left;">
Bacteroidota
</td>
<td style="text-align:left;">
Bacteroidia
</td>
<td style="text-align:left;">
Bacteroidales
</td>
<td style="text-align:left;">
Rikenellaceae
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
12
</td>
<td style="text-align:left;">
k__Bacteria|k__Pseudomonadati|p__Bacteroidota|c__Bacteroidia|o__Bacteroidales|f__Rikenellaceae
</td>
</tr>
<tr>
<td style="text-align:left;">
Bacteroides
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
0.510
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Pseudomonadati
</td>
<td style="text-align:left;">
Bacteroidota
</td>
<td style="text-align:left;">
Bacteroidia
</td>
<td style="text-align:left;">
Bacteroidales
</td>
<td style="text-align:left;">
Bacteroidaceae
</td>
<td style="text-align:left;">
Bacteroides
</td>
<td style="text-align:right;">
14
</td>
<td style="text-align:left;">
k__Bacteria|k__Pseudomonadati|p__Bacteroidota|c__Bacteroidia|o__Bacteroidales|f__Bacteroidaceae|g__Bacteroides
</td>
</tr>
<tr>
<td style="text-align:left;">
Coriobacteriaceae
</td>
<td style="text-align:left;">
family
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
0.450
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
Coriobacteriia
</td>
<td style="text-align:left;">
Coriobacteriales
</td>
<td style="text-align:left;">
Coriobacteriaceae
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Actinomycetota|c__Coriobacteriia|o__Coriobacteriales|f__Coriobacteriaceae
</td>
</tr>
<tr>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
phylum
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
1.000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
47
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Bacillota
</td>
</tr>
<tr>
<td style="text-align:left;">
Ruminococcus
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1.000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Eubacteriales
</td>
<td style="text-align:left;">
Oscillospiraceae
</td>
<td style="text-align:left;">
Ruminococcus
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Bacillota|c__Clostridia|o__Eubacteriales|f__Oscillospiraceae|g__Ruminococcus
</td>
</tr>
<tr>
<td style="text-align:left;">
Veillonella
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.016
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Negativicutes
</td>
<td style="text-align:left;">
Veillonellales
</td>
<td style="text-align:left;">
Veillonellaceae
</td>
<td style="text-align:left;">
Veillonella
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Bacillota|c__Negativicutes|o__Veillonellales|f__Veillonellaceae|g__Veillonella
</td>
</tr>
<tr>
<td style="text-align:left;">
Odoribacter
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1.000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Pseudomonadati
</td>
<td style="text-align:left;">
Bacteroidota
</td>
<td style="text-align:left;">
Bacteroidia
</td>
<td style="text-align:left;">
Bacteroidales
</td>
<td style="text-align:left;">
Odoribacteraceae
</td>
<td style="text-align:left;">
Odoribacter
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:left;">
k__Bacteria|k__Pseudomonadati|p__Bacteroidota|c__Bacteroidia|o__Bacteroidales|f__Odoribacteraceae|g__Odoribacter
</td>
</tr>
<tr>
<td style="text-align:left;">
Christensenellaceae
</td>
<td style="text-align:left;">
family
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
0.220
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Christensenellales
</td>
<td style="text-align:left;">
Christensenellaceae
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Bacillota|c__Clostridia|o__Christensenellales|f__Christensenellaceae
</td>
</tr>
<tr>
<td style="text-align:left;">
Clostridium
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
0.690
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Eubacteriales
</td>
<td style="text-align:left;">
Clostridiaceae
</td>
<td style="text-align:left;">
Clostridium
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Bacillota|c__Clostridia|o__Eubacteriales|f__Clostridiaceae|g__Clostridium
</td>
</tr>
<tr>
<td style="text-align:left;">
Eubacterium
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
0.690
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Eubacteriales
</td>
<td style="text-align:left;">
Eubacteriaceae
</td>
<td style="text-align:left;">
Eubacterium
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Bacillota|c__Clostridia|o__Eubacteriales|f__Eubacteriaceae|g__Eubacterium
</td>
</tr>
<tr>
<td style="text-align:left;">
Peptococcaceae
</td>
<td style="text-align:left;">
family
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1.000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Eubacteriales
</td>
<td style="text-align:left;">
Peptococcaceae
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Bacillota|c__Clostridia|o__Eubacteriales|f__Peptococcaceae
</td>
</tr>
<tr>
<td style="text-align:left;">
Odoribacteraceae
</td>
<td style="text-align:left;">
family
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1.000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Pseudomonadati
</td>
<td style="text-align:left;">
Bacteroidota
</td>
<td style="text-align:left;">
Bacteroidia
</td>
<td style="text-align:left;">
Bacteroidales
</td>
<td style="text-align:left;">
Odoribacteraceae
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:left;">
k__Bacteria|k__Pseudomonadati|p__Bacteroidota|c__Bacteroidia|o__Bacteroidales|f__Odoribacteraceae
</td>
</tr>
<tr>
<td style="text-align:left;">
Victivallaceae
</td>
<td style="text-align:left;">
family
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
0.690
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Pseudomonadati
</td>
<td style="text-align:left;">
Lentisphaerota
</td>
<td style="text-align:left;">
Lentisphaeria
</td>
<td style="text-align:left;">
Victivallales
</td>
<td style="text-align:left;">
Victivallaceae
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
k__Bacteria|k__Pseudomonadati|p__Lentisphaerota|c__Lentisphaeria|o__Victivallales|f__Victivallaceae
</td>
</tr>
<tr>
<td style="text-align:left;">
Verrucomicrobiaceae
</td>
<td style="text-align:left;">
family
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
0.031
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Pseudomonadati
</td>
<td style="text-align:left;">
Verrucomicrobiota
</td>
<td style="text-align:left;">
Verrucomicrobiia
</td>
<td style="text-align:left;">
Verrucomicrobiales
</td>
<td style="text-align:left;">
Verrucomicrobiaceae
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
k__Bacteria|k__Pseudomonadati|p__Verrucomicrobiota|c__Verrucomicrobiia|o__Verrucomicrobiales|f__Verrucomicrobiaceae
</td>
</tr>
<tr>
<td style="text-align:left;">
Bacteroidota
</td>
<td style="text-align:left;">
phylum
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.062
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Pseudomonadati
</td>
<td style="text-align:left;">
Bacteroidota
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
41
</td>
<td style="text-align:left;">
k__Bacteria|k__Pseudomonadati|p__Bacteroidota
</td>
</tr>
<tr>
<td style="text-align:left;">
Bifidobacterium
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.630
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
Actinomycetes
</td>
<td style="text-align:left;">
Bifidobacteriales
</td>
<td style="text-align:left;">
Bifidobacteriaceae
</td>
<td style="text-align:left;">
Bifidobacterium
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Actinomycetota|c__Actinomycetes|o__Bifidobacteriales|f__Bifidobacteriaceae|g__Bifidobacterium
</td>
</tr>
<tr>
<td style="text-align:left;">
Bifidobacterium longum
</td>
<td style="text-align:left;">
species
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.630
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
Actinomycetes
</td>
<td style="text-align:left;">
Bifidobacteriales
</td>
<td style="text-align:left;">
Bifidobacteriaceae
</td>
<td style="text-align:left;">
Bifidobacterium
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Actinomycetota|c__Actinomycetes|o__Bifidobacteriales|f__Bifidobacteriaceae|g__Bifidobacterium|s__Bifidobacterium
longum
</td>
</tr>
<tr>
<td style="text-align:left;">
Oscillospiraceae
</td>
<td style="text-align:left;">
family
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.120
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Eubacteriales
</td>
<td style="text-align:left;">
Oscillospiraceae
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
19
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Bacillota|c__Clostridia|o__Eubacteriales|f__Oscillospiraceae
</td>
</tr>
<tr>
<td style="text-align:left;">
Faecalibacterium
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.630
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Eubacteriales
</td>
<td style="text-align:left;">
Oscillospiraceae
</td>
<td style="text-align:left;">
Faecalibacterium
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Bacillota|c__Clostridia|o__Eubacteriales|f__Oscillospiraceae|g__Faecalibacterium
</td>
</tr>
<tr>
<td style="text-align:left;">
Faecalibacterium prausnitzii
</td>
<td style="text-align:left;">
species
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1.000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Eubacteriales
</td>
<td style="text-align:left;">
Oscillospiraceae
</td>
<td style="text-align:left;">
Faecalibacterium
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Bacillota|c__Clostridia|o__Eubacteriales|f__Oscillospiraceae|g__Faecalibacterium|s__Faecalibacterium
prausnitzii
</td>
</tr>
<tr>
<td style="text-align:left;">
Bifidobacterium adolescentis
</td>
<td style="text-align:left;">
species
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1.000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
Actinomycetes
</td>
<td style="text-align:left;">
Bifidobacteriales
</td>
<td style="text-align:left;">
Bifidobacteriaceae
</td>
<td style="text-align:left;">
Bifidobacterium
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Actinomycetota|c__Actinomycetes|o__Bifidobacteriales|f__Bifidobacteriaceae|g__Bifidobacterium|s__Bifidobacterium
adolescentis
</td>
</tr>
<tr>
<td style="text-align:left;">
Bifidobacterium bifidum
</td>
<td style="text-align:left;">
species
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
Actinomycetes
</td>
<td style="text-align:left;">
Bifidobacteriales
</td>
<td style="text-align:left;">
Bifidobacteriaceae
</td>
<td style="text-align:left;">
Bifidobacterium
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Actinomycetota|c__Actinomycetes|o__Bifidobacteriales|f__Bifidobacteriaceae|g__Bifidobacterium|s__Bifidobacterium
bifidum
</td>
</tr>
<tr>
<td style="text-align:left;">
Lactobacillus
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Bacilli
</td>
<td style="text-align:left;">
Lactobacillales
</td>
<td style="text-align:left;">
Lactobacillaceae
</td>
<td style="text-align:left;">
Lactobacillus
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Bacillota|c__Bacilli|o__Lactobacillales|f__Lactobacillaceae|g__Lactobacillus
</td>
</tr>
<tr>
<td style="text-align:left;">
Lachnospiraceae
</td>
<td style="text-align:left;">
family
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Lachnospirales
</td>
<td style="text-align:left;">
Lachnospiraceae
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
12
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Bacillota|c__Clostridia|o__Lachnospirales|f__Lachnospiraceae
</td>
</tr>
<tr>
<td style="text-align:left;">
Blautia
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Lachnospirales
</td>
<td style="text-align:left;">
Lachnospiraceae
</td>
<td style="text-align:left;">
Blautia
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Bacillota|c__Clostridia|o__Lachnospirales|f__Lachnospiraceae|g__Blautia
</td>
</tr>
<tr>
<td style="text-align:left;">
Bacteroides fragilis
</td>
<td style="text-align:left;">
species
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1.000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Pseudomonadati
</td>
<td style="text-align:left;">
Bacteroidota
</td>
<td style="text-align:left;">
Bacteroidia
</td>
<td style="text-align:left;">
Bacteroidales
</td>
<td style="text-align:left;">
Bacteroidaceae
</td>
<td style="text-align:left;">
Bacteroides
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
k__Bacteria|k__Pseudomonadati|p__Bacteroidota|c__Bacteroidia|o__Bacteroidales|f__Bacteroidaceae|g__Bacteroides|s__Bacteroides
fragilis
</td>
</tr>
<tr>
<td style="text-align:left;">
Bacteroides thetaiotaomicron
</td>
<td style="text-align:left;">
species
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
0.250
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Pseudomonadati
</td>
<td style="text-align:left;">
Bacteroidota
</td>
<td style="text-align:left;">
Bacteroidia
</td>
<td style="text-align:left;">
Bacteroidales
</td>
<td style="text-align:left;">
Bacteroidaceae
</td>
<td style="text-align:left;">
Bacteroides
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
k__Bacteria|k__Pseudomonadati|p__Bacteroidota|c__Bacteroidia|o__Bacteroidales|f__Bacteroidaceae|g__Bacteroides|s__Bacteroides
thetaiotaomicron
</td>
</tr>
<tr>
<td style="text-align:left;">
Parabacteroides
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.250
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Pseudomonadati
</td>
<td style="text-align:left;">
Bacteroidota
</td>
<td style="text-align:left;">
Bacteroidia
</td>
<td style="text-align:left;">
Bacteroidales
</td>
<td style="text-align:left;">
Tannerellaceae
</td>
<td style="text-align:left;">
Parabacteroides
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
k__Bacteria|k__Pseudomonadati|p__Bacteroidota|c__Bacteroidia|o__Bacteroidales|f__Tannerellaceae|g__Parabacteroides
</td>
</tr>
<tr>
<td style="text-align:left;">
Enterobacteriaceae
</td>
<td style="text-align:left;">
family
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1.000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Pseudomonadati
</td>
<td style="text-align:left;">
Pseudomonadota
</td>
<td style="text-align:left;">
Gammaproteobacteria
</td>
<td style="text-align:left;">
Enterobacterales
</td>
<td style="text-align:left;">
Enterobacteriaceae
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:left;">
k__Bacteria|k__Pseudomonadati|p__Pseudomonadota|c__Gammaproteobacteria|o__Enterobacterales|f__Enterobacteriaceae
</td>
</tr>
<tr>
<td style="text-align:left;">
Deltaproteobacteria
</td>
<td style="text-align:left;">
class
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.500
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Deltaproteobacteria
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
k__Bacteria|c__Deltaproteobacteria
</td>
</tr>
<tr>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
phylum
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.500
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
21
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Actinomycetota
</td>
</tr>
<tr>
<td style="text-align:left;">
Bifidobacterium breve
</td>
<td style="text-align:left;">
species
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
Actinomycetes
</td>
<td style="text-align:left;">
Bifidobacteriales
</td>
<td style="text-align:left;">
Bifidobacteriaceae
</td>
<td style="text-align:left;">
Bifidobacterium
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Actinomycetota|c__Actinomycetes|o__Bifidobacteriales|f__Bifidobacteriaceae|g__Bifidobacterium|s__Bifidobacterium
breve
</td>
</tr>
<tr>
<td style="text-align:left;">
Bifidobacterium catenulatum
</td>
<td style="text-align:left;">
species
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
Actinomycetes
</td>
<td style="text-align:left;">
Bifidobacteriales
</td>
<td style="text-align:left;">
Bifidobacteriaceae
</td>
<td style="text-align:left;">
Bifidobacterium
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Actinomycetota|c__Actinomycetes|o__Bifidobacteriales|f__Bifidobacteriaceae|g__Bifidobacterium|s__Bifidobacterium
catenulatum
</td>
</tr>
<tr>
<td style="text-align:left;">
Bifidobacterium pseudocatenulatum
</td>
<td style="text-align:left;">
species
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
Actinomycetes
</td>
<td style="text-align:left;">
Bifidobacteriales
</td>
<td style="text-align:left;">
Bifidobacteriaceae
</td>
<td style="text-align:left;">
Bifidobacterium
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Actinomycetota|c__Actinomycetes|o__Bifidobacteriales|f__Bifidobacteriaceae|g__Bifidobacterium|s__Bifidobacterium
pseudocatenulatum
</td>
</tr>
<tr>
<td style="text-align:left;">
Bifidobacterium ruminantium
</td>
<td style="text-align:left;">
species
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
Actinomycetes
</td>
<td style="text-align:left;">
Bifidobacteriales
</td>
<td style="text-align:left;">
Bifidobacteriaceae
</td>
<td style="text-align:left;">
Bifidobacterium
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Actinomycetota|c__Actinomycetes|o__Bifidobacteriales|f__Bifidobacteriaceae|g__Bifidobacterium|s__Bifidobacterium
ruminantium
</td>
</tr>
<tr>
<td style="text-align:left;">
Atopobiaceae
</td>
<td style="text-align:left;">
family
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
Coriobacteriia
</td>
<td style="text-align:left;">
Coriobacteriales
</td>
<td style="text-align:left;">
Atopobiaceae
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Actinomycetota|c__Coriobacteriia|o__Coriobacteriales|f__Atopobiaceae
</td>
</tr>
<tr>
<td style="text-align:left;">
Slackia
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.500
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
Coriobacteriia
</td>
<td style="text-align:left;">
Eggerthellales
</td>
<td style="text-align:left;">
Eggerthellaceae
</td>
<td style="text-align:left;">
Slackia
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Actinomycetota|c__Coriobacteriia|o__Eggerthellales|f__Eggerthellaceae|g__Slackia
</td>
</tr>
<tr>
<td style="text-align:left;">
Staphylococcus
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Bacilli
</td>
<td style="text-align:left;">
Bacillales
</td>
<td style="text-align:left;">
Staphylococcaceae
</td>
<td style="text-align:left;">
Staphylococcus
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Bacillota|c__Bacilli|o__Bacillales|f__Staphylococcaceae|g__Staphylococcus
</td>
</tr>
<tr>
<td style="text-align:left;">
Enterococcus
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.500
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Bacilli
</td>
<td style="text-align:left;">
Lactobacillales
</td>
<td style="text-align:left;">
Enterococcaceae
</td>
<td style="text-align:left;">
Enterococcus
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Bacillota|c__Bacilli|o__Lactobacillales|f__Enterococcaceae|g__Enterococcus
</td>
</tr>
<tr>
<td style="text-align:left;">
Lactococcus
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Bacilli
</td>
<td style="text-align:left;">
Lactobacillales
</td>
<td style="text-align:left;">
Streptococcaceae
</td>
<td style="text-align:left;">
Lactococcus
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Bacillota|c__Bacilli|o__Lactobacillales|f__Streptococcaceae|g__Lactococcus
</td>
</tr>
<tr>
<td style="text-align:left;">
Clostridiaceae
</td>
<td style="text-align:left;">
family
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
0.500
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Eubacteriales
</td>
<td style="text-align:left;">
Clostridiaceae
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
11
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Bacillota|c__Clostridia|o__Eubacteriales|f__Clostridiaceae
</td>
</tr>
<tr>
<td style="text-align:left;">
Clostridium perfringens
</td>
<td style="text-align:left;">
species
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
0.500
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Eubacteriales
</td>
<td style="text-align:left;">
Clostridiaceae
</td>
<td style="text-align:left;">
Clostridium
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Bacillota|c__Clostridia|o__Eubacteriales|f__Clostridiaceae|g__Clostridium|s__Clostridium
perfringens
</td>
</tr>
<tr>
<td style="text-align:left;">
Anaerotruncus
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Eubacteriales
</td>
<td style="text-align:left;">
Oscillospiraceae
</td>
<td style="text-align:left;">
Anaerotruncus
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Bacillota|c__Clostridia|o__Eubacteriales|f__Oscillospiraceae|g__Anaerotruncus
</td>
</tr>
<tr>
<td style="text-align:left;">
Oscillospira
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Eubacteriales
</td>
<td style="text-align:left;">
Oscillospiraceae
</td>
<td style="text-align:left;">
Oscillospira
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Bacillota|c__Clostridia|o__Eubacteriales|f__Oscillospiraceae|g__Oscillospira
</td>
</tr>
<tr>
<td style="text-align:left;">
butyrate-producing bacterium M21/2
</td>
<td style="text-align:left;">
species
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Eubacteriales
</td>
<td style="text-align:left;">
butyrate-producing bacterium M21/2
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Bacillota|c__Clostridia|o__Eubacteriales|s__butyrate-producing
bacterium M21/2
</td>
</tr>
<tr>
<td style="text-align:left;">
Clostridiales bacterium
</td>
<td style="text-align:left;">
species
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Eubacteriales
</td>
<td style="text-align:left;">
Clostridiales bacterium
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Bacillota|c__Clostridia|o__Eubacteriales|s__Clostridiales
bacterium
</td>
</tr>
<tr>
<td style="text-align:left;">
Anaerostipes
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.500
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Lachnospirales
</td>
<td style="text-align:left;">
Lachnospiraceae
</td>
<td style="text-align:left;">
Anaerostipes
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Bacillota|c__Clostridia|o__Lachnospirales|f__Lachnospiraceae|g__Anaerostipes
</td>
</tr>
<tr>
<td style="text-align:left;">
Butyrivibrio
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.500
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Lachnospirales
</td>
<td style="text-align:left;">
Lachnospiraceae
</td>
<td style="text-align:left;">
Butyrivibrio
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
k__Bacteria|k__Bacillati|p__Bacillota|c__Clostridia|o__Lachnospirales|f__Lachnospiraceae|g__Butyrivibrio
</td>
</tr>
</tbody>
</table>
</div>
<div class="section level2">
<h2 id="cluster-analysis-for-signatures-of-increased-abundance-and-of-decreased-abundance-in-low-ses-calculated-using-jaccard-distance-to-create-a-distance-matrix-">Cluster Analysis for signatures of increased abundance and of
decreased abundance in low SES calculated using Jaccard Distance to
create a distance matrix.<a class="anchor" aria-label="anchor" href="#cluster-analysis-for-signatures-of-increased-abundance-and-of-decreased-abundance-in-low-ses-calculated-using-jaccard-distance-to-create-a-distance-matrix-"></a>
</h2>
<div class="sourceCode" id="cb16"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">allsigs</span> <span class="op">&lt;-</span> <span class="fu">bugsigdbr</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/bugsigdbr/man/getSignatures.html" class="external-link">getSignatures</a></span><span class="op">(</span><span class="va">dat.sub_SES</span> , tax.id.type <span class="op">=</span> <span class="st">"taxname"</span><span class="op">)</span></span>
<span><span class="va">allsigs</span> <span class="op">&lt;-</span> <span class="va">allsigs</span><span class="op">[</span><span class="fu"><a href="https://rdrr.io/r/base/lapply.html" class="external-link">sapply</a></span><span class="op">(</span><span class="va">allsigs</span>, <span class="va">length</span><span class="op">)</span> <span class="op">&gt;</span> <span class="fl">1</span><span class="op">]</span> <span class="co">#require length &gt; 1</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/length.html" class="external-link">length</a></span><span class="op">(</span><span class="va">allsigs</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## [1] 19</span></span></code></pre>
<div class="sourceCode" id="cb18"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">mydists</span> <span class="op">&lt;-</span> <span class="fu">BugSigDBStats</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/BugSigDBStats/man/calcPairwiseOverlaps.html" class="external-link">calcPairwiseOverlaps</a></span><span class="op">(</span><span class="va">allsigs</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/dim.html" class="external-link">dim</a></span><span class="op">(</span><span class="va">mydists</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## [1] 38  8</span></span></code></pre>
</div>
<div class="section level2">
<h2 id="visualize-the-distribution-of-the-signature-lengths">Visualize the distribution of the signature lengths<a class="anchor" aria-label="anchor" href="#visualize-the-distribution-of-the-signature-lengths"></a>
</h2>
<div class="sourceCode" id="cb20"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://ggplot2.tidyverse.org" class="external-link">ggplot2</a></span><span class="op">)</span></span>
<span><span class="va">siglengths</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/lapply.html" class="external-link">sapply</a></span><span class="op">(</span><span class="va">allsigs</span>, <span class="va">length</span><span class="op">)</span></span>
<span><span class="va">siglengths.df</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/data.frame.html" class="external-link">data.frame</a></span><span class="op">(</span>siglengths <span class="op">=</span> <span class="va">siglengths</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/ggplot.html" class="external-link">ggplot</a></span><span class="op">(</span><span class="va">siglengths.df</span>, <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/aes.html" class="external-link">aes</a></span><span class="op">(</span>x<span class="op">=</span><span class="va">siglengths</span><span class="op">)</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_bar.html" class="external-link">geom_bar</a></span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
<p><img src="capstoneanalysis_ifeanyi_files/figure-html/unnamed-chunk-10-1.png" width="700"></p>
<div class="sourceCode" id="cb21"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/base/table.html" class="external-link">table</a></span><span class="op">(</span><span class="va">siglengths</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## siglengths</span></span>
<span><span class="co">##  2  3  4  5  6  7  8 10 15 </span></span>
<span><span class="co">##  1  5  1  3  2  2  3  1  1</span></span></code></pre>
</div>
<div class="section level2">
<h2 id="create-a-matrix-of-jaccard-similarities-0-for-no-overlap-1-for-100-overlap">Create a matrix of Jaccard similarities (0 for no overlap, 1 for
100% overlap)<a class="anchor" aria-label="anchor" href="#create-a-matrix-of-jaccard-similarities-0-for-no-overlap-1-for-100-overlap"></a>
</h2>
<div class="sourceCode" id="cb23"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">jmat</span> <span class="op">&lt;-</span> <span class="fu">BugSigDBStats</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/BugSigDBStats/man/calcJaccardSimilarity.html" class="external-link">calcJaccardSimilarity</a></span><span class="op">(</span><span class="va">allsigs</span><span class="op">)</span></span></code></pre></div>
<p>##Create a Clustered heatmap</p>
<div class="sourceCode" id="cb24"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/jokergoo/ComplexHeatmap" class="external-link">ComplexHeatmap</a></span><span class="op">)</span></span>
<span></span>
<span><span class="va">ha</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/ComplexHeatmap/man/HeatmapAnnotation.html" class="external-link">HeatmapAnnotation</a></span><span class="op">(</span>`Signature Length` <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/pkg/ComplexHeatmap/man/anno_barplot.html" class="external-link">anno_barplot</a></span><span class="op">(</span><span class="va">siglengths</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="va">hr</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/ComplexHeatmap/man/rowAnnotation.html" class="external-link">rowAnnotation</a></span><span class="op">(</span></span>
<span>  `Signature Length` <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/pkg/ComplexHeatmap/man/anno_barplot.html" class="external-link">anno_barplot</a></span><span class="op">(</span><span class="va">siglengths</span><span class="op">)</span></span>
<span>  <span class="op">)</span></span>
<span><span class="va">hm</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/ComplexHeatmap/man/Heatmap.html" class="external-link">Heatmap</a></span><span class="op">(</span></span>
<span>  <span class="va">jmat</span>,</span>
<span>  top_annotation <span class="op">=</span> <span class="va">ha</span>,</span>
<span><span class="co">#  left_annotation = hr,</span></span>
<span><span class="co">#  column_names_max_height = unit(23, "cm"),</span></span>
<span>  column_names_rot <span class="op">=</span> <span class="fl">45</span>,</span>
<span><span class="co">#  row_names_max_width = unit(15, "cm"),</span></span>
<span><span class="co">#  get rid of study labels</span></span>
<span>  row_labels <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/grep.html" class="external-link">sub</a></span><span class="op">(</span><span class="st">"bsdb:"</span>, <span class="st">""</span>, <span class="fu"><a href="https://rdrr.io/r/base/grep.html" class="external-link">sub</a></span><span class="op">(</span><span class="st">"_.+"</span>, <span class="st">""</span>, <span class="fu"><a href="https://rdrr.io/r/base/colnames.html" class="external-link">rownames</a></span><span class="op">(</span><span class="va">jmat</span><span class="op">)</span><span class="op">)</span>, fixed <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span>,  </span>
<span>  column_labels <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/grep.html" class="external-link">sub</a></span><span class="op">(</span><span class="st">"bsdb:"</span>, <span class="st">""</span>, <span class="fu"><a href="https://rdrr.io/r/base/grep.html" class="external-link">sub</a></span><span class="op">(</span><span class="st">"_.+"</span>, <span class="st">""</span>, <span class="fu"><a href="https://rdrr.io/r/base/colnames.html" class="external-link">colnames</a></span><span class="op">(</span><span class="va">jmat</span><span class="op">)</span><span class="op">)</span>, fixed <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="va">hm</span></span></code></pre></div>
<p><img src="capstoneanalysis_ifeanyi_files/figure-html/unnamed-chunk-12-1.png" width="816"></p>
</div>
<div class="section level2">
<h2 id="create-a-wide-format-dataframe-for-regression-analysis">Create a wide format dataframe for Regression Analysis<a class="anchor" aria-label="anchor" href="#create-a-wide-format-dataframe-for-regression-analysis"></a>
</h2>
<div class="sourceCode" id="cb25"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">dat_withsigs</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/stats/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">dat.sub_SES</span> , <span class="op">!</span><span class="fu"><a href="https://rdrr.io/r/base/NA.html" class="external-link">is.na</a></span><span class="op">(</span><span class="va">dat.sub_SES</span><span class="op">$</span><span class="va">`NCBI Taxonomy IDs`</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="va">sigs</span> <span class="op">&lt;-</span> <span class="fu">bugsigdbr</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/bugsigdbr/man/getSignatures.html" class="external-link">getSignatures</a></span><span class="op">(</span><span class="va">dat_withsigs</span>, tax.id.type <span class="op">=</span> <span class="st">"taxname"</span><span class="op">)</span></span>
<span><span class="va">cmat</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/t.html" class="external-link">t</a></span><span class="op">(</span><span class="fu">safe</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/safe/man/getCmatrix.html" class="external-link">getCmatrix</a></span><span class="op">(</span><span class="va">sigs</span>, as.matrix <span class="op">=</span> <span class="cn">TRUE</span>, min.size <span class="op">=</span> <span class="fl">0</span>, prune <span class="op">=</span> <span class="cn">FALSE</span><span class="op">)</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## WARNING: rows are sorted elements of keyword.list</span></span>
<span><span class="co">## 25 categories formed</span></span></code></pre>
<div class="sourceCode" id="cb27"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">cdf</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/data.frame.html" class="external-link">data.frame</a></span><span class="op">(</span><span class="va">cmat</span>, stringsAsFactors <span class="op">=</span> <span class="cn">FALSE</span>, check.names <span class="op">=</span> <span class="cn">FALSE</span><span class="op">)</span></span>
<span><span class="va">cdf</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/cbind.html" class="external-link">cbind</a></span><span class="op">(</span><span class="va">dat_withsigs</span>, <span class="va">cdf</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/dim.html" class="external-link">dim</a></span><span class="op">(</span><span class="va">cdf</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## [1]  25 129</span></span></code></pre>
<p>An arbitrary example of meta-regression:</p>
<div class="sourceCode" id="cb29"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">fit</span> <span class="op">&lt;-</span></span>
<span>  <span class="fu"><a href="https://rdrr.io/r/stats/glm.html" class="external-link">glm</a></span><span class="op">(</span></span>
<span>    <span class="va">Prevotella</span> <span class="op">~</span> <span class="va">`Location of subjects`</span> <span class="op">+</span> <span class="va">`Sequencing type`</span> <span class="op">+</span> <span class="va">`Abundance in Group 1`</span>,</span>
<span>    family <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/stats/family.html" class="external-link">binomial</a></span><span class="op">(</span>link <span class="op">=</span> <span class="st">"logit"</span><span class="op">)</span>,</span>
<span>    data <span class="op">=</span> <span class="va">cdf</span></span>
<span>  <span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/summary.html" class="external-link">summary</a></span><span class="op">(</span><span class="va">fit</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## </span></span>
<span><span class="co">## Call:</span></span>
<span><span class="co">## glm(formula = Prevotella ~ `Location of subjects` + `Sequencing type` + </span></span>
<span><span class="co">##     `Abundance in Group 1`, family = binomial(link = "logit"), </span></span>
<span><span class="co">##     data = cdf)</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## Coefficients:</span></span>
<span><span class="co">##                                                  Estimate Std. Error z value</span></span>
<span><span class="co">## (Intercept)                                    -1.977e+01  1.145e+04  -0.002</span></span>
<span><span class="co">## `Location of subjects`China                     2.103e+01  1.145e+04   0.002</span></span>
<span><span class="co">## `Location of subjects`Finland,Malawi            1.953e+01  1.983e+04   0.001</span></span>
<span><span class="co">## `Location of subjects`Indonesia                 2.103e+01  1.145e+04   0.002</span></span>
<span><span class="co">## `Location of subjects`Israel                    2.103e+01  1.145e+04   0.002</span></span>
<span><span class="co">## `Location of subjects`Malaysia                  1.219e-09  1.402e+04   0.000</span></span>
<span><span class="co">## `Location of subjects`Mexico                    2.103e+01  1.145e+04   0.002</span></span>
<span><span class="co">## `Location of subjects`Papua New Guinea          1.729e+00  2.111e+04   0.000</span></span>
<span><span class="co">## `Location of subjects`United Kingdom            1.224e-09  1.619e+04   0.000</span></span>
<span><span class="co">## `Location of subjects`United States of America  1.953e+01  1.145e+04   0.002</span></span>
<span><span class="co">## `Sequencing type`WMS                           -1.953e+01  1.145e+04  -0.002</span></span>
<span><span class="co">## `Abundance in Group 1`increased                -2.527e+00  1.517e+00  -1.666</span></span>
<span><span class="co">##                                                Pr(&gt;|z|)  </span></span>
<span><span class="co">## (Intercept)                                      0.9986  </span></span>
<span><span class="co">## `Location of subjects`China                      0.9985  </span></span>
<span><span class="co">## `Location of subjects`Finland,Malawi             0.9992  </span></span>
<span><span class="co">## `Location of subjects`Indonesia                  0.9985  </span></span>
<span><span class="co">## `Location of subjects`Israel                     0.9985  </span></span>
<span><span class="co">## `Location of subjects`Malaysia                   1.0000  </span></span>
<span><span class="co">## `Location of subjects`Mexico                     0.9985  </span></span>
<span><span class="co">## `Location of subjects`Papua New Guinea           0.9999  </span></span>
<span><span class="co">## `Location of subjects`United Kingdom             1.0000  </span></span>
<span><span class="co">## `Location of subjects`United States of America   0.9986  </span></span>
<span><span class="co">## `Sequencing type`WMS                             0.9986  </span></span>
<span><span class="co">## `Abundance in Group 1`increased                  0.0958 .</span></span>
<span><span class="co">## ---</span></span>
<span><span class="co">## Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## (Dispersion parameter for binomial family taken to be 1)</span></span>
<span><span class="co">## </span></span>
<span><span class="co">##     Null deviance: 25.020  on 24  degrees of freedom</span></span>
<span><span class="co">## Residual deviance: 12.082  on 13  degrees of freedom</span></span>
<span><span class="co">## AIC: 36.082</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## Number of Fisher Scoring iterations: 19</span></span></code></pre>
</div>
<div class="section level2">
<h2 id="create-another-heatmap-on-correlations-of-presenceabsence-of-taxa-">Create another heatmap on correlations of presence/absence of
taxa.<a class="anchor" aria-label="anchor" href="#create-another-heatmap-on-correlations-of-presenceabsence-of-taxa-"></a>
</h2>
<div class="sourceCode" id="cb31"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">sigcors</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/stats/cor.html" class="external-link">cor</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/t.html" class="external-link">t</a></span><span class="op">(</span><span class="va">cmat</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="va">siglengths</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/lapply.html" class="external-link">sapply</a></span><span class="op">(</span><span class="va">sigs</span>, <span class="va">length</span><span class="op">)</span></span>
<span><span class="va">ha</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/ComplexHeatmap/man/HeatmapAnnotation.html" class="external-link">HeatmapAnnotation</a></span><span class="op">(</span>`Signature Length` <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/pkg/ComplexHeatmap/man/anno_barplot.html" class="external-link">anno_barplot</a></span><span class="op">(</span><span class="va">siglengths</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="va">hr</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/ComplexHeatmap/man/rowAnnotation.html" class="external-link">rowAnnotation</a></span><span class="op">(</span>`Signature Length` <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/pkg/ComplexHeatmap/man/anno_barplot.html" class="external-link">anno_barplot</a></span><span class="op">(</span><span class="va">siglengths</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="va">hm</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/ComplexHeatmap/man/Heatmap.html" class="external-link">Heatmap</a></span><span class="op">(</span></span>
<span>  <span class="va">sigcors</span>,</span>
<span>  top_annotation <span class="op">=</span> <span class="va">ha</span>, left_annotation <span class="op">=</span> <span class="va">hr</span>,</span>
<span>  row_names_max_width <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/grid/unit.html" class="external-link">unit</a></span><span class="op">(</span><span class="fl">.05</span>, <span class="st">"cm"</span><span class="op">)</span>,</span>
<span>  column_names_max_height <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/grid/unit.html" class="external-link">unit</a></span><span class="op">(</span><span class="fl">.1</span>, <span class="st">"cm"</span><span class="op">)</span>,</span>
<span> <span class="co"># row_labels = sub(".+:", "", rownames(sigcors)), ##removing study just to make signature names legible</span></span>
<span>  column_labels <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/grep.html" class="external-link">sub</a></span><span class="op">(</span><span class="st">".+:"</span>, <span class="st">""</span>, <span class="fu"><a href="https://rdrr.io/r/base/colnames.html" class="external-link">colnames</a></span><span class="op">(</span><span class="va">sigcors</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="op">)</span></span>
<span><span class="va">hm</span></span></code></pre></div>
<p><img src="capstoneanalysis_ifeanyi_files/figure-html/unnamed-chunk-15-1.png" width="700"></p>
</div>
<div class="section level2">
<h2 id="cluster-analysis-for-signatures-of-increased-abundance-and-of-decreased-abundance-in-non-whitecaucasions-calculated-using-jaccard-distance-to-create-a-distance-matrix-">Cluster Analysis for signatures of increased abundance and of
decreased abundance in non-white/caucasions calculated using Jaccard
Distance to create a distance matrix.<a class="anchor" aria-label="anchor" href="#cluster-analysis-for-signatures-of-increased-abundance-and-of-decreased-abundance-in-non-whitecaucasions-calculated-using-jaccard-distance-to-create-a-distance-matrix-"></a>
</h2>
<div class="sourceCode" id="cb32"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">allsige</span> <span class="op">&lt;-</span> <span class="fu">bugsigdbr</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/bugsigdbr/man/getSignatures.html" class="external-link">getSignatures</a></span><span class="op">(</span><span class="va">dat.sub_ethnic</span> , tax.id.type <span class="op">=</span> <span class="st">"taxname"</span><span class="op">)</span></span>
<span><span class="va">allsige</span> <span class="op">&lt;-</span> <span class="va">allsige</span><span class="op">[</span><span class="fu"><a href="https://rdrr.io/r/base/lapply.html" class="external-link">sapply</a></span><span class="op">(</span><span class="va">allsige</span>, <span class="va">length</span><span class="op">)</span> <span class="op">&gt;</span> <span class="fl">1</span><span class="op">]</span> <span class="co">#require length &gt; 1</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/length.html" class="external-link">length</a></span><span class="op">(</span><span class="va">allsige</span> <span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## [1] 15</span></span></code></pre>
<div class="sourceCode" id="cb34"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">mydistse</span> <span class="op">&lt;-</span> <span class="fu">BugSigDBStats</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/BugSigDBStats/man/calcPairwiseOverlaps.html" class="external-link">calcPairwiseOverlaps</a></span><span class="op">(</span><span class="va">allsige</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/dim.html" class="external-link">dim</a></span><span class="op">(</span><span class="va">mydistse</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## [1] 52  8</span></span></code></pre>
</div>
<div class="section level2">
<h2 id="visualize-the-distribution-of-the-signature-lengths-1">Visualize the distribution of the signature lengths<a class="anchor" aria-label="anchor" href="#visualize-the-distribution-of-the-signature-lengths-1"></a>
</h2>
<div class="sourceCode" id="cb36"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://ggplot2.tidyverse.org" class="external-link">ggplot2</a></span><span class="op">)</span></span>
<span><span class="va">siglengthse</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/lapply.html" class="external-link">sapply</a></span><span class="op">(</span><span class="va">allsige</span>, <span class="va">length</span><span class="op">)</span></span>
<span><span class="va">siglengthse.df</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/data.frame.html" class="external-link">data.frame</a></span><span class="op">(</span>siglengthse <span class="op">=</span> <span class="va">siglengthse</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/ggplot.html" class="external-link">ggplot</a></span><span class="op">(</span><span class="va">siglengthse.df</span>, <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/aes.html" class="external-link">aes</a></span><span class="op">(</span>x<span class="op">=</span><span class="va">siglengthse</span><span class="op">)</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_bar.html" class="external-link">geom_bar</a></span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
<p><img src="capstoneanalysis_ifeanyi_files/figure-html/unnamed-chunk-17-1.png" width="700"></p>
<div class="sourceCode" id="cb37"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/base/table.html" class="external-link">table</a></span><span class="op">(</span><span class="va">siglengthse</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## siglengthse</span></span>
<span><span class="co">##  2  4  5  6  8  9 11 </span></span>
<span><span class="co">##  4  1  4  2  2  1  1</span></span></code></pre>
</div>
<div class="section level2">
<h2 id="create-a-matrix-of-jaccard-similarities-0-for-no-overlap-1-for-100-overlap-1">Create a matrix of Jaccard similarities (0 for no overlap, 1 for
100% overlap)<a class="anchor" aria-label="anchor" href="#create-a-matrix-of-jaccard-similarities-0-for-no-overlap-1-for-100-overlap-1"></a>
</h2>
<div class="sourceCode" id="cb39"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">jmate</span> <span class="op">&lt;-</span> <span class="fu">BugSigDBStats</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/BugSigDBStats/man/calcJaccardSimilarity.html" class="external-link">calcJaccardSimilarity</a></span><span class="op">(</span><span class="va">allsige</span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level2">
<h2 id="create-a-clustered-heatmap">Create a Clustered heatmap<a class="anchor" aria-label="anchor" href="#create-a-clustered-heatmap"></a>
</h2>
<div class="sourceCode" id="cb40"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/jokergoo/ComplexHeatmap" class="external-link">ComplexHeatmap</a></span><span class="op">)</span></span>
<span></span>
<span><span class="va">hae</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/ComplexHeatmap/man/HeatmapAnnotation.html" class="external-link">HeatmapAnnotation</a></span><span class="op">(</span>`Signature Length` <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/pkg/ComplexHeatmap/man/anno_barplot.html" class="external-link">anno_barplot</a></span><span class="op">(</span><span class="va">siglengthse</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="va">hre</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/ComplexHeatmap/man/rowAnnotation.html" class="external-link">rowAnnotation</a></span><span class="op">(</span>`Signature Length` <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/pkg/ComplexHeatmap/man/anno_barplot.html" class="external-link">anno_barplot</a></span><span class="op">(</span><span class="va">siglengthse</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="va">hme</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/ComplexHeatmap/man/Heatmap.html" class="external-link">Heatmap</a></span><span class="op">(</span></span>
<span>  <span class="va">jmate</span>,</span>
<span>  top_annotation <span class="op">=</span> <span class="va">hae</span>, left_annotation <span class="op">=</span> <span class="va">hre</span>,</span>
<span>  row_names_max_width <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/grid/unit.html" class="external-link">unit</a></span><span class="op">(</span><span class="fl">.10</span>,<span class="st">"cm"</span><span class="op">)</span>,</span>
<span>  column_names_max_height <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/grid/unit.html" class="external-link">unit</a></span><span class="op">(</span><span class="fl">1.0</span>, <span class="st">"cm"</span><span class="op">)</span>,</span>
<span><span class="co">#  row_labels = sub(".+:", "", rownames(jmate)),  #get rid of study labels</span></span>
<span>  column_labels <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/grep.html" class="external-link">sub</a></span><span class="op">(</span><span class="st">".+:"</span>, <span class="st">""</span>, <span class="fu"><a href="https://rdrr.io/r/base/colnames.html" class="external-link">colnames</a></span><span class="op">(</span><span class="va">jmate</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="va">hme</span></span></code></pre></div>
<p><img src="capstoneanalysis_ifeanyi_files/figure-html/unnamed-chunk-19-1.png" width="700"></p>
</div>
<div class="section level2">
<h2 id="create-a-wide-format-dataframe">Create a wide format dataframe<a class="anchor" aria-label="anchor" href="#create-a-wide-format-dataframe"></a>
</h2>
<div class="sourceCode" id="cb41"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">dat_withsigse</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/stats/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">dat.sub_ethnic</span> , <span class="op">!</span><span class="fu"><a href="https://rdrr.io/r/base/NA.html" class="external-link">is.na</a></span><span class="op">(</span><span class="va">dat.sub_ethnic</span><span class="op">$</span><span class="va">`NCBI Taxonomy IDs`</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="va">sigse</span> <span class="op">&lt;-</span> <span class="fu">bugsigdbr</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/bugsigdbr/man/getSignatures.html" class="external-link">getSignatures</a></span><span class="op">(</span><span class="va">dat_withsigse</span>, tax.id.type <span class="op">=</span> <span class="st">"taxname"</span><span class="op">)</span></span>
<span><span class="va">cmate</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/t.html" class="external-link">t</a></span><span class="op">(</span><span class="fu">safe</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/safe/man/getCmatrix.html" class="external-link">getCmatrix</a></span><span class="op">(</span><span class="va">sigse</span>, as.matrix <span class="op">=</span> <span class="cn">TRUE</span>, min.size <span class="op">=</span> <span class="fl">0</span>, prune <span class="op">=</span> <span class="cn">FALSE</span><span class="op">)</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## WARNING: rows are sorted elements of keyword.list</span></span>
<span><span class="co">## 31 categories formed</span></span></code></pre>
<div class="sourceCode" id="cb43"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">cdfe</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/data.frame.html" class="external-link">data.frame</a></span><span class="op">(</span><span class="va">cmate</span>, stringsAsFactors <span class="op">=</span> <span class="cn">FALSE</span>, check.names <span class="op">=</span> <span class="cn">FALSE</span><span class="op">)</span></span>
<span><span class="va">cdfe</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/cbind.html" class="external-link">cbind</a></span><span class="op">(</span><span class="va">dat_withsigse</span>, <span class="va">cdfe</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/colnames.html" class="external-link">colnames</a></span><span class="op">(</span><span class="va">cdfe</span><span class="op">)</span><span class="op">[</span><span class="fl">1</span><span class="op">:</span><span class="fl">25</span><span class="op">]</span></span></code></pre></div>
<pre><code><span><span class="co">##  [1] "BSDB ID"               "Study"                 "Study design"         </span></span>
<span><span class="co">##  [4] "PMID"                  "DOI"                   "URL"                  </span></span>
<span><span class="co">##  [7] "Authors list"          "Title"                 "Journal"              </span></span>
<span><span class="co">## [10] "Year"                  "Keywords"              "Experiment"           </span></span>
<span><span class="co">## [13] "Location of subjects"  "Host species"          "Body site"            </span></span>
<span><span class="co">## [16] "UBERON ID"             "Condition"             "EFO ID"               </span></span>
<span><span class="co">## [19] "Group 0 name"          "Group 1 name"          "Group 1 definition"   </span></span>
<span><span class="co">## [22] "Group 0 sample size"   "Group 1 sample size"   "Antibiotics exclusion"</span></span>
<span><span class="co">## [25] "Sequencing type"</span></span></code></pre>
</div>
<div class="section level2">
<h2 id="create-another-heatmap-on-correlations-of-presenceabsence-of-taxa--1">Create another heatmap on correlations of presence/absence of
taxa.<a class="anchor" aria-label="anchor" href="#create-another-heatmap-on-correlations-of-presenceabsence-of-taxa--1"></a>
</h2>
<div class="sourceCode" id="cb45"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">sigcorse</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/stats/cor.html" class="external-link">cor</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/t.html" class="external-link">t</a></span><span class="op">(</span><span class="va">cmate</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="va">siglengthse</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/lapply.html" class="external-link">sapply</a></span><span class="op">(</span><span class="va">sigse</span>, <span class="va">length</span><span class="op">)</span></span>
<span><span class="va">hae</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/ComplexHeatmap/man/HeatmapAnnotation.html" class="external-link">HeatmapAnnotation</a></span><span class="op">(</span>`Signature Length` <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/pkg/ComplexHeatmap/man/anno_barplot.html" class="external-link">anno_barplot</a></span><span class="op">(</span><span class="va">siglengthse</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="va">hre</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/ComplexHeatmap/man/rowAnnotation.html" class="external-link">rowAnnotation</a></span><span class="op">(</span>`Signature Length` <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/pkg/ComplexHeatmap/man/anno_barplot.html" class="external-link">anno_barplot</a></span><span class="op">(</span><span class="va">siglengthse</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="va">hme</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/ComplexHeatmap/man/Heatmap.html" class="external-link">Heatmap</a></span><span class="op">(</span></span>
<span>  <span class="va">sigcorse</span>,</span>
<span>  top_annotation <span class="op">=</span> <span class="va">hae</span>, left_annotation <span class="op">=</span> <span class="va">hre</span>,</span>
<span>  row_names_max_width <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/grid/unit.html" class="external-link">unit</a></span><span class="op">(</span><span class="fl">.05</span>, <span class="st">"cm"</span><span class="op">)</span>,</span>
<span>  column_names_max_height <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/grid/unit.html" class="external-link">unit</a></span><span class="op">(</span><span class="fl">.1</span>, <span class="st">"cm"</span><span class="op">)</span>,</span>
<span> <span class="co"># row_labels = sub(".+:", "", rownames(sigcors)), ##removing study just to make signature names legible</span></span>
<span>  column_labels <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/grep.html" class="external-link">sub</a></span><span class="op">(</span><span class="st">".+:"</span>, <span class="st">""</span>, <span class="fu"><a href="https://rdrr.io/r/base/colnames.html" class="external-link">colnames</a></span><span class="op">(</span><span class="va">sigcorse</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="op">)</span></span>
<span><span class="va">hme</span></span></code></pre></div>
<p><img src="capstoneanalysis_ifeanyi_files/figure-html/unnamed-chunk-21-1.png" width="700"></p>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

      </div>

</div>



      <footer><div class="copyright">
  <p></p>
<p>Developed by Ludwig Geistlinger, Chloe Mirzayi, Fatima Zohra, Levi Waldron.</p>
</div>

<div class="pkgdown">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.1.1.</p>
</div>

      </footer>
</div>






  </body>
</html>
