<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>COVID-19 associated microbiome • Reviews of published signatures based on bugsigdb.org</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.7.1/jquery.min.js" integrity="sha512-v2CJ7UaYy4JwqLDIrZUI/4hqeoQieOmAZNXBeQyjo21dadnwR+8ZaIJVT8EE2iyI61OV8e6M8PP2/4hpQINQ/g==" crossorigin="anonymous" referrerpolicy="no-referrer"></script><!-- Bootstrap --><link href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.4.0/flatly/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="COVID-19 associated microbiome">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">


    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">Reviews of published signatures based on bugsigdb.org</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">0.99.5</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Articles

    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/Bearkland%20capstone%20DED%2012.2.2024.html">Differential Abundance in the ocular microbiome in DED</a>
    </li>
    <li>
      <a href="../articles/c-section_meconium_shaimaa.html">Delivery Mode and the Meconium Microbiome</a>
    </li>
    <li>
      <a href="../articles/capstoneanalysis_clare.html">COVID-19 associated microbiome</a>
    </li>
    <li>
      <a href="../articles/capstoneanalysis_fatima.html">An analysis of Most frequent taxa in Major Depression and Bipolar Disorder</a>
    </li>
    <li>
      <a href="../articles/capstoneanalysis_ifeanyi.html">Distinguishing socioeconomic and racial/ethnic variation of the human gut microbiome by re-analysis of studies from around the world</a>
    </li>
    <li>
      <a href="../articles/capstoneanalysis_kweku.html">The Irritable Bowel Syndrome-associated Microbiome</a>
    </li>
    <li>
      <a href="../articles/fieldworkanalysis_samara.html">The Endometriosis-associated Microbiome</a>
    </li>
    <li>
      <a href="../articles/healthysig.html">Analysis of prevalent stool genera in bugsigdb.org</a>
    </li>
  </ul>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/waldronlab/bugSigSimple" class="external-link">
    <span class="fa fa-github"></span>

  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->



      </header><script src="capstoneanalysis_clare_files/kePrint-0.0.1/kePrint.js"></script><link href="capstoneanalysis_clare_files/lightable-0.0.1/lightable.css" rel="stylesheet">
<div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>COVID-19 associated microbiome</h1>
                        <h4 data-toc-skip class="author">Clare
Grieve</h4>
            
            <h4 data-toc-skip class="date">30 January, 2025</h4>
      

      <div class="hidden name"><code>capstoneanalysis_clare.rmd</code></div>

    </div>

    
    
<div class="section level2">
<h2 id="analysis-objectives">Analysis objectives<a class="anchor" aria-label="anchor" href="#analysis-objectives"></a>
</h2>
<ol style="list-style-type: decimal">
<li>Import, recode, and subset data from bugsigdb.org<br>
</li>
<li>Create a table of studies</li>
<li>Create a clustered heatmap showing similarity of signatures from
independent studies</li>
<li>Calculate the frequency of appearance of each taxa in independent
signatures, and identify the most frequently reported taxa</li>
<li>Estimate the probability of the most frequently identified taxa
occuring so frequently by chance</li>
</ol>
</div>
<div class="section level2">
<h2 id="packages-installation">Packages installation<a class="anchor" aria-label="anchor" href="#packages-installation"></a>
</h2>
<p>Install packages (not evaluated in vignette)</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/install.packages.html" class="external-link">install.packages</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"devtools"</span>, <span class="st">"tidyverse"</span>, <span class="st">"kableExtra"</span>, <span class="st">"gt"</span>, <span class="st">"glue"</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="fu">devtools</span><span class="fu">::</span><span class="fu"><a href="https://remotes.r-lib.org/reference/install_github.html" class="external-link">install_github</a></span><span class="op">(</span><span class="st">"waldronlab/bugSigSimple"</span><span class="op">)</span></span>
<span><span class="fu">devtools</span><span class="fu">::</span><span class="fu"><a href="https://remotes.r-lib.org/reference/install_github.html" class="external-link">install_github</a></span><span class="op">(</span><span class="st">"waldronlab/BugSigDBStats"</span><span class="op">)</span></span>
<span><span class="fu">devtools</span><span class="fu">::</span><span class="fu"><a href="https://remotes.r-lib.org/reference/install_github.html" class="external-link">install_github</a></span><span class="op">(</span><span class="st">"waldronlab/bugsigdbr"</span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level2">
<h2 id="data-import-recoding-and-subset">Data import, recoding, and subset<a class="anchor" aria-label="anchor" href="#data-import-recoding-and-subset"></a>
</h2>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://waldronlab.io/bugSigSimple" class="external-link">bugSigSimple</a></span><span class="op">)</span></span>
<span><span class="va">dat</span> <span class="op">&lt;-</span> <span class="fu">bugsigdbr</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/bugsigdbr/man/importBugSigDB.html" class="external-link">importBugSigDB</a></span><span class="op">(</span>cache <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span> </span></code></pre></div>
<pre><code><span><span class="co">## Using cached version from 2025-01-30 21:57:36</span></span></code></pre>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/base/dim.html" class="external-link">dim</a></span><span class="op">(</span><span class="va">dat</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## [1] 5520   50</span></span></code></pre>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/base/names.html" class="external-link">names</a></span><span class="op">(</span><span class="va">dat</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">##  [1] "BSDB ID"                    "Study"                     </span></span>
<span><span class="co">##  [3] "Study design"               "PMID"                      </span></span>
<span><span class="co">##  [5] "DOI"                        "URL"                       </span></span>
<span><span class="co">##  [7] "Authors list"               "Title"                     </span></span>
<span><span class="co">##  [9] "Journal"                    "Year"                      </span></span>
<span><span class="co">## [11] "Keywords"                   "Experiment"                </span></span>
<span><span class="co">## [13] "Location of subjects"       "Host species"              </span></span>
<span><span class="co">## [15] "Body site"                  "UBERON ID"                 </span></span>
<span><span class="co">## [17] "Condition"                  "EFO ID"                    </span></span>
<span><span class="co">## [19] "Group 0 name"               "Group 1 name"              </span></span>
<span><span class="co">## [21] "Group 1 definition"         "Group 0 sample size"       </span></span>
<span><span class="co">## [23] "Group 1 sample size"        "Antibiotics exclusion"     </span></span>
<span><span class="co">## [25] "Sequencing type"            "16S variable region"       </span></span>
<span><span class="co">## [27] "Sequencing platform"        "Statistical test"          </span></span>
<span><span class="co">## [29] "Significance threshold"     "MHT correction"            </span></span>
<span><span class="co">## [31] "LDA Score above"            "Matched on"                </span></span>
<span><span class="co">## [33] "Confounders controlled for" "Pielou"                    </span></span>
<span><span class="co">## [35] "Shannon"                    "Chao1"                     </span></span>
<span><span class="co">## [37] "Simpson"                    "Inverse Simpson"           </span></span>
<span><span class="co">## [39] "Richness"                   "Signature page name"       </span></span>
<span><span class="co">## [41] "Source"                     "Curated date"              </span></span>
<span><span class="co">## [43] "Curator"                    "Revision editor"           </span></span>
<span><span class="co">## [45] "Description"                "Abundance in Group 1"      </span></span>
<span><span class="co">## [47] "MetaPhlAn taxon names"      "NCBI Taxonomy IDs"         </span></span>
<span><span class="co">## [49] "State"                      "Reviewer"</span></span></code></pre>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://tidyverse.tidyverse.org" class="external-link">tidyverse</a></span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## ── <span style="font-weight: bold;">Attaching core tidyverse packages</span> ──────────────────────── tidyverse 2.0.0 ──</span></span>
<span><span class="co">## <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">dplyr    </span> 1.1.4     <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">readr    </span> 2.1.5</span></span>
<span><span class="co">## <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">forcats  </span> 1.0.0     <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">stringr  </span> 1.5.1</span></span>
<span><span class="co">## <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">ggplot2  </span> 3.5.1     <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">tibble   </span> 3.2.1</span></span>
<span><span class="co">## <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">lubridate</span> 1.9.4     <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">tidyr    </span> 1.3.1</span></span>
<span><span class="co">## <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">purrr    </span> 1.0.2     </span></span>
<span><span class="co">## ── <span style="font-weight: bold;">Conflicts</span> ────────────────────────────────────────── tidyverse_conflicts() ──</span></span>
<span><span class="co">## <span style="color: #BB0000;">✖</span> <span style="color: #0000BB;">dplyr</span>::<span style="color: #00BB00;">filter()</span> masks <span style="color: #0000BB;">stats</span>::filter()</span></span>
<span><span class="co">## <span style="color: #BB0000;">✖</span> <span style="color: #0000BB;">dplyr</span>::<span style="color: #00BB00;">lag()</span>    masks <span style="color: #0000BB;">stats</span>::lag()</span></span>
<span><span class="co">## <span style="color: #00BBBB;">ℹ</span> Use the conflicted package (<span style="color: #0000BB; font-style: italic;">&lt;http://conflicted.r-lib.org/&gt;</span>) to force all conflicts to become errors</span></span></code></pre>
<div class="sourceCode" id="cb10"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span> <span class="va">efo</span> <span class="op">&lt;-</span> <span class="fu">bugsigdbr</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/bugsigdbr/man/getOntology.html" class="external-link">getOntology</a></span><span class="op">(</span><span class="st">"efo"</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## Loading required namespace: ontologyIndex</span></span>
<span><span class="co">## Using cached version from 2025-01-30 21:55:38</span></span></code></pre>
<div class="sourceCode" id="cb12"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">covid_all</span> <span class="op">&lt;-</span> <span class="fu">bugsigdbr</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/bugsigdbr/man/subsetByOntology.html" class="external-link">subsetByOntology</a></span><span class="op">(</span><span class="va">dat</span>, column <span class="op">=</span> <span class="st">"Condition"</span>, <span class="st">"COVID-19"</span>, <span class="va">efo</span><span class="op">)</span> <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html" class="external-link">%&gt;%</a></span></span>
<span>  <span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate.html" class="external-link">mutate</a></span><span class="op">(</span>studyexp <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/paste.html" class="external-link">paste</a></span><span class="op">(</span><span class="va">Study</span>, <span class="va">Experiment</span>, sep <span class="op">=</span> <span class="st">"_"</span><span class="op">)</span><span class="op">)</span> <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html" class="external-link">%&gt;%</a></span></span>
<span>  <span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate.html" class="external-link">mutate</a></span><span class="op">(</span></span>
<span>    site <span class="op">=</span> <span class="fu"><a href="https://dplyr.tidyverse.org/reference/recode.html" class="external-link">recode</a></span><span class="op">(</span><span class="va">`Body site`</span>,</span>
<span>      <span class="st">"Feces"</span> <span class="op">=</span> <span class="st">"Gut"</span>,</span>
<span>      <span class="st">"Rectum"</span> <span class="op">=</span> <span class="st">"Gut"</span>,</span>
<span>      <span class="st">"Nasopharynx"</span> <span class="op">=</span> <span class="st">"aURT"</span>,</span>
<span>      <span class="st">"Oropharynx"</span> <span class="op">=</span> <span class="st">"aURT"</span>,</span>
<span>      <span class="st">"Nasopharynx,Oropharynx"</span> <span class="op">=</span> <span class="st">"aURT"</span>,</span>
<span>      <span class="st">"Nasal cavity"</span> <span class="op">=</span> <span class="st">"aURT"</span>,</span>
<span>      <span class="st">"Surface of tongue"</span> <span class="op">=</span> <span class="st">"aURT"</span>,</span>
<span>      <span class="st">"Nasopharynx,Throat"</span> <span class="op">=</span> <span class="st">"aURT"</span>,</span>
<span>      <span class="st">"Throat"</span> <span class="op">=</span> <span class="st">"aURT"</span>,</span>
<span>      <span class="st">"Tongue"</span> <span class="op">=</span> <span class="st">"aURT"</span>,</span>
<span>      <span class="st">"Lung"</span> <span class="op">=</span> <span class="st">"LRT"</span>,</span>
<span>      <span class="st">"Sputum"</span> <span class="op">=</span> <span class="st">"LRT"</span></span>
<span>    <span class="op">)</span></span>
<span>  <span class="op">)</span> <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html" class="external-link">%&gt;%</a></span></span>
<span>  <span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate.html" class="external-link">mutate</a></span><span class="op">(</span>comparison1 <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/paste.html" class="external-link">paste</a></span><span class="op">(</span><span class="va">`Group 0 name`</span>, <span class="va">`Group 1 name`</span>, sep <span class="op">=</span> <span class="st">" vs "</span><span class="op">)</span><span class="op">)</span></span></code></pre></div>
<div class="section level3">
<h3 id="table-of-studies">Table of studies<a class="anchor" aria-label="anchor" href="#table-of-studies"></a>
</h3>
<div class="sourceCode" id="cb13"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu">bugSigSimple</span><span class="fu">::</span><span class="fu"><a href="../reference/createStudyTable.html">createStudyTable</a></span><span class="op">(</span><span class="va">covid_all</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## <span style="color: #949494;"># A tibble: 44 × 5</span></span></span>
<span><span class="co">##    Study          Condition Cases Controls `Study Design`                       </span></span>
<span><span class="co">##    <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>          <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>     <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span>    <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span> <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>                                </span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 1</span> Bellato 2023   COVID-19    156       36 cross-sectional observational, not c…</span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 2</span> Braun 2021     COVID-19     26       29 cross-sectional observational, not c…</span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 3</span> Cao 2021       COVID-19     13        8 case-control                         </span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 4</span> Chen 2021      COVID-19     30       30 time series / longitudinal observati…</span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 5</span> De 2020        COVID-19     18       22 cross-sectional observational, not c…</span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 6</span> Engen 2021     COVID-19      9       10 cross-sectional observational, not c…</span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 7</span> Gaibani 2021   COVID-19     24       24 case-control                         </span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 8</span> Galperine 2023 COVID-19     57       43 time series / longitudinal observati…</span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 9</span> Gao 2021       COVID-19     94       48 prospective cohort                   </span></span>
<span><span class="co">## <span style="color: #BCBCBC;">10</span> Gu 2020        COVID-19     30       30 cross-sectional observational, not c…</span></span>
<span><span class="co">## <span style="color: #949494;"># ℹ 34 more rows</span></span></span></code></pre>
</div>
<div class="section level3">
<h3 id="taxon-frequency-tables-by-body-site">Taxon frequency tables by body site<a class="anchor" aria-label="anchor" href="#taxon-frequency-tables-by-body-site"></a>
</h3>
<div class="sourceCode" id="cb15"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://dplyr.tidyverse.org" class="external-link">dplyr</a></span><span class="op">)</span></span>
<span><span class="va">gut_sigs</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">covid_all</span>, </span>
<span>                           <span class="va">site</span> <span class="op">==</span> <span class="st">"Gut"</span><span class="op">)</span> <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html" class="external-link">%&gt;%</a></span></span>
<span>    <span class="fu"><a href="https://tidyr.tidyverse.org/reference/drop_na.html" class="external-link">drop_na</a></span><span class="op">(</span><span class="va">Source</span><span class="op">)</span></span>
<span></span>
<span><span class="va">naso_sigs</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">covid_all</span>, </span>
<span>                           <span class="va">site</span> <span class="op">==</span> <span class="st">"aURT"</span><span class="op">)</span> <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html" class="external-link">%&gt;%</a></span></span>
<span>    <span class="fu"><a href="https://tidyr.tidyverse.org/reference/drop_na.html" class="external-link">drop_na</a></span><span class="op">(</span><span class="va">Source</span><span class="op">)</span></span>
<span>  </span>
<span><span class="va">resp_sigs</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">covid_all</span>, </span>
<span>                           <span class="va">site</span> <span class="op">==</span> <span class="st">"LRT"</span><span class="op">)</span> <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html" class="external-link">%&gt;%</a></span></span>
<span>  <span class="fu"><a href="https://tidyr.tidyverse.org/reference/drop_na.html" class="external-link">drop_na</a></span><span class="op">(</span><span class="va">Source</span><span class="op">)</span></span></code></pre></div>
<p>In this table, the Binomial Test p-value corresponds to the null
hypothesis</p>
<p>H0: the proportion of signatures in which the taxon is reported
increased or decreased, relative to the total number of signatures in
which it is reported, is equal to 0.5</p>
<div class="sourceCode" id="cb16"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu">kableExtra</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/kableExtra/man/kbl.html" class="external-link">kbl</a></span><span class="op">(</span><span class="fu">bugSigSimple</span><span class="fu">::</span><span class="fu"><a href="../reference/createTaxonTable.html">createTaxonTable</a></span><span class="op">(</span><span class="va">gut_sigs</span><span class="op">)</span><span class="op">)</span></span></code></pre></div>
<table class="table">
<thead><tr>
<th style="text-align:left;">
Taxon Name
</th>
<th style="text-align:left;">
Taxonomic Level
</th>
<th style="text-align:right;">
total_signatures
</th>
<th style="text-align:right;">
increased_signatures
</th>
<th style="text-align:right;">
decreased_signatures
</th>
<th style="text-align:right;">
Binomial Test pval
</th>
<th style="text-align:left;">
kingdom
</th>
<th style="text-align:left;">
phylum
</th>
<th style="text-align:left;">
class
</th>
<th style="text-align:left;">
order
</th>
<th style="text-align:left;">
family
</th>
<th style="text-align:left;">
genus
</th>
<th style="text-align:left;">
species
</th>
<th style="text-align:right;">
n_signatures
</th>
<th style="text-align:left;">
metaphlan_name
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
Faecalibacterium
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
15
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
14
</td>
<td style="text-align:right;">
0.00098
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Eubacteriales
</td>
<td style="text-align:left;">
Oscillospiraceae
</td>
<td style="text-align:left;">
Faecalibacterium
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
21
</td>
<td style="text-align:left;">
k__Bacteria|p__Bacillota|c__Clostridia|o__Eubacteriales|f__Oscillospiraceae|g__Faecalibacterium
</td>
</tr>
<tr>
<td style="text-align:left;">
Anaerostipes
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
15
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
11
</td>
<td style="text-align:right;">
0.12000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Lachnospirales
</td>
<td style="text-align:left;">
Lachnospiraceae
</td>
<td style="text-align:left;">
Anaerostipes
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
17
</td>
<td style="text-align:left;">
k__Bacteria|p__Bacillota|c__Clostridia|o__Lachnospirales|f__Lachnospiraceae|g__Anaerostipes
</td>
</tr>
<tr>
<td style="text-align:left;">
Streptococcus
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
12
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
0.03900
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Bacilli
</td>
<td style="text-align:left;">
Lactobacillales
</td>
<td style="text-align:left;">
Streptococcaceae
</td>
<td style="text-align:left;">
Streptococcus
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
18
</td>
<td style="text-align:left;">
k__Bacteria|p__Bacillota|c__Bacilli|o__Lactobacillales|f__Streptococcaceae|g__Streptococcus
</td>
</tr>
<tr>
<td style="text-align:left;">
Clostridium
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
12
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
1.00000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Eubacteriales
</td>
<td style="text-align:left;">
Clostridiaceae
</td>
<td style="text-align:left;">
Clostridium
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
18
</td>
<td style="text-align:left;">
k__Bacteria|p__Bacillota|c__Clostridia|o__Eubacteriales|f__Clostridiaceae|g__Clostridium
</td>
</tr>
<tr>
<td style="text-align:left;">
Roseburia
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
12
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
12
</td>
<td style="text-align:right;">
0.00049
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Lachnospirales
</td>
<td style="text-align:left;">
Lachnospiraceae
</td>
<td style="text-align:left;">
Roseburia
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
19
</td>
<td style="text-align:left;">
k__Bacteria|p__Bacillota|c__Clostridia|o__Lachnospirales|f__Lachnospiraceae|g__Roseburia
</td>
</tr>
<tr>
<td style="text-align:left;">
Enterococcus
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
11
</td>
<td style="text-align:right;">
11
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.00098
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Bacilli
</td>
<td style="text-align:left;">
Lactobacillales
</td>
<td style="text-align:left;">
Enterococcaceae
</td>
<td style="text-align:left;">
Enterococcus
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
18
</td>
<td style="text-align:left;">
k__Bacteria|p__Bacillota|c__Bacilli|o__Lactobacillales|f__Enterococcaceae|g__Enterococcus
</td>
</tr>
<tr>
<td style="text-align:left;">
Ruminococcus
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
0.34000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Eubacteriales
</td>
<td style="text-align:left;">
Oscillospiraceae
</td>
<td style="text-align:left;">
Ruminococcus
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
20
</td>
<td style="text-align:left;">
k__Bacteria|p__Bacillota|c__Clostridia|o__Eubacteriales|f__Oscillospiraceae|g__Ruminococcus
</td>
</tr>
<tr>
<td style="text-align:left;">
Lactobacillaceae
</td>
<td style="text-align:left;">
family
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
1.00000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Bacilli
</td>
<td style="text-align:left;">
Lactobacillales
</td>
<td style="text-align:left;">
Lactobacillaceae
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
19
</td>
<td style="text-align:left;">
k__Bacteria|p__Bacillota|c__Bacilli|o__Lactobacillales|f__Lactobacillaceae
</td>
</tr>
<tr>
<td style="text-align:left;">
Faecalibacterium prausnitzii
</td>
<td style="text-align:left;">
species
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
0.00390
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Eubacteriales
</td>
<td style="text-align:left;">
Oscillospiraceae
</td>
<td style="text-align:left;">
Faecalibacterium
</td>
<td style="text-align:left;">
Faecalibacterium prausnitzii
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:left;">
k__Bacteria|p__Bacillota|c__Clostridia|o__Eubacteriales|f__Oscillospiraceae|g__Faecalibacterium|s__Faecalibacterium
prausnitzii
</td>
</tr>
<tr>
<td style="text-align:left;">
Anaerobutyricum hallii
</td>
<td style="text-align:left;">
species
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
0.51000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Lachnospirales
</td>
<td style="text-align:left;">
Lachnospiraceae
</td>
<td style="text-align:left;">
Anaerobutyricum
</td>
<td style="text-align:left;">
Anaerobutyricum hallii
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:left;">
k__Bacteria|p__Bacillota|c__Clostridia|o__Lachnospirales|f__Lachnospiraceae|g__Anaerobutyricum|s__Anaerobutyricum
hallii
</td>
</tr>
</tbody>
</table>
<div class="sourceCode" id="cb17"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu">kableExtra</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/kableExtra/man/kbl.html" class="external-link">kbl</a></span><span class="op">(</span><span class="fu">bugSigSimple</span><span class="fu">::</span><span class="fu"><a href="../reference/createTaxonTable.html">createTaxonTable</a></span><span class="op">(</span><span class="va">naso_sigs</span><span class="op">)</span><span class="op">)</span></span></code></pre></div>
<table class="table">
<thead><tr>
<th style="text-align:left;">
Taxon Name
</th>
<th style="text-align:left;">
Taxonomic Level
</th>
<th style="text-align:right;">
total_signatures
</th>
<th style="text-align:right;">
increased_signatures
</th>
<th style="text-align:right;">
decreased_signatures
</th>
<th style="text-align:right;">
Binomial Test pval
</th>
<th style="text-align:left;">
kingdom
</th>
<th style="text-align:left;">
phylum
</th>
<th style="text-align:left;">
class
</th>
<th style="text-align:left;">
order
</th>
<th style="text-align:left;">
family
</th>
<th style="text-align:left;">
genus
</th>
<th style="text-align:left;">
species
</th>
<th style="text-align:right;">
n_signatures
</th>
<th style="text-align:left;">
metaphlan_name
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
Veillonella
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
15
</td>
<td style="text-align:right;">
12
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
0.035
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Negativicutes
</td>
<td style="text-align:left;">
Veillonellales
</td>
<td style="text-align:left;">
Veillonellaceae
</td>
<td style="text-align:left;">
Veillonella
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
17
</td>
<td style="text-align:left;">
k__Bacteria|p__Bacillota|c__Negativicutes|o__Veillonellales|f__Veillonellaceae|g__Veillonella
</td>
</tr>
<tr>
<td style="text-align:left;">
Prevotella
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
12
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
1.000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacteroidota
</td>
<td style="text-align:left;">
Bacteroidia
</td>
<td style="text-align:left;">
Bacteroidales
</td>
<td style="text-align:left;">
Prevotellaceae
</td>
<td style="text-align:left;">
Prevotella
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
22
</td>
<td style="text-align:left;">
k__Bacteria|p__Bacteroidota|c__Bacteroidia|o__Bacteroidales|f__Prevotellaceae|g__Prevotella
</td>
</tr>
<tr>
<td style="text-align:left;">
Haemophilus
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
11
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
0.065
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Pseudomonadota
</td>
<td style="text-align:left;">
Gammaproteobacteria
</td>
<td style="text-align:left;">
Pasteurellales
</td>
<td style="text-align:left;">
Pasteurellaceae
</td>
<td style="text-align:left;">
Haemophilus
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
13
</td>
<td style="text-align:left;">
k__Bacteria|p__Pseudomonadota|c__Gammaproteobacteria|o__Pasteurellales|f__Pasteurellaceae|g__Haemophilus
</td>
</tr>
<tr>
<td style="text-align:left;">
Streptococcus
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
0.750
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Bacilli
</td>
<td style="text-align:left;">
Lactobacillales
</td>
<td style="text-align:left;">
Streptococcaceae
</td>
<td style="text-align:left;">
Streptococcus
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
15
</td>
<td style="text-align:left;">
k__Bacteria|p__Bacillota|c__Bacilli|o__Lactobacillales|f__Streptococcaceae|g__Streptococcus
</td>
</tr>
<tr>
<td style="text-align:left;">
Oribacterium
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
0.110
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Lachnospirales
</td>
<td style="text-align:left;">
Lachnospiraceae
</td>
<td style="text-align:left;">
Oribacterium
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:left;">
k__Bacteria|p__Bacillota|c__Clostridia|o__Lachnospirales|f__Lachnospiraceae|g__Oribacterium
</td>
</tr>
<tr>
<td style="text-align:left;">
Rothia
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
0.510
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
Actinomycetes
</td>
<td style="text-align:left;">
Micrococcales
</td>
<td style="text-align:left;">
Micrococcaceae
</td>
<td style="text-align:left;">
Rothia
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
12
</td>
<td style="text-align:left;">
k__Bacteria|p__Actinomycetota|c__Actinomycetes|o__Micrococcales|f__Micrococcaceae|g__Rothia
</td>
</tr>
<tr>
<td style="text-align:left;">
Neisseria
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
0.039
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Pseudomonadota
</td>
<td style="text-align:left;">
Betaproteobacteria
</td>
<td style="text-align:left;">
Neisseriales
</td>
<td style="text-align:left;">
Neisseriaceae
</td>
<td style="text-align:left;">
Neisseria
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
13
</td>
<td style="text-align:left;">
k__Bacteria|p__Pseudomonadota|c__Betaproteobacteria|o__Neisseriales|f__Neisseriaceae|g__Neisseria
</td>
</tr>
<tr>
<td style="text-align:left;">
Filifactor
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
0.070
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Peptostreptococcales
</td>
<td style="text-align:left;">
Filifactoraceae
</td>
<td style="text-align:left;">
Filifactor
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:left;">
k__Bacteria|p__Bacillota|c__Clostridia|o__Peptostreptococcales|f__Filifactoraceae|g__Filifactor
</td>
</tr>
<tr>
<td style="text-align:left;">
Alloprevotella
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
1.000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacteroidota
</td>
<td style="text-align:left;">
Bacteroidia
</td>
<td style="text-align:left;">
Bacteroidales
</td>
<td style="text-align:left;">
Prevotellaceae
</td>
<td style="text-align:left;">
Alloprevotella
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:left;">
k__Bacteria|p__Bacteroidota|c__Bacteroidia|o__Bacteroidales|f__Prevotellaceae|g__Alloprevotella
</td>
</tr>
<tr>
<td style="text-align:left;">
Bifidobacterium
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
0.016
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
Actinomycetes
</td>
<td style="text-align:left;">
Bifidobacteriales
</td>
<td style="text-align:left;">
Bifidobacteriaceae
</td>
<td style="text-align:left;">
Bifidobacterium
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:left;">
k__Bacteria|p__Actinomycetota|c__Actinomycetes|o__Bifidobacteriales|f__Bifidobacteriaceae|g__Bifidobacterium
</td>
</tr>
</tbody>
</table>
<div class="sourceCode" id="cb18"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu">kableExtra</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/kableExtra/man/kbl.html" class="external-link">kbl</a></span><span class="op">(</span><span class="fu">bugSigSimple</span><span class="fu">::</span><span class="fu"><a href="../reference/createTaxonTable.html">createTaxonTable</a></span><span class="op">(</span><span class="va">resp_sigs</span><span class="op">)</span><span class="op">)</span></span></code></pre></div>
<table class="table">
<thead><tr>
<th style="text-align:left;">
Taxon Name
</th>
<th style="text-align:left;">
Taxonomic Level
</th>
<th style="text-align:right;">
total_signatures
</th>
<th style="text-align:right;">
increased_signatures
</th>
<th style="text-align:right;">
decreased_signatures
</th>
<th style="text-align:right;">
Binomial Test pval
</th>
<th style="text-align:left;">
kingdom
</th>
<th style="text-align:left;">
phylum
</th>
<th style="text-align:left;">
class
</th>
<th style="text-align:left;">
order
</th>
<th style="text-align:left;">
family
</th>
<th style="text-align:left;">
genus
</th>
<th style="text-align:left;">
species
</th>
<th style="text-align:right;">
n_signatures
</th>
<th style="text-align:left;">
metaphlan_name
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
Rothia mucilaginosa
</td>
<td style="text-align:left;">
species
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.0
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
Actinomycetes
</td>
<td style="text-align:left;">
Micrococcales
</td>
<td style="text-align:left;">
Micrococcaceae
</td>
<td style="text-align:left;">
Rothia
</td>
<td style="text-align:left;">
Rothia mucilaginosa
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
k__Bacteria|p__Actinomycetota|c__Actinomycetes|o__Micrococcales|f__Micrococcaceae|g__Rothia|s__Rothia
mucilaginosa
</td>
</tr>
<tr>
<td style="text-align:left;">
Streptococcus
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.0
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Bacilli
</td>
<td style="text-align:left;">
Lactobacillales
</td>
<td style="text-align:left;">
Streptococcaceae
</td>
<td style="text-align:left;">
Streptococcus
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
k__Bacteria|p__Bacillota|c__Bacilli|o__Lactobacillales|f__Streptococcaceae|g__Streptococcus
</td>
</tr>
<tr>
<td style="text-align:left;">
Lachnoclostridium phytofermentans
</td>
<td style="text-align:left;">
species
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.0
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Lachnospirales
</td>
<td style="text-align:left;">
Lachnospiraceae
</td>
<td style="text-align:left;">
Lachnoclostridium
</td>
<td style="text-align:left;">
Lachnoclostridium phytofermentans
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
k__Bacteria|p__Bacillota|c__Clostridia|o__Lachnospirales|f__Lachnospiraceae|g__Lachnoclostridium|s__Lachnoclostridium
phytofermentans
</td>
</tr>
<tr>
<td style="text-align:left;">
Porphyromonas
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
0.5
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacteroidota
</td>
<td style="text-align:left;">
Bacteroidia
</td>
<td style="text-align:left;">
Bacteroidales
</td>
<td style="text-align:left;">
Porphyromonadaceae
</td>
<td style="text-align:left;">
Porphyromonas
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
k__Bacteria|p__Bacteroidota|c__Bacteroidia|o__Bacteroidales|f__Porphyromonadaceae|g__Porphyromonas
</td>
</tr>
<tr>
<td style="text-align:left;">
Tannerella
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
0.5
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacteroidota
</td>
<td style="text-align:left;">
Bacteroidia
</td>
<td style="text-align:left;">
Bacteroidales
</td>
<td style="text-align:left;">
Tannerellaceae
</td>
<td style="text-align:left;">
Tannerella
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
k__Bacteria|p__Bacteroidota|c__Bacteroidia|o__Bacteroidales|f__Tannerellaceae|g__Tannerella
</td>
</tr>
<tr>
<td style="text-align:left;">
Capnocytophaga
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.0
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacteroidota
</td>
<td style="text-align:left;">
Flavobacteriia
</td>
<td style="text-align:left;">
Flavobacteriales
</td>
<td style="text-align:left;">
Flavobacteriaceae
</td>
<td style="text-align:left;">
Capnocytophaga
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
k__Bacteria|p__Bacteroidota|c__Flavobacteriia|o__Flavobacteriales|f__Flavobacteriaceae|g__Capnocytophaga
</td>
</tr>
<tr>
<td style="text-align:left;">
Neisseria
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
0.5
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Pseudomonadota
</td>
<td style="text-align:left;">
Betaproteobacteria
</td>
<td style="text-align:left;">
Neisseriales
</td>
<td style="text-align:left;">
Neisseriaceae
</td>
<td style="text-align:left;">
Neisseria
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
k__Bacteria|p__Pseudomonadota|c__Betaproteobacteria|o__Neisseriales|f__Neisseriaceae|g__Neisseria
</td>
</tr>
<tr>
<td style="text-align:left;">
Actinomyces
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.0
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
Actinomycetes
</td>
<td style="text-align:left;">
Actinomycetales
</td>
<td style="text-align:left;">
Actinomycetaceae
</td>
<td style="text-align:left;">
Actinomyces
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
k__Bacteria|p__Actinomycetota|c__Actinomycetes|o__Actinomycetales|f__Actinomycetaceae|g__Actinomyces
</td>
</tr>
<tr>
<td style="text-align:left;">
Actinomyces pacaensis
</td>
<td style="text-align:left;">
species
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
1.0
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
Actinomycetes
</td>
<td style="text-align:left;">
Actinomycetales
</td>
<td style="text-align:left;">
Actinomycetaceae
</td>
<td style="text-align:left;">
Actinomyces
</td>
<td style="text-align:left;">
Actinomyces pacaensis
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
k__Bacteria|p__Actinomycetota|c__Actinomycetes|o__Actinomycetales|f__Actinomycetaceae|g__Actinomyces|s__Actinomyces
pacaensis
</td>
</tr>
<tr>
<td style="text-align:left;">
Schaalia meyeri
</td>
<td style="text-align:left;">
species
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
1.0
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
Actinomycetes
</td>
<td style="text-align:left;">
Actinomycetales
</td>
<td style="text-align:left;">
Actinomycetaceae
</td>
<td style="text-align:left;">
Schaalia
</td>
<td style="text-align:left;">
Schaalia meyeri
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
k__Bacteria|p__Actinomycetota|c__Actinomycetes|o__Actinomycetales|f__Actinomycetaceae|g__Schaalia|s__Schaalia
meyeri
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="section level2">
<h2 id="gut-microbiota-analysis">gut microbiota analysis<a class="anchor" aria-label="anchor" href="#gut-microbiota-analysis"></a>
</h2>
<p>Look specifically at case-control comparisons</p>
<div class="sourceCode" id="cb19"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">healthy</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/grep.html" class="external-link">grepl</a></span><span class="op">(</span><span class="st">".*(healthy.*COVID|COVID.*healthy).*"</span>, <span class="va">gut_sigs</span><span class="op">$</span><span class="va">comparison1</span>, ignore.case <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span>
<span></span>
<span><span class="va">cc_gut_sigs1</span> <span class="op">&lt;-</span> <span class="va">gut_sigs</span> <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html" class="external-link">%&gt;%</a></span></span>
<span>      <span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">healthy</span> <span class="op">==</span> <span class="cn">TRUE</span><span class="op">)</span> </span></code></pre></div>
<div class="section level3">
<h3 id="overall-frequencies-of-taxa-increased-in-cases-for-casecontrol-feces-studies">Overall frequencies of taxa increased in cases for case/control
feces studies<a class="anchor" aria-label="anchor" href="#overall-frequencies-of-taxa-increased-in-cases-for-casecontrol-feces-studies"></a>
</h3>
<p>Identifying a taxon reported consistently in 8 out of 8 studies is
<em>much</em> more compelling that the FDR value here would suggest,
since this taxon also passed a significance threshold in every one of
those studies.</p>
<div class="sourceCode" id="cb20"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">cc_gut_sigs1_taxontable</span> <span class="op">&lt;-</span> <span class="fu">bugSigSimple</span><span class="fu">::</span><span class="fu"><a href="../reference/createTaxonTable.html">createTaxonTable</a></span><span class="op">(</span><span class="va">cc_gut_sigs1</span><span class="op">)</span> <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html" class="external-link">%&gt;%</a></span> </span>
<span>  <span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate.html" class="external-link">mutate</a></span><span class="op">(</span>FDR <span class="op">=</span>  <span class="fu"><a href="https://rdrr.io/r/stats/p.adjust.html" class="external-link">p.adjust</a></span><span class="op">(</span>p <span class="op">=</span> <span class="va">`Binomial Test pval`</span>, method<span class="op">=</span><span class="st">"fdr"</span><span class="op">)</span><span class="op">)</span> <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html" class="external-link">%&gt;%</a></span></span>
<span>  <span class="fu"><a href="https://dplyr.tidyverse.org/reference/relocate.html" class="external-link">relocate</a></span><span class="op">(</span><span class="va">FDR</span>, .after <span class="op">=</span> <span class="va">`Binomial Test pval`</span><span class="op">)</span></span>
<span><span class="fu">kableExtra</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/kableExtra/man/kbl.html" class="external-link">kbl</a></span><span class="op">(</span><span class="va">cc_gut_sigs1_taxontable</span><span class="op">)</span></span></code></pre></div>
<table class="table">
<thead><tr>
<th style="text-align:left;">
Taxon Name
</th>
<th style="text-align:left;">
Taxonomic Level
</th>
<th style="text-align:right;">
total_signatures
</th>
<th style="text-align:right;">
increased_signatures
</th>
<th style="text-align:right;">
decreased_signatures
</th>
<th style="text-align:right;">
Binomial Test pval
</th>
<th style="text-align:right;">
FDR
</th>
<th style="text-align:left;">
kingdom
</th>
<th style="text-align:left;">
phylum
</th>
<th style="text-align:left;">
class
</th>
<th style="text-align:left;">
order
</th>
<th style="text-align:left;">
family
</th>
<th style="text-align:left;">
genus
</th>
<th style="text-align:left;">
species
</th>
<th style="text-align:right;">
n_signatures
</th>
<th style="text-align:left;">
metaphlan_name
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
Faecalibacterium
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
0.0039
</td>
<td style="text-align:right;">
0.0156000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Eubacteriales
</td>
<td style="text-align:left;">
Oscillospiraceae
</td>
<td style="text-align:left;">
Faecalibacterium
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
14
</td>
<td style="text-align:left;">
k__Bacteria|p__Bacillota|c__Clostridia|o__Eubacteriales|f__Oscillospiraceae|g__Faecalibacterium
</td>
</tr>
<tr>
<td style="text-align:left;">
Anaerostipes
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
0.0390
</td>
<td style="text-align:right;">
0.0557143
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Lachnospirales
</td>
<td style="text-align:left;">
Lachnospiraceae
</td>
<td style="text-align:left;">
Anaerostipes
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:left;">
k__Bacteria|p__Bacillota|c__Clostridia|o__Lachnospirales|f__Lachnospiraceae|g__Anaerostipes
</td>
</tr>
<tr>
<td style="text-align:left;">
Roseburia
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
0.0039
</td>
<td style="text-align:right;">
0.0156000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Lachnospirales
</td>
<td style="text-align:left;">
Lachnospiraceae
</td>
<td style="text-align:left;">
Roseburia
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
11
</td>
<td style="text-align:left;">
k__Bacteria|p__Bacillota|c__Clostridia|o__Lachnospirales|f__Lachnospiraceae|g__Roseburia
</td>
</tr>
<tr>
<td style="text-align:left;">
Enterococcus
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.0078
</td>
<td style="text-align:right;">
0.0156000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Bacilli
</td>
<td style="text-align:left;">
Lactobacillales
</td>
<td style="text-align:left;">
Enterococcaceae
</td>
<td style="text-align:left;">
Enterococcus
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
11
</td>
<td style="text-align:left;">
k__Bacteria|p__Bacillota|c__Bacilli|o__Lactobacillales|f__Enterococcaceae|g__Enterococcus
</td>
</tr>
<tr>
<td style="text-align:left;">
Lactobacillaceae
</td>
<td style="text-align:left;">
family
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
0.7300
</td>
<td style="text-align:right;">
0.7300000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Bacilli
</td>
<td style="text-align:left;">
Lactobacillales
</td>
<td style="text-align:left;">
Lactobacillaceae
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
12
</td>
<td style="text-align:left;">
k__Bacteria|p__Bacillota|c__Bacilli|o__Lactobacillales|f__Lactobacillaceae
</td>
</tr>
<tr>
<td style="text-align:left;">
Streptococcus
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.0078
</td>
<td style="text-align:right;">
0.0156000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Bacilli
</td>
<td style="text-align:left;">
Lactobacillales
</td>
<td style="text-align:left;">
Streptococcaceae
</td>
<td style="text-align:left;">
Streptococcus
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:left;">
k__Bacteria|p__Bacillota|c__Bacilli|o__Lactobacillales|f__Streptococcaceae|g__Streptococcus
</td>
</tr>
<tr>
<td style="text-align:left;">
Clostridium
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
0.7300
</td>
<td style="text-align:right;">
0.7300000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Eubacteriales
</td>
<td style="text-align:left;">
Clostridiaceae
</td>
<td style="text-align:left;">
Clostridium
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:left;">
k__Bacteria|p__Bacillota|c__Clostridia|o__Eubacteriales|f__Clostridiaceae|g__Clostridium
</td>
</tr>
<tr>
<td style="text-align:left;">
Coprococcus
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
0.0078
</td>
<td style="text-align:right;">
0.0156000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Lachnospirales
</td>
<td style="text-align:left;">
Lachnospiraceae
</td>
<td style="text-align:left;">
Coprococcus
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
13
</td>
<td style="text-align:left;">
k__Bacteria|p__Bacillota|c__Clostridia|o__Lachnospirales|f__Lachnospiraceae|g__Coprococcus
</td>
</tr>
<tr>
<td style="text-align:left;">
Ruminococcus
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
0.1300
</td>
<td style="text-align:right;">
0.1625000
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Eubacteriales
</td>
<td style="text-align:left;">
Oscillospiraceae
</td>
<td style="text-align:left;">
Ruminococcus
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
14
</td>
<td style="text-align:left;">
k__Bacteria|p__Bacillota|c__Clostridia|o__Eubacteriales|f__Oscillospiraceae|g__Ruminococcus
</td>
</tr>
<tr>
<td style="text-align:left;">
Blautia obeum
</td>
<td style="text-align:left;">
species
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
0.0160
</td>
<td style="text-align:right;">
0.0266667
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Clostridia
</td>
<td style="text-align:left;">
Lachnospirales
</td>
<td style="text-align:left;">
Lachnospiraceae
</td>
<td style="text-align:left;">
Blautia
</td>
<td style="text-align:left;">
Blautia obeum
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:left;">
k__Bacteria|p__Bacillota|c__Clostridia|o__Lachnospirales|f__Lachnospiraceae|g__Blautia|s__Blautia
obeum
</td>
</tr>
</tbody>
</table>
</div>
<div class="section level3">
<h3 id="monte-carlo-simulation-for-increased-abundance-taxa">Monte-Carlo simulation for increased abundance taxa<a class="anchor" aria-label="anchor" href="#monte-carlo-simulation-for-increased-abundance-taxa"></a>
</h3>
<p>Just for the increased cc_gut_sigs1 for now. I am inclined to skip
this analysis in favor of the clustering and binomial test analysis.</p>
<div class="sourceCode" id="cb21"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://waldronlab.io/bugSigSimple" class="external-link">bugSigSimple</a></span><span class="op">)</span></span>
<span><span class="va">gut.sigs.increased</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">cc_gut_sigs1</span>, <span class="va">`Abundance in Group 1`</span> <span class="op">==</span> <span class="st">"increased"</span><span class="op">)</span> <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html" class="external-link">%&gt;%</a></span> </span>
<span>  <span class="fu">bugsigdbr</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/bugsigdbr/man/getSignatures.html" class="external-link">getSignatures</a></span><span class="op">(</span>tax.id.type <span class="op">=</span> <span class="st">"taxname"</span><span class="op">)</span></span>
<span> <span class="va">my.siglengths.inc</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/lapply.html" class="external-link">sapply</a></span><span class="op">(</span><span class="va">gut.sigs.increased</span>, <span class="va">length</span><span class="op">)</span></span>
<span></span>
<span><span class="fu"><a href="../reference/getCriticalN.html">getCriticalN</a></span><span class="op">(</span><span class="va">gut.sigs.increased</span>, <span class="va">my.siglengths.inc</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## 95% </span></span>
<span><span class="co">##  10</span></span></code></pre>
<div class="sourceCode" id="cb23"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># Compare to observed - enterococcus &amp; streptococcus are the only taxa that equal the critical limit</span></span>
<span><span class="fu"><a href="../reference/frequencySigs.html">frequencySigs</a></span><span class="op">(</span><span class="va">gut.sigs.increased</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">##            Enterococcus           Streptococcus      Enterobacteriaceae </span></span>
<span><span class="co">##                       8                       8                       5 </span></span>
<span><span class="co">## Enterocloster citroniae         Enterococcaceae        Lactobacillaceae </span></span>
<span><span class="co">##                       5                       5                       5 </span></span>
<span><span class="co">##           Lactobacillus                  Rothia          Actinomycetota </span></span>
<span><span class="co">##                       5                       5                       4 </span></span>
<span><span class="co">##               Atopobium </span></span>
<span><span class="co">##                       4</span></span></code></pre>
</div>
<div class="section level3">
<h3 id="overall-frequencies-of-taxa-decreased-in-cases-for-casecontrol-feces-studies">Overall frequencies of taxa decreased in cases for case/control
feces studies<a class="anchor" aria-label="anchor" href="#overall-frequencies-of-taxa-decreased-in-cases-for-casecontrol-feces-studies"></a>
</h3>
<div class="sourceCode" id="cb25"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="../reference/createTaxonTable.html">createTaxonTable</a></span><span class="op">(</span><span class="va">cc_gut_sigs1</span>, n<span class="op">=</span><span class="fl">40</span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level3">
<h3 id="nasopharyngeal-microbiota-analysis">nasopharyngeal microbiota analysis<a class="anchor" aria-label="anchor" href="#nasopharyngeal-microbiota-analysis"></a>
</h3>
<p>Look specifically at case-control comparisons</p>
<div class="sourceCode" id="cb26"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://dplyr.tidyverse.org" class="external-link">dplyr</a></span><span class="op">)</span></span>
<span><span class="va">healthy</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/grep.html" class="external-link">grepl</a></span><span class="op">(</span><span class="st">".*(control.*COVID|COVID.*control).*"</span>, <span class="va">naso_sigs</span><span class="op">$</span><span class="va">comparison1</span>, ignore.case <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span>
<span></span>
<span><span class="va">cc_naso_sigs1</span> <span class="op">&lt;-</span> <span class="va">naso_sigs</span> <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html" class="external-link">%&gt;%</a></span></span>
<span>      <span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">healthy</span> <span class="op">==</span> <span class="cn">TRUE</span><span class="op">)</span> <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html" class="external-link">%&gt;%</a></span></span>
<span>  <span class="fu"><a href="https://rdrr.io/r/base/subset.html" class="external-link">subset</a></span><span class="op">(</span><span class="va">Study</span> <span class="op">!=</span> <span class="st">"Study 458"</span><span class="op">)</span></span></code></pre></div>
</div>
</div>
<div class="section level2">
<h2 id="cluster-analysis">Cluster analysis<a class="anchor" aria-label="anchor" href="#cluster-analysis"></a>
</h2>
<p>Note, this EDA should really be done <em>before</em> hypothesis
testing.</p>
<p>First calculate pairwise overlaps for all signatures of length &gt;
1:</p>
<div class="sourceCode" id="cb27"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">allsigs</span> <span class="op">&lt;-</span> <span class="fu">bugsigdbr</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/bugsigdbr/man/getSignatures.html" class="external-link">getSignatures</a></span><span class="op">(</span><span class="va">covid_all</span>, tax.id.type <span class="op">=</span> <span class="st">"taxname"</span><span class="op">)</span></span>
<span><span class="va">allsigs</span> <span class="op">&lt;-</span> <span class="va">allsigs</span><span class="op">[</span><span class="fu"><a href="https://rdrr.io/r/base/lapply.html" class="external-link">sapply</a></span><span class="op">(</span><span class="va">allsigs</span>, <span class="va">length</span><span class="op">)</span> <span class="op">&gt;</span> <span class="fl">1</span><span class="op">]</span> <span class="co">#require length &gt; 1</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/dim.html" class="external-link">dim</a></span><span class="op">(</span><span class="va">allsigs</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## NULL</span></span></code></pre>
<div class="sourceCode" id="cb29"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">mydists</span> <span class="op">&lt;-</span> <span class="fu">BugSigDBStats</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/BugSigDBStats/man/calcPairwiseOverlaps.html" class="external-link">calcPairwiseOverlaps</a></span><span class="op">(</span><span class="va">allsigs</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/dim.html" class="external-link">dim</a></span><span class="op">(</span><span class="va">mydists</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## [1] 2722    8</span></span></code></pre>
<p>Create a matrix of Jaccard similarities (0 for no overlap, 1 for 100%
overlap)</p>
<div class="sourceCode" id="cb31"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">jmat</span> <span class="op">&lt;-</span> <span class="fu">BugSigDBStats</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/BugSigDBStats/man/calcJaccardSimilarity.html" class="external-link">calcJaccardSimilarity</a></span><span class="op">(</span><span class="va">allsigs</span><span class="op">)</span></span></code></pre></div>
<div class="sourceCode" id="cb32"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/jokergoo/ComplexHeatmap" class="external-link">ComplexHeatmap</a></span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## Loading required package: grid</span></span></code></pre>
<pre><code><span><span class="co">## ========================================</span></span>
<span><span class="co">## ComplexHeatmap version 2.22.0</span></span>
<span><span class="co">## Bioconductor page: http://bioconductor.org/packages/ComplexHeatmap/</span></span>
<span><span class="co">## Github page: https://github.com/jokergoo/ComplexHeatmap</span></span>
<span><span class="co">## Documentation: http://jokergoo.github.io/ComplexHeatmap-reference</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## If you use it in published research, please cite either one:</span></span>
<span><span class="co">## - Gu, Z. Complex Heatmap Visualization. iMeta 2022.</span></span>
<span><span class="co">## - Gu, Z. Complex heatmaps reveal patterns and correlations in multidimensional </span></span>
<span><span class="co">##     genomic data. Bioinformatics 2016.</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## The new InteractiveComplexHeatmap package can directly export static </span></span>
<span><span class="co">## complex heatmaps into an interactive Shiny app with zero effort. Have a try!</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## This message can be suppressed by:</span></span>
<span><span class="co">##   suppressPackageStartupMessages(library(ComplexHeatmap))</span></span>
<span><span class="co">## ========================================</span></span></code></pre>
<div class="sourceCode" id="cb35"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">siglengths</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/lapply.html" class="external-link">sapply</a></span><span class="op">(</span><span class="va">allsigs</span>, <span class="va">length</span><span class="op">)</span></span>
<span><span class="va">ha</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/ComplexHeatmap/man/HeatmapAnnotation.html" class="external-link">HeatmapAnnotation</a></span><span class="op">(</span>`Signature Length` <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/pkg/ComplexHeatmap/man/anno_barplot.html" class="external-link">anno_barplot</a></span><span class="op">(</span><span class="va">siglengths</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="va">hr</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/ComplexHeatmap/man/rowAnnotation.html" class="external-link">rowAnnotation</a></span><span class="op">(</span>`Signature Length` <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/pkg/ComplexHeatmap/man/anno_barplot.html" class="external-link">anno_barplot</a></span><span class="op">(</span><span class="va">siglengths</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="va">hm</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/ComplexHeatmap/man/Heatmap.html" class="external-link">Heatmap</a></span><span class="op">(</span></span>
<span>  <span class="va">jmat</span>,</span>
<span>  top_annotation <span class="op">=</span> <span class="va">ha</span>, left_annotation <span class="op">=</span> <span class="va">hr</span>,</span>
<span>  row_names_max_width <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/grid/unit.html" class="external-link">unit</a></span><span class="op">(</span><span class="fl">20</span>, <span class="st">"cm"</span><span class="op">)</span>,</span>
<span>  column_names_max_height <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/grid/unit.html" class="external-link">unit</a></span><span class="op">(</span><span class="fl">20</span>, <span class="st">"cm"</span><span class="op">)</span>,</span>
<span>  row_labels <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/grep.html" class="external-link">sub</a></span><span class="op">(</span><span class="st">".+:"</span>, <span class="st">""</span>, <span class="fu"><a href="https://rdrr.io/r/base/colnames.html" class="external-link">rownames</a></span><span class="op">(</span><span class="va">jmat</span><span class="op">)</span><span class="op">)</span>,</span>
<span>  column_labels <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/grep.html" class="external-link">sub</a></span><span class="op">(</span><span class="st">".+:"</span>, <span class="st">""</span>, <span class="fu"><a href="https://rdrr.io/r/base/colnames.html" class="external-link">colnames</a></span><span class="op">(</span><span class="va">jmat</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## The automatically generated colors map from the 1^st and 99^th of the</span></span>
<span><span class="co">## values in the matrix. There are outliers in the matrix whose patterns</span></span>
<span><span class="co">## might be hidden by this color mapping. You can manually set the color</span></span>
<span><span class="co">## to `col` argument.</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## Use `suppressMessages()` to turn off this message.</span></span></code></pre>
<div class="sourceCode" id="cb37"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">hm</span></span></code></pre></div>
<p><img src="capstoneanalysis_clare_files/figure-html/unnamed-chunk-16-1.png" width="1920"></p>
<p>Use this interactively to make an interactive heatmap (not evaluated
in vignette). Some expanding of the default size is required to see
anything. Creating a sub-heatmap, then exporting it as a table, allows
in-depth identification of the subgroups.</p>
<div class="sourceCode" id="cb38"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/jokergoo/InteractiveComplexHeatmap" class="external-link">InteractiveComplexHeatmap</a></span><span class="op">)</span></span>
<span><span class="va">hm</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/ComplexHeatmap/man/draw-dispatch.html" class="external-link">draw</a></span><span class="op">(</span><span class="va">hm</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/pkg/InteractiveComplexHeatmap/man/htShiny.html" class="external-link">htShiny</a></span><span class="op">(</span><span class="va">hm</span><span class="op">)</span></span></code></pre></div>
<div class="sourceCode" id="cb39"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">hc</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/stats/hclust.html" class="external-link">hclust</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/stats/dist.html" class="external-link">as.dist</a></span><span class="op">(</span><span class="va">jmat</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/graphics/plot.default.html" class="external-link">plot</a></span><span class="op">(</span><span class="va">hc</span><span class="op">)</span></span></code></pre></div>
<p><img src="capstoneanalysis_clare_files/figure-html/unnamed-chunk-18-1.png" width="1920"></p>
<p>This tree can be cut to show the clusters, for example. The clusters
of more than 1 signature but less than ~10 signatures are most likely to
be something interesting.</p>
<div class="sourceCode" id="cb40"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">clusts</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/sort.html" class="external-link">sort</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/stats/cutree.html" class="external-link">cutree</a></span><span class="op">(</span><span class="va">hc</span>, h <span class="op">=</span> <span class="fl">0.05</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/lapply.html" class="external-link">lapply</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/unique.html" class="external-link">unique</a></span><span class="op">(</span><span class="va">clusts</span><span class="op">)</span>, <span class="kw">function</span><span class="op">(</span><span class="va">i</span><span class="op">)</span> <span class="fu"><a href="https://rdrr.io/r/base/names.html" class="external-link">names</a></span><span class="op">(</span><span class="va">clusts</span><span class="op">)</span><span class="op">[</span><span class="va">clusts</span> <span class="op">==</span> <span class="va">i</span><span class="op">]</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## [[1]]</span></span>
<span><span class="co">##  [1] "bsdb:428/1/1_COVID-19:COVID-19-cases_vs_Healthy-controls_UP"                                                                                       </span></span>
<span><span class="co">##  [2] "bsdb:428/1/2_COVID-19:COVID-19-cases_vs_Healthy-controls_DOWN"                                                                                     </span></span>
<span><span class="co">##  [3] "bsdb:428/3/1_COVID-19:Recovered-COVID-19-cases_vs_Healthy-controls_UP"                                                                             </span></span>
<span><span class="co">##  [4] "bsdb:430/1/1_COVID-19:Patients-with-high-SARS-CoV-2-infectivity_vs_Patients-with-low-to-none-SARS-CoV-2-infectivity_DOWN"                          </span></span>
<span><span class="co">##  [5] "bsdb:441/1/1_COVID-19:COVID-19-cases_vs_Healthy-controls_UP"                                                                                       </span></span>
<span><span class="co">##  [6] "bsdb:441/3/2_COVID-19:COVID-19-patients_vs_H1N1-patients_DOWN"                                                                                     </span></span>
<span><span class="co">##  [7] "bsdb:449/1/2_COVID-19:COVID-19-cases_vs_Healthy-controls_DOWN"                                                                                     </span></span>
<span><span class="co">##  [8] "bsdb:453/3/1_COVID-19:Recovered-COVID-19-patients_vs_Non-infected-patients-with-unrelated-respiratory-medical-conditions_UP"                       </span></span>
<span><span class="co">##  [9] "bsdb:453/3/2_COVID-19:Recovered-COVID-19-patients_vs_Non-infected-patients-with-unrelated-respiratory-medical-conditions_DOWN"                     </span></span>
<span><span class="co">## [10] "bsdb:458/1/1_COVID-19:COVID-19-patients_vs_Non-intubated-patients-with-non-incubation-viral-pneumonia-or-non-incubation-non-infectious-diseases_UP"</span></span>
<span><span class="co">## [11] "bsdb:459/1/2_COVID-19:COVID-19-positive-patients_vs_COVID-19-negative-controls_UP"                                                                 </span></span>
<span><span class="co">## [12] "bsdb:464/1/2_COVID-19:Confirmed-COVID-19-patients_vs_Healthy-controls_UP"                                                                          </span></span>
<span><span class="co">## [13] "bsdb:469/1/1_COVID-19:COVID-19-patients_vs_Non-COVID-19-controls_DOWN"                                                                             </span></span>
<span><span class="co">## [14] "bsdb:474/1/2_COVID-19:COVID-19-patients_vs_Non-COVID-patients-with-URTI-or-COPD_DOWN"                                                              </span></span>
<span><span class="co">## [15] "bsdb:479/2/2_COVID-19:Mild-moderate-COVID-19-patients_vs_Healthy-controls_UP"                                                                      </span></span>
<span><span class="co">## [16] "bsdb:482/1/2_COVID-19:Severe-COVID-19-patients_vs_Mild-COVID-19-patients_UP"                                                                       </span></span>
<span><span class="co">## [17] "bsdb:483/1/1_COVID-19:Antibiotic-naive-COVID-19-patients_vs_Uninfected-controls,-pneumonia-patients,-antibiotics-treated-COVID-19-patients_UP"     </span></span>
<span><span class="co">## [18] "bsdb:483/3/1_COVID-19:Pneumonia-controls_vs_Uninfected-controls,-COVID-19-patients_DOWN"                                                           </span></span>
<span><span class="co">## [19] "bsdb:484/1/1_COVID-19:COVID-19-positive-patients_vs_COVID-19-negative-patients-with-fever-and-cough_UP"                                            </span></span>
<span><span class="co">## [20] "bsdb:485/1/1_COVID-19:COVID-19-patients-with-fever_vs_COVID-19-patients-without-fever_UP"                                                          </span></span>
<span><span class="co">## [21] "bsdb:511/1/2_COVID-19:COVID-19-patients_vs_Healthy-controls_DOWN"                                                                                  </span></span>
<span><span class="co">## [22] "bsdb:512/3/1_COVID-19:COVID-19-positive-patients_vs_COVID-19-negative-healthcare-workers-(controls)_UP"                                            </span></span>
<span><span class="co">## [23] "bsdb:513/3/1_COVID-19:COVID-19-positive-patients-with-antibiotic-use_vs_COVID-19-positive-patients-without-antibiotic-use_UP"                      </span></span>
<span><span class="co">## [24] "bsdb:734/1/1_COVID-19:COVID-19-infection-cases._vs_Control-group_UP"                                                                               </span></span>
<span><span class="co">## [25] "bsdb:734/1/2_COVID-19:COVID-19-infection-cases._vs_Control-group_DOWN"                                                                             </span></span>
<span><span class="co">## [26] "bsdb:831/1/1_COVID-19:COVID-19-Patients_vs_Non-Covid-19-Patients---pneumonia-patients_UP"                                                          </span></span>
<span><span class="co">## [27] "bsdb:940/1/2_COVID-19:Covid-19-group_vs_Control-group_DOWN"                                                                                        </span></span>
<span><span class="co">## [28] "bsdb:940/2/1_COVID-19:Post-Covid-19-group_vs_Covid-19-group_UP"                                                                                    </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## [[2]]</span></span>
<span><span class="co">##  [1] "bsdb:428/2/1_COVID-19:Recovered-COVID-19-cases_vs_Healthy-controls_UP"                                                         </span></span>
<span><span class="co">##  [2] "bsdb:428/2/2_COVID-19:Recovered-COVID-19-cases_vs_Healthy-controls_DOWN"                                                       </span></span>
<span><span class="co">##  [3] "bsdb:441/1/2_COVID-19:COVID-19-cases_vs_Healthy-controls_DOWN"                                                                 </span></span>
<span><span class="co">##  [4] "bsdb:441/4/1_COVID-19:COVID-19-cases_vs_Healthy-controls_UP"                                                                   </span></span>
<span><span class="co">##  [5] "bsdb:453/1/1_COVID-19:SARS-CoV-2-infected-patients_vs_Non-infected-patients-with-unrelated-respiratory-medical-conditions_UP"  </span></span>
<span><span class="co">##  [6] "bsdb:464/2/2_COVID-19:Recovered-COVID-19-patients_vs_Healthy-controls_UP"                                                      </span></span>
<span><span class="co">##  [7] "bsdb:474/1/1_COVID-19:COVID-19-patients_vs_Non-COVID-patients-with-URTI-or-COPD_UP"                                            </span></span>
<span><span class="co">##  [8] "bsdb:476/1/1_COVID-19:Recovered-COVID-19-patients_vs_Healthy-controls_UP"                                                      </span></span>
<span><span class="co">##  [9] "bsdb:478/1/2_COVID-19:COVID-19-patients_vs_Healthy-controls_UP"                                                                </span></span>
<span><span class="co">## [10] "bsdb:479/2/1_COVID-19:Mild-moderate-COVID-19-patients_vs_Healthy-controls_DOWN"                                                </span></span>
<span><span class="co">## [11] "bsdb:481/1/1_COVID-19:COVID-19-patients_vs_Healthy-controls_UP"                                                                </span></span>
<span><span class="co">## [12] "bsdb:481/3/1_COVID-19:Severe-COVID-19-patients_vs_Mild-COVID-19-patients_UP"                                                   </span></span>
<span><span class="co">## [13] "bsdb:487/1/1_COVID-19:Recovered-COVID-19-samples-(respiratory-negative)_vs_Infected-COVID-19-samples-(respiratory-positive)_UP"</span></span>
<span><span class="co">## [14] "bsdb:498/1/2_COVID-19:Symptomatic,-mild-to-moderate-COVID-19-patients_vs_Asymptomatic-uninfected-controls_DOWN"                </span></span>
<span><span class="co">## [15] "bsdb:511/3/2_COVID-19:COVID-19-patients-not-treated-with-antibiotics_vs_Healthy-controls_DOWN"                                 </span></span>
<span><span class="co">## [16] "bsdb:512/2/1_COVID-19:COVID-19-positive-patients_vs_COVID-19-negative-controls_UP"                                             </span></span>
<span><span class="co">## [17] "bsdb:828/3/1_COVID-19,Respiratory-failure-requiring-assisted-ventilation:Intubated_vs_Non-intubated_UP"                        </span></span>
<span><span class="co">## [18] "bsdb:940/1/1_COVID-19:Covid-19-group_vs_Control-group_UP"                                                                      </span></span>
<span><span class="co">## [19] "bsdb:940/3/1_COVID-19:Post-Covid-19-group_vs_Control-group_UP"                                                                 </span></span>
<span><span class="co">## [20] "bsdb:940/5/2_COVID-19:Covid-19-C-group_vs_Control-group_DOWN"                                                                  </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## [[3]]</span></span>
<span><span class="co">##  [1] "bsdb:428/3/2_COVID-19:Recovered-COVID-19-cases_vs_Healthy-controls_DOWN"                                                                             </span></span>
<span><span class="co">##  [2] "bsdb:453/2/1_COVID-19:SARS-CoV-2-recovered-individuals_vs_SARS-CoV-2-infected-patients_DOWN"                                                         </span></span>
<span><span class="co">##  [3] "bsdb:458/1/2_COVID-19:COVID-19-patients_vs_Non-intubated-patients-with-non-incubation-viral-pneumonia-or-non-incubation-non-infectious-diseases_DOWN"</span></span>
<span><span class="co">##  [4] "bsdb:478/2/1_COVID-19:COVID-19-patients_vs_Flu-patients_DOWN"                                                                                        </span></span>
<span><span class="co">##  [5] "bsdb:479/1/1_COVID-19:ICU-COVID-19-patients_vs_Healthy-controls_DOWN"                                                                                </span></span>
<span><span class="co">##  [6] "bsdb:479/4/2_COVID-19:ICU-COVID-19-patients_vs_Patients-infected-with-other-human-coronaviruses_UP"                                                  </span></span>
<span><span class="co">##  [7] "bsdb:481/2/1_COVID-19:Antibiotic-treated-COVID-19-patients_vs_Antibiotic-naive-COVID-19-patients_UP"                                                 </span></span>
<span><span class="co">##  [8] "bsdb:485/2/2_COVID-19:COVID-19-patients-with-fever_vs_COVID-19-patients-without-fever_DOWN"                                                          </span></span>
<span><span class="co">##  [9] "bsdb:487/2/1_COVID-19:Recovered-COVID-19-samples-(respiratory-negative)_vs_Healthy-controls_DOWN"                                                    </span></span>
<span><span class="co">## [10] "bsdb:782/3/1_COVID-19:Feces-samples-from-COVID-19-patients_vs_samples-from-Healthy-patients_UP"                                                      </span></span>
<span><span class="co">## [11] "bsdb:940/2/2_COVID-19:Post-Covid-19-group_vs_Covid-19-group_DOWN"                                                                                    </span></span>
<span><span class="co">## [12] "bsdb:940/4/2_COVID-19:Covid-19-N-group_vs_Control-group_DOWN"                                                                                        </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## [[4]]</span></span>
<span><span class="co">##  [1] "bsdb:428/4/1_COVID-19:COVID-19-patients-treated-with-antibiotics_vs_Healthy-controls_DOWN"                                     </span></span>
<span><span class="co">##  [2] "bsdb:428/4/2_COVID-19:COVID-19-patients-treated-with-antibiotics_vs_Healthy-controls_UP"                                       </span></span>
<span><span class="co">##  [3] "bsdb:430/1/2_COVID-19:Patients-with-high-SARS-CoV-2-infectivity_vs_Patients-with-low-to-none-SARS-CoV-2-infectivity_UP"        </span></span>
<span><span class="co">##  [4] "bsdb:453/1/2_COVID-19:SARS-CoV-2-infected-patients_vs_Non-infected-patients-with-unrelated-respiratory-medical-conditions_DOWN"</span></span>
<span><span class="co">##  [5] "bsdb:453/4/2_COVID-19:Positive-for-COVID-19-viral-RNA-in-feces_vs_Negative-for-COVID-19-viral-RNA-in-feces_DOWN"               </span></span>
<span><span class="co">##  [6] "bsdb:464/2/1_COVID-19:Recovered-COVID-19-patients_vs_Healthy-controls_DOWN"                                                    </span></span>
<span><span class="co">##  [7] "bsdb:477/1/1_COVID-19:Severe-COVID-19-patients_vs_COVID-19-negative-controls_UP"                                               </span></span>
<span><span class="co">##  [8] "bsdb:479/3/1_COVID-19:ICU-COVID-19-patients_vs_Mild-Moderate-COVID-19-patients_DOWN"                                           </span></span>
<span><span class="co">##  [9] "bsdb:484/2/1_COVID-19:COVID-19-positive-patients_vs_Healthy-controls_UP"                                                       </span></span>
<span><span class="co">## [10] "bsdb:498/1/1_COVID-19:Symptomatic,-mild-to-moderate-COVID-19-patients_vs_Asymptomatic-uninfected-controls_UP"                  </span></span>
<span><span class="co">## [11] "bsdb:940/4/1_COVID-19:Covid-19-N-group_vs_Control-group_UP"                                                                    </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## [[5]]</span></span>
<span><span class="co">##  [1] "bsdb:441/3/1_COVID-19:COVID-19-patients_vs_H1N1-patients_UP"                                           </span></span>
<span><span class="co">##  [2] "bsdb:473/1/1_COVID-19:COVID-19-positive-patients_vs_COVID-19-negative-controls_DOWN"                   </span></span>
<span><span class="co">##  [3] "bsdb:478/3/1_COVID-19:Flu-patients_vs_Healthy-controls_UP"                                             </span></span>
<span><span class="co">##  [4] "bsdb:479/1/2_COVID-19:ICU-COVID-19-patients_vs_Healthy-controls_UP"                                    </span></span>
<span><span class="co">##  [5] "bsdb:481/2/2_COVID-19:Antibiotic-treated-COVID-19-patients_vs_Antibiotic-naive-COVID-19-patients_DOWN" </span></span>
<span><span class="co">##  [6] "bsdb:484/4/1_COVID-19:COVID-19-positive-patients_vs_COVID-19-negative-patients-with-fever-and-cough_UP"</span></span>
<span><span class="co">##  [7] "bsdb:500/1/1_COVID-19:COVID-19-patients_vs_COVID-19-negative-patients-with-pneumonia_UP"               </span></span>
<span><span class="co">##  [8] "bsdb:511/2/1_COVID-19:COVID-19-patients-(severe/critical)_vs_COVID-19-patients-(mild/moderate)_DOWN"   </span></span>
<span><span class="co">##  [9] "bsdb:831/1/2_COVID-19:COVID-19-Patients_vs_Non-Covid-19-Patients---pneumonia-patients_DOWN"            </span></span>
<span><span class="co">## [10] "bsdb:940/5/1_COVID-19:Covid-19-C-group_vs_Control-group_UP"                                            </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## [[6]]</span></span>
<span><span class="co">## [1] "bsdb:449/1/1_COVID-19:COVID-19-cases_vs_Healthy-controls_UP"                                                                 </span></span>
<span><span class="co">## [2] "bsdb:453/2/2_COVID-19:SARS-CoV-2-recovered-individuals_vs_SARS-CoV-2-infected-patients_UP"                                   </span></span>
<span><span class="co">## [3] "bsdb:459/1/1_COVID-19:COVID-19-positive-patients_vs_COVID-19-negative-controls_DOWN"                                         </span></span>
<span><span class="co">## [4] "bsdb:478/5/1_COVID-19:Flu-patients_vs_Healthy-controls_DOWN"                                                                 </span></span>
<span><span class="co">## [5] "bsdb:481/1/2_COVID-19:COVID-19-patients_vs_Healthy-controls_DOWN"                                                            </span></span>
<span><span class="co">## [6] "bsdb:486/1/2_COVID-19:Asymptomatic-COVID-19-patients_vs_Healthy-controls_DOWN"                                               </span></span>
<span><span class="co">## [7] "bsdb:486/2/1_COVID-19:Asymptomatic-COVID-19-patients_vs_Healthy-controls_UP"                                                 </span></span>
<span><span class="co">## [8] "bsdb:489/1/1_COVID-19:COVID-positive-patients-admitted-to-ICU-(i-COVID-19)_vs_Pneumonia-control-patients-(COVID-negative)_UP"</span></span>
<span><span class="co">## [9] "bsdb:513/1/1_COVID-19:COVID-19-positive-patients_vs_COVID-19-negative-controls_UP"                                           </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## [[7]]</span></span>
<span><span class="co">## [1] "bsdb:453/4/1_COVID-19:Positive-for-COVID-19-viral-RNA-in-feces_vs_Negative-for-COVID-19-viral-RNA-in-feces_UP"                                              </span></span>
<span><span class="co">## [2] "bsdb:464/3/2_COVID-19:COVID-19-patients_vs_Healthy-controls_UP"                                                                                             </span></span>
<span><span class="co">## [3] "bsdb:476/1/2_COVID-19:Recovered-COVID-19-patients_vs_Healthy-controls_DOWN"                                                                                 </span></span>
<span><span class="co">## [4] "bsdb:489/2/1_COVID-19:COVID-positive-patients-admitted-to-ICU-(i-COVID-19)_vs_COVID-positive-patients-admitted-to-infectious-disease-ward-(w-COVID-19)_UP"  </span></span>
<span><span class="co">## [5] "bsdb:489/2/2_COVID-19:COVID-positive-patients-admitted-to-ICU-(i-COVID-19)_vs_COVID-positive-patients-admitted-to-infectious-disease-ward-(w-COVID-19)_DOWN"</span></span>
<span><span class="co">## [6] "bsdb:498/2/1_COVID-19:COVID-19-patients-with-high-viral-load_vs_COVID-19-patients-with-low-viral-load_UP"                                                   </span></span>
<span><span class="co">## [7] "bsdb:546/1/2_COVID-19:COVID-19-cases_vs_Non-infected-controls_DOWN"                                                                                         </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## [[8]]</span></span>
<span><span class="co">## [1] "bsdb:464/1/1_COVID-19:Confirmed-COVID-19-patients_vs_Healthy-controls_DOWN"                                                                    </span></span>
<span><span class="co">## [2] "bsdb:478/4/2_COVID-19:COVID-19-patients_vs_Healthy-controls_UP"                                                                                </span></span>
<span><span class="co">## [3] "bsdb:479/3/2_COVID-19:ICU-COVID-19-patients_vs_Mild-Moderate-COVID-19-patients_UP"                                                             </span></span>
<span><span class="co">## [4] "bsdb:483/2/1_COVID-19:Antibiotic-treated-COVID-19-patients_vs_Antibiotic-naive-COVID-19-patients,-pneumonia-patients,-uninfected-controls_DOWN"</span></span>
<span><span class="co">## [5] "bsdb:485/1/2_COVID-19:COVID-19-patients-with-fever_vs_COVID-19-patients-without-fever_DOWN"                                                    </span></span>
<span><span class="co">## [6] "bsdb:831/2/1_COVID-19:COVID-19-patients---Ventilated_vs_COVID-19-patients---Non-Ventilated_UP"                                                 </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## [[9]]</span></span>
<span><span class="co">## [1] "bsdb:464/3/1_COVID-19:COVID-19-patients_vs_Healthy-controls_DOWN"                                                                          </span></span>
<span><span class="co">## [2] "bsdb:478/1/1_COVID-19:COVID-19-patients_vs_Healthy-controls_DOWN"                                                                          </span></span>
<span><span class="co">## [3] "bsdb:478/5/2_COVID-19:Flu-patients_vs_Healthy-controls_UP"                                                                                 </span></span>
<span><span class="co">## [4] "bsdb:489/3/2_COVID-19:COVID-positive-patients-admitted-to-infectious-disease-ward-(w-COVID-19)_vs_Pneumonia-controls-(COVID-negative)_DOWN"</span></span>
<span><span class="co">## [5] "bsdb:511/5/1_COVID-19:COVID-19-patients-not-treated-with-antibiotics_vs_Healthy-controls_UP"                                               </span></span>
<span><span class="co">## [6] "bsdb:965/1/1_COVID-19:COVID-19-patients_vs_healthy-controls_UP"                                                                            </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## [[10]]</span></span>
<span><span class="co">## [1] "bsdb:478/3/2_COVID-19:Flu-patients_vs_Healthy-controls_DOWN"                                         </span></span>
<span><span class="co">## [2] "bsdb:479/4/1_COVID-19:ICU-COVID-19-patients_vs_Patients-infected-with-other-human-coronaviruses_DOWN"</span></span>
<span><span class="co">## [3] "bsdb:482/1/1_COVID-19:Severe-COVID-19-patients_vs_Mild-COVID-19-patients_DOWN"                       </span></span>
<span><span class="co">## [4] "bsdb:485/2/1_COVID-19:COVID-19-patients-with-fever_vs_COVID-19-patients-without-fever_UP"            </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## [[11]]</span></span>
<span><span class="co">## [1] "bsdb:478/4/1_COVID-19:COVID-19-patients_vs_Healthy-controls_DOWN"                                          </span></span>
<span><span class="co">## [2] "bsdb:486/1/1_COVID-19:Asymptomatic-COVID-19-patients_vs_Healthy-controls_UP"                               </span></span>
<span><span class="co">## [3] "bsdb:498/2/2_COVID-19:COVID-19-patients-with-high-viral-load_vs_COVID-19-patients-with-low-viral-load_DOWN"</span></span>
<span><span class="co">## [4] "bsdb:511/3/1_COVID-19:COVID-19-patients-not-treated-with-antibiotics_vs_Healthy-controls_UP"               </span></span>
<span><span class="co">## [5] "bsdb:511/4/2_COVID-19:COVID-19-patients_vs_Healthy-controls_DOWN"                                          </span></span>
<span><span class="co">## [6] "bsdb:547/1/1_COVID-19:Recovered-COVID-19-patients_vs_Healthy-controls_UP"                                  </span></span>
<span><span class="co">## [7] "bsdb:548/1/2_COVID-19:COVID-19-cases_vs_Healthy-controls_DOWN"                                             </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## [[12]]</span></span>
<span><span class="co">## [1] "bsdb:486/2/2_COVID-19:Asymptomatic-COVID-19-patients_vs_Healthy-controls_DOWN"                           </span></span>
<span><span class="co">## [2] "bsdb:511/1/1_COVID-19:COVID-19-patients_vs_Healthy-controls_UP"                                          </span></span>
<span><span class="co">## [3] "bsdb:512/1/1_COVID-19:COVID-19-positive-patients_vs_COVID-19-negative-controls-and-healthcare-workers_UP"</span></span>
<span><span class="co">## [4] "bsdb:782/2/1_COVID-19:Sputum-samples-from-COVID-19-patients_vs_samples-from-Healthy-patients_UP"         </span></span>
<span><span class="co">## [5] "bsdb:782/2/2_COVID-19:Sputum-samples-from-COVID-19-patients_vs_samples-from-Healthy-patients_DOWN"       </span></span>
<span><span class="co">## [6] "bsdb:1040/1/2_COVID-19:RPs-(Recovered-Patients)_vs_HCs-(Healthy-Controls)_DOWN"                          </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## [[13]]</span></span>
<span><span class="co">## [1] "bsdb:486/3/1_COVID-19:Severe-COVID-19-patients_vs_Healthy-controls_UP"                                                         </span></span>
<span><span class="co">## [2] "bsdb:489/1/2_COVID-19:COVID-positive-patients-admitted-to-ICU-(i-COVID-19)_vs_Pneumonia-control-patients-(COVID-negative)_DOWN"</span></span>
<span><span class="co">## [3] "bsdb:496/2/1_COVID-19:COVID-19-patients_vs_Seasonal-flu-patients_UP"                                                           </span></span>
<span><span class="co">## [4] "bsdb:977/3/2_COVID-19:Covid-19-patients_vs_non-Covid-19-patients_DOWN"                                                         </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## [[14]]</span></span>
<span><span class="co">## [1] "bsdb:486/3/2_COVID-19:Severe-COVID-19-patients_vs_Healthy-controls_DOWN"                                      </span></span>
<span><span class="co">## [2] "bsdb:511/4/1_COVID-19:COVID-19-patients_vs_Healthy-controls_UP"                                               </span></span>
<span><span class="co">## [3] "bsdb:965/2/2_COVID-19:COVID-19-patients_vs_NO-COVID-19-(Patients-with-diagnosis-different-from-COVID-19)_DOWN"</span></span>
<span><span class="co">## [4] "bsdb:977/6/2_COVID-19:Hospitalized-patients_vs_local-controls_DOWN"                                           </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## [[15]]</span></span>
<span><span class="co">## [1] "bsdb:487/2/2_COVID-19:Recovered-COVID-19-samples-(respiratory-negative)_vs_Healthy-controls_UP"       </span></span>
<span><span class="co">## [2] "bsdb:487/3/1_COVID-19:Infected-COVID-19-samples-(respiratory-positive)_vs_Healthy-controls_DOWN"      </span></span>
<span><span class="co">## [3] "bsdb:782/1/1_COVID-19:Pharyngeal-samples-from-COVID-19-patients_vs_samples-from-Healthy-patients_DOWN"</span></span>
<span><span class="co">## [4] "bsdb:831/2/2_COVID-19:COVID-19-patients---Ventilated_vs_COVID-19-patients---Non-Ventilated_DOWN"      </span></span>
<span><span class="co">## [5] "bsdb:977/6/1_COVID-19:Hospitalized-patients_vs_local-controls_UP"                                     </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## [[16]]</span></span>
<span><span class="co">## [1] "bsdb:487/3/2_COVID-19:Infected-COVID-19-samples-(respiratory-positive)_vs_Healthy-controls_UP"</span></span>
<span><span class="co">## [2] "bsdb:977/1/1_COVID-19:COVID-19-patients_vs_local-controls_UP"                                 </span></span>
<span><span class="co">## [3] "bsdb:977/1/2_COVID-19:COVID-19-patients_vs_local-controls_DOWN"                               </span></span>
<span><span class="co">## [4] "bsdb:1040/1/1_COVID-19:RPs-(Recovered-Patients)_vs_HCs-(Healthy-Controls)_UP"                 </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## [[17]]</span></span>
<span><span class="co">## [1] "bsdb:489/3/1_COVID-19:COVID-positive-patients-admitted-to-infectious-disease-ward-(w-COVID-19)_vs_Pneumonia-controls-(COVID-negative)_UP"</span></span>
<span><span class="co">## [2] "bsdb:496/2/2_COVID-19:COVID-19-patients_vs_Seasonal-flu-patients_DOWN"                                                                   </span></span>
<span><span class="co">## [3] "bsdb:548/1/1_COVID-19:COVID-19-cases_vs_Healthy-controls_UP"                                                                             </span></span>
<span><span class="co">## [4] "bsdb:782/3/2_COVID-19:Feces-samples-from-COVID-19-patients_vs_samples-from-Healthy-patients_DOWN"                                        </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## [[18]]</span></span>
<span><span class="co">## [1] "bsdb:496/1/1_COVID-19:COVID-19-patients_vs_Healthy-controls_UP"                               </span></span>
<span><span class="co">## [2] "bsdb:511/5/2_COVID-19:COVID-19-patients-not-treated-with-antibiotics_vs_Healthy-controls_DOWN"</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## [[19]]</span></span>
<span><span class="co">## [1] "bsdb:496/1/2_COVID-19:COVID-19-patients_vs_Healthy-controls_DOWN"                                   </span></span>
<span><span class="co">## [2] "bsdb:782/1/2_COVID-19:Pharyngeal-samples-from-COVID-19-patients_vs_samples-from-Healthy-patients_UP"</span></span>
<span><span class="co">## [3] "bsdb:977/3/1_COVID-19:Covid-19-patients_vs_non-Covid-19-patients_UP"                                </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## [[20]]</span></span>
<span><span class="co">## [1] "bsdb:500/1/2_COVID-19:COVID-19-patients_vs_COVID-19-negative-patients-with-pneumonia_DOWN"</span></span>
<span><span class="co">## [2] "bsdb:547/1/2_COVID-19:Recovered-COVID-19-patients_vs_Healthy-controls_DOWN"               </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## [[21]]</span></span>
<span><span class="co">## [1] "bsdb:513/2/1_COVID-19:COVID-19-positive-patients-with-breathing-assistance_vs_COVID-19-positive-patients-without-breathing-assistance_UP"</span></span>
<span><span class="co">## [2] "bsdb:965/1/2_COVID-19:COVID-19-patients_vs_healthy-controls_DOWN"</span></span></code></pre>
</div>
<div class="section level2">
<h2 id="create-a-wide-format-dataframe">Create a wide-format dataframe<a class="anchor" aria-label="anchor" href="#create-a-wide-format-dataframe"></a>
</h2>
<p>This would be suitable for regression analysis.</p>
<div class="sourceCode" id="cb42"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">covid_withsigs</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/stats/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">covid_all</span>, <span class="op">!</span><span class="fu"><a href="https://rdrr.io/r/base/NA.html" class="external-link">is.na</a></span><span class="op">(</span><span class="va">covid_all</span><span class="op">$</span><span class="va">`NCBI Taxonomy IDs`</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="va">sigs</span> <span class="op">&lt;-</span> <span class="fu">bugsigdbr</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/bugsigdbr/man/getSignatures.html" class="external-link">getSignatures</a></span><span class="op">(</span><span class="va">covid_withsigs</span>, tax.id.type <span class="op">=</span> <span class="st">"taxname"</span><span class="op">)</span></span>
<span><span class="va">cmat</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/t.html" class="external-link">t</a></span><span class="op">(</span><span class="fu">safe</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/safe/man/getCmatrix.html" class="external-link">getCmatrix</a></span><span class="op">(</span><span class="va">sigs</span>, as.matrix <span class="op">=</span> <span class="cn">TRUE</span>, min.size <span class="op">=</span> <span class="fl">0</span>, prune <span class="op">=</span> <span class="cn">FALSE</span><span class="op">)</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## WARNING: rows are sorted elements of keyword.list</span></span>
<span><span class="co">## 170 categories formed</span></span></code></pre>
<div class="sourceCode" id="cb44"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">cdf</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/data.frame.html" class="external-link">data.frame</a></span><span class="op">(</span><span class="va">cmat</span>, stringsAsFactors <span class="op">=</span> <span class="cn">FALSE</span>, check.names <span class="op">=</span> <span class="cn">FALSE</span><span class="op">)</span></span>
<span><span class="va">cdf</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/cbind.html" class="external-link">cbind</a></span><span class="op">(</span><span class="va">covid_withsigs</span>, <span class="va">cdf</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/colnames.html" class="external-link">colnames</a></span><span class="op">(</span><span class="va">cdf</span><span class="op">)</span><span class="op">[</span><span class="fl">1</span><span class="op">:</span><span class="fl">54</span><span class="op">]</span></span></code></pre></div>
<pre><code><span><span class="co">##  [1] "BSDB ID"                    "Study"                     </span></span>
<span><span class="co">##  [3] "Study design"               "PMID"                      </span></span>
<span><span class="co">##  [5] "DOI"                        "URL"                       </span></span>
<span><span class="co">##  [7] "Authors list"               "Title"                     </span></span>
<span><span class="co">##  [9] "Journal"                    "Year"                      </span></span>
<span><span class="co">## [11] "Keywords"                   "Experiment"                </span></span>
<span><span class="co">## [13] "Location of subjects"       "Host species"              </span></span>
<span><span class="co">## [15] "Body site"                  "UBERON ID"                 </span></span>
<span><span class="co">## [17] "Condition"                  "EFO ID"                    </span></span>
<span><span class="co">## [19] "Group 0 name"               "Group 1 name"              </span></span>
<span><span class="co">## [21] "Group 1 definition"         "Group 0 sample size"       </span></span>
<span><span class="co">## [23] "Group 1 sample size"        "Antibiotics exclusion"     </span></span>
<span><span class="co">## [25] "Sequencing type"            "16S variable region"       </span></span>
<span><span class="co">## [27] "Sequencing platform"        "Statistical test"          </span></span>
<span><span class="co">## [29] "Significance threshold"     "MHT correction"            </span></span>
<span><span class="co">## [31] "LDA Score above"            "Matched on"                </span></span>
<span><span class="co">## [33] "Confounders controlled for" "Pielou"                    </span></span>
<span><span class="co">## [35] "Shannon"                    "Chao1"                     </span></span>
<span><span class="co">## [37] "Simpson"                    "Inverse Simpson"           </span></span>
<span><span class="co">## [39] "Richness"                   "Signature page name"       </span></span>
<span><span class="co">## [41] "Source"                     "Curated date"              </span></span>
<span><span class="co">## [43] "Curator"                    "Revision editor"           </span></span>
<span><span class="co">## [45] "Description"                "Abundance in Group 1"      </span></span>
<span><span class="co">## [47] "MetaPhlAn taxon names"      "NCBI Taxonomy IDs"         </span></span>
<span><span class="co">## [49] "State"                      "Reviewer"                  </span></span>
<span><span class="co">## [51] "studyexp"                   "site"                      </span></span>
<span><span class="co">## [53] "comparison1"                "[Clostridium] colinum"</span></span></code></pre>
<p>Note this has a number of columns that are mostly zeros, it could be
filtered significantly for any regression or machine learning
analysis:</p>
<div class="sourceCode" id="cb46"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/base/table.html" class="external-link">table</a></span><span class="op">(</span><span class="va">cdf</span><span class="op">[[</span><span class="st">"[Brevibacterium] frigoritolerans"</span><span class="op">]</span><span class="op">]</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## &lt; table of extent 0 &gt;</span></span></code></pre>
<p>Create another heatmap on correlations of presence/absence of taxa.
This is not necessary because the previous Jaccard Index heatmap is
probably better, it is just a demonstration of doing something with the
taxa presence/absence directly.</p>
<div class="sourceCode" id="cb48"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">sigcors</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/stats/cor.html" class="external-link">cor</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/t.html" class="external-link">t</a></span><span class="op">(</span><span class="va">cmat</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="va">siglengths</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/lapply.html" class="external-link">sapply</a></span><span class="op">(</span><span class="va">sigs</span>, <span class="va">length</span><span class="op">)</span></span>
<span><span class="va">ha</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/ComplexHeatmap/man/HeatmapAnnotation.html" class="external-link">HeatmapAnnotation</a></span><span class="op">(</span>`Signature Length` <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/pkg/ComplexHeatmap/man/anno_barplot.html" class="external-link">anno_barplot</a></span><span class="op">(</span><span class="va">siglengths</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="va">hr</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/ComplexHeatmap/man/rowAnnotation.html" class="external-link">rowAnnotation</a></span><span class="op">(</span>`Signature Length` <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/pkg/ComplexHeatmap/man/anno_barplot.html" class="external-link">anno_barplot</a></span><span class="op">(</span><span class="va">siglengths</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="va">hm</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/ComplexHeatmap/man/Heatmap.html" class="external-link">Heatmap</a></span><span class="op">(</span></span>
<span>  <span class="va">sigcors</span>,</span>
<span>  top_annotation <span class="op">=</span> <span class="va">ha</span>, left_annotation <span class="op">=</span> <span class="va">hr</span>,</span>
<span>  row_names_max_width <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/grid/unit.html" class="external-link">unit</a></span><span class="op">(</span><span class="fl">20</span>, <span class="st">"cm"</span><span class="op">)</span>,</span>
<span>  column_names_max_height <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/grid/unit.html" class="external-link">unit</a></span><span class="op">(</span><span class="fl">20</span>, <span class="st">"cm"</span><span class="op">)</span>,</span>
<span>  row_labels <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/grep.html" class="external-link">sub</a></span><span class="op">(</span><span class="st">".+:"</span>, <span class="st">""</span>, <span class="fu"><a href="https://rdrr.io/r/base/colnames.html" class="external-link">rownames</a></span><span class="op">(</span><span class="va">sigcors</span><span class="op">)</span><span class="op">)</span>, <span class="co">##removing study just to make signature names legible</span></span>
<span>  column_labels <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/grep.html" class="external-link">sub</a></span><span class="op">(</span><span class="st">".+:"</span>, <span class="st">""</span>, <span class="fu"><a href="https://rdrr.io/r/base/colnames.html" class="external-link">colnames</a></span><span class="op">(</span><span class="va">sigcors</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## The automatically generated colors map from the 1^st and 99^th of the</span></span>
<span><span class="co">## values in the matrix. There are outliers in the matrix whose patterns</span></span>
<span><span class="co">## might be hidden by this color mapping. You can manually set the color</span></span>
<span><span class="co">## to `col` argument.</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## Use `suppressMessages()` to turn off this message.</span></span></code></pre>
<div class="sourceCode" id="cb50"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">hm</span></span></code></pre></div>
<p><img src="capstoneanalysis_clare_files/figure-html/unnamed-chunk-22-1.png" width="1920"></p>
<p>Use this interactively to make an interactive heatmap:</p>
<div class="sourceCode" id="cb51"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/jokergoo/InteractiveComplexHeatmap" class="external-link">InteractiveComplexHeatmap</a></span><span class="op">)</span></span>
<span><span class="va">hm</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/ComplexHeatmap/man/draw-dispatch.html" class="external-link">draw</a></span><span class="op">(</span><span class="va">hm</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/pkg/InteractiveComplexHeatmap/man/htShiny.html" class="external-link">htShiny</a></span><span class="op">(</span><span class="va">hm</span><span class="op">)</span></span></code></pre></div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

      </div>

</div>



      <footer><div class="copyright">
  <p></p>
<p>Developed by Ludwig Geistlinger, Chloe Mirzayi, Fatima Zohra, Levi Waldron.</p>
</div>

<div class="pkgdown">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.1.1.</p>
</div>

      </footer>
</div>






  </body>
</html>
