<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>The Endometriosis-associated Microbiome â€¢ Reviews of published signatures based on bugsigdb.org</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.7.1/jquery.min.js" integrity="sha512-v2CJ7UaYy4JwqLDIrZUI/4hqeoQieOmAZNXBeQyjo21dadnwR+8ZaIJVT8EE2iyI61OV8e6M8PP2/4hpQINQ/g==" crossorigin="anonymous" referrerpolicy="no-referrer"></script><!-- Bootstrap --><link href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.4.0/flatly/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="The Endometriosis-associated Microbiome">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">


    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">Reviews of published signatures based on bugsigdb.org</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">0.99.6</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Articles

    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/c-section_meconium_shaimaa.html">Delivery Mode and the Meconium Microbiome</a>
    </li>
    <li>
      <a href="../articles/capstoneanalysis_BearklandM.html">Differential Abundance in the ocular microbiome in Dry Eye Disaease</a>
    </li>
    <li>
      <a href="../articles/capstoneanalysis_clare.html">COVID-19 associated microbiome</a>
    </li>
    <li>
      <a href="../articles/capstoneanalysis_fatima.html">An analysis of Most frequent taxa in Major Depression and Bipolar Disorder</a>
    </li>
    <li>
      <a href="../articles/capstoneanalysis_ifeanyi.html">Distinguishing socioeconomic and racial/ethnic variation of the human gut microbiome by re-analysis of studies from around the world</a>
    </li>
    <li>
      <a href="../articles/capstoneanalysis_kweku.html">The Irritable Bowel Syndrome-associated Microbiome</a>
    </li>
    <li>
      <a href="../articles/fieldworkanalysis_samara.html">The Endometriosis-associated Microbiome</a>
    </li>
    <li>
      <a href="../articles/healthysig.html">Analysis of prevalent stool genera in BugSigDB</a>
    </li>
  </ul>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/waldronlab/bugSigSimple" class="external-link">
    <span class="fa fa-github"></span>

  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->



      </header><script src="fieldworkanalysis_samara_files/kePrint-0.0.1/kePrint.js"></script><link href="fieldworkanalysis_samara_files/lightable-0.0.1/lightable.css" rel="stylesheet">
<div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>The Endometriosis-associated Microbiome</h1>
            
            <h4 data-toc-skip class="date">05 May, 2025</h4>
      

      <div class="hidden name"><code>fieldworkanalysis_samara.Rmd</code></div>

    </div>

    
    
<div class="section level2">
<h2 id="analysis-objectives">Analysis objectives<a class="anchor" aria-label="anchor" href="#analysis-objectives"></a>
</h2>
<ol style="list-style-type: decimal">
<li>Import, recode, and subset data from bugsigdb.org<br>
</li>
<li>Create a table of studies</li>
<li>Calculate the frequency of appearance of each taxa in independent
signatures and identify the most frequently reported taxa</li>
<li>Estimate the probability of the most frequently identified taxa
occuring by chance</li>
</ol>
</div>
<div class="section level2">
<h2 id="making-sure-packages-are-installed">Making sure packages are installed<a class="anchor" aria-label="anchor" href="#making-sure-packages-are-installed"></a>
</h2>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/install.packages.html" class="external-link">install.packages</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"devtools"</span>, <span class="st">"tidyverse"</span>, <span class="st">"kableExtra"</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="fu">devtools</span><span class="fu">::</span><span class="fu"><a href="https://remotes.r-lib.org/reference/install_github.html" class="external-link">install_github</a></span><span class="op">(</span><span class="st">"waldronlab/bugSigSimple"</span><span class="op">)</span></span>
<span><span class="fu">devtools</span><span class="fu">::</span><span class="fu"><a href="https://remotes.r-lib.org/reference/install_github.html" class="external-link">install_github</a></span><span class="op">(</span><span class="st">"waldronlab/BugSigDBStats"</span><span class="op">)</span></span>
<span><span class="fu">devtools</span><span class="fu">::</span><span class="fu"><a href="https://remotes.r-lib.org/reference/install_github.html" class="external-link">install_github</a></span><span class="op">(</span><span class="st">"waldronlab/bugsigdbr"</span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level2">
<h2 id="load-and-subset-data">Load and subset data<a class="anchor" aria-label="anchor" href="#load-and-subset-data"></a>
</h2>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://waldronlab.io/bugSigSimple" class="external-link">bugSigSimple</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/waldronlab/BugSigDBStats" class="external-link">BugSigDBStats</a></span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## </span></span>
<span><span class="co">## Attaching package: 'BugSigDBStats'</span></span></code></pre>
<pre><code><span><span class="co">## The following object is masked from 'package:bugSigSimple':</span></span>
<span><span class="co">## </span></span>
<span><span class="co">##     getMostFrequentTaxa</span></span></code></pre>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/waldronlab/bugsigdbr" class="external-link">bugsigdbr</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://tidyverse.tidyverse.org" class="external-link">tidyverse</a></span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## â”€â”€ <span style="font-weight: bold;">Attaching core tidyverse packages</span> â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€ tidyverse 2.0.0 â”€â”€</span></span>
<span><span class="co">## <span style="color: #00BB00;">âœ”</span> <span style="color: #0000BB;">dplyr    </span> 1.1.4     <span style="color: #00BB00;">âœ”</span> <span style="color: #0000BB;">readr    </span> 2.1.5</span></span>
<span><span class="co">## <span style="color: #00BB00;">âœ”</span> <span style="color: #0000BB;">forcats  </span> 1.0.0     <span style="color: #00BB00;">âœ”</span> <span style="color: #0000BB;">stringr  </span> 1.5.1</span></span>
<span><span class="co">## <span style="color: #00BB00;">âœ”</span> <span style="color: #0000BB;">ggplot2  </span> 3.5.2     <span style="color: #00BB00;">âœ”</span> <span style="color: #0000BB;">tibble   </span> 3.2.1</span></span>
<span><span class="co">## <span style="color: #00BB00;">âœ”</span> <span style="color: #0000BB;">lubridate</span> 1.9.4     <span style="color: #00BB00;">âœ”</span> <span style="color: #0000BB;">tidyr    </span> 1.3.1</span></span>
<span><span class="co">## <span style="color: #00BB00;">âœ”</span> <span style="color: #0000BB;">purrr    </span> 1.0.4</span></span></code></pre>
<pre><code><span><span class="co">## â”€â”€ <span style="font-weight: bold;">Conflicts</span> â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€ tidyverse_conflicts() â”€â”€</span></span>
<span><span class="co">## <span style="color: #BB0000;">âœ–</span> <span style="color: #0000BB;">dplyr</span>::<span style="color: #00BB00;">filter()</span> masks <span style="color: #0000BB;">stats</span>::filter()</span></span>
<span><span class="co">## <span style="color: #BB0000;">âœ–</span> <span style="color: #0000BB;">dplyr</span>::<span style="color: #00BB00;">lag()</span>    masks <span style="color: #0000BB;">stats</span>::lag()</span></span>
<span><span class="co">## <span style="color: #00BBBB;">â„¹</span> Use the conflicted package (<span style="color: #0000BB; font-style: italic;">&lt;http://conflicted.r-lib.org/&gt;</span>) to force all conflicts to become errors</span></span></code></pre>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://stringr.tidyverse.org" class="external-link">stringr</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="http://haozhu233.github.io/kableExtra/" class="external-link">kableExtra</a></span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## </span></span>
<span><span class="co">## Attaching package: 'kableExtra'</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## The following object is masked from 'package:dplyr':</span></span>
<span><span class="co">## </span></span>
<span><span class="co">##     group_rows</span></span></code></pre>
<div class="sourceCode" id="cb10"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://dplyr.tidyverse.org" class="external-link">dplyr</a></span><span class="op">)</span></span></code></pre></div>
<div class="sourceCode" id="cb11"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://waldronlab.io/bugSigSimple" class="external-link">bugSigSimple</a></span><span class="op">)</span></span>
<span><span class="va">dat</span> <span class="op">&lt;-</span> <span class="fu">bugsigdbr</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/bugsigdbr/man/importBugSigDB.html" class="external-link">importBugSigDB</a></span><span class="op">(</span>cache<span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## Using cached version from 2025-05-05 19:25:55</span></span></code></pre>
<div class="sourceCode" id="cb13"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/base/dim.html" class="external-link">dim</a></span><span class="op">(</span><span class="va">dat</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## [1] 8163   50</span></span></code></pre>
<div class="sourceCode" id="cb15"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/base/names.html" class="external-link">names</a></span><span class="op">(</span><span class="va">dat</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">##  [1] "BSDB ID"                    "Study"                     </span></span>
<span><span class="co">##  [3] "Study design"               "PMID"                      </span></span>
<span><span class="co">##  [5] "DOI"                        "URL"                       </span></span>
<span><span class="co">##  [7] "Authors list"               "Title"                     </span></span>
<span><span class="co">##  [9] "Journal"                    "Year"                      </span></span>
<span><span class="co">## [11] "Keywords"                   "Experiment"                </span></span>
<span><span class="co">## [13] "Location of subjects"       "Host species"              </span></span>
<span><span class="co">## [15] "Body site"                  "UBERON ID"                 </span></span>
<span><span class="co">## [17] "Condition"                  "EFO ID"                    </span></span>
<span><span class="co">## [19] "Group 0 name"               "Group 1 name"              </span></span>
<span><span class="co">## [21] "Group 1 definition"         "Group 0 sample size"       </span></span>
<span><span class="co">## [23] "Group 1 sample size"        "Antibiotics exclusion"     </span></span>
<span><span class="co">## [25] "Sequencing type"            "16S variable region"       </span></span>
<span><span class="co">## [27] "Sequencing platform"        "Statistical test"          </span></span>
<span><span class="co">## [29] "Significance threshold"     "MHT correction"            </span></span>
<span><span class="co">## [31] "LDA Score above"            "Matched on"                </span></span>
<span><span class="co">## [33] "Confounders controlled for" "Pielou"                    </span></span>
<span><span class="co">## [35] "Shannon"                    "Chao1"                     </span></span>
<span><span class="co">## [37] "Simpson"                    "Inverse Simpson"           </span></span>
<span><span class="co">## [39] "Richness"                   "Signature page name"       </span></span>
<span><span class="co">## [41] "Source"                     "Curated date"              </span></span>
<span><span class="co">## [43] "Curator"                    "Revision editor"           </span></span>
<span><span class="co">## [45] "Description"                "Abundance in Group 1"      </span></span>
<span><span class="co">## [47] "MetaPhlAn taxon names"      "NCBI Taxonomy IDs"         </span></span>
<span><span class="co">## [49] "State"                      "Reviewer"</span></span></code></pre>
</div>
<div class="section level2">
<h2 id="subsetting">Subsetting<a class="anchor" aria-label="anchor" href="#subsetting"></a>
</h2>
<p>Subsetting only studies done on humans - 6 of 17 studies were
excluded because they were either done on mice or not statistically
reliable (Cregger et. al)</p>
<div class="sourceCode" id="cb17"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">subset.dat</span> <span class="op">&lt;-</span></span>
<span>  <span class="va">dat</span><span class="op">[</span><span class="fu"><a href="https://rdrr.io/r/base/which.html" class="external-link">which</a></span><span class="op">(</span></span>
<span>    <span class="va">dat</span><span class="op">$</span><span class="va">PMID</span> <span class="op">==</span> <span class="st">"30778155"</span> <span class="op">|</span></span>
<span>      <span class="va">dat</span><span class="op">$</span><span class="va">PMID</span> <span class="op">==</span> <span class="st">"32192080"</span> <span class="op">|</span></span>
<span>      <span class="va">dat</span><span class="op">$</span><span class="va">PMID</span> <span class="op">==</span> <span class="st">"31087436"</span> <span class="op">|</span></span>
<span>      <span class="va">dat</span><span class="op">$</span><span class="va">PMID</span> <span class="op">==</span> <span class="st">"26901400"</span> <span class="op">|</span></span>
<span>      <span class="va">dat</span><span class="op">$</span><span class="va">PMID</span> <span class="op">==</span> <span class="st">"33839907"</span> <span class="op">|</span></span>
<span>      <span class="va">dat</span><span class="op">$</span><span class="va">PMID</span> <span class="op">==</span> <span class="st">"32046455"</span> <span class="op">|</span></span>
<span>      <span class="va">dat</span><span class="op">$</span><span class="va">PMID</span> <span class="op">==</span> <span class="st">"33925708"</span> <span class="op">|</span></span>
<span>      <span class="va">dat</span><span class="op">$</span><span class="va">PMID</span> <span class="op">==</span> <span class="st">"32299442"</span> <span class="op">|</span></span>
<span>      <span class="va">dat</span><span class="op">$</span><span class="va">PMID</span> <span class="op">==</span> <span class="st">"33313185"</span> <span class="op">|</span></span>
<span>      <span class="va">dat</span><span class="op">$</span><span class="va">PMID</span> <span class="op">==</span> <span class="st">"34268384"</span> <span class="op">|</span> <span class="va">dat</span><span class="op">$</span><span class="va">PMID</span> <span class="op">==</span> <span class="st">"33660232"</span></span>
<span>  <span class="op">)</span>, <span class="op">]</span></span></code></pre></div>
</div>
<div class="section level2">
<h2 id="all-studies">All studies<a class="anchor" aria-label="anchor" href="#all-studies"></a>
</h2>
<p>Summary of studies and most frequent taxa increased and decreased in
endometriosis patients for all studies</p>
<div class="sourceCode" id="cb18"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu">bugSigSimple</span><span class="fu">::</span><span class="fu"><a href="../reference/createStudyTable.html">createStudyTable</a></span><span class="op">(</span><span class="va">subset.dat</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## <span style="color: #949494;"># A tibble: 11 Ã— 9</span></span></span>
<span><span class="co">##    `Study code` MaxCases MaxControls `Study design` Condition N_signatures PMID </span></span>
<span><span class="co">##    <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>           <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span>       <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span> <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>          <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>            <span style="color: #949494; font-style: italic;">&lt;int&gt;</span> <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span></span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 1</span> AkiyamaK_20â€¦       30          39 cross-sectionâ€¦ Endometrâ€¦            1 3108â€¦</span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 2</span> AtaB_2019          14          14 prospective câ€¦ Endometrâ€¦            2 3077â€¦</span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 3</span> ChaoX_2021         37          66 case-control   Endometrâ€¦            4 3426â€¦</span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 4</span> ChenS_2020         14         120 case-control   Adenomyoâ€¦            3 3331â€¦</span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 5</span> HernandesC_â€¦       21          11 case-control   Endometrâ€¦            6 3219â€¦</span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 6</span> KhanKN_2016        32          32 case-control   Endometrâ€¦            5 2690â€¦</span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 7</span> LeeSR_2021         45          45 case-control   Endometrâ€¦            2 3392â€¦</span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 8</span> PerrottaAR_â€¦        8          12 cross-sectionâ€¦ Endometrâ€¦            2 3204â€¦</span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 9</span> ShanJ_2021         12          12 case-control   Endometrâ€¦            2 3383â€¦</span></span>
<span><span class="co">## <span style="color: #BCBCBC;">10</span> SvenssonA_2â€¦       66         198 case-control   Endometrâ€¦            4 3366â€¦</span></span>
<span><span class="co">## <span style="color: #BCBCBC;">11</span> WeiW_2020          36          14 cross-sectionâ€¦ Endometrâ€¦            2 3229â€¦</span></span>
<span><span class="co">## <span style="color: #949494;"># â„¹ 2 more variables: DOI &lt;chr&gt;, URL &lt;chr&gt;</span></span></span></code></pre>
<div class="sourceCode" id="cb20"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="../reference/getMostFrequentTaxa.html">getMostFrequentTaxa</a></span><span class="op">(</span><span class="va">subset.dat</span>, n<span class="op">=</span><span class="fl">30</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">##    1301    1380    1578     194     543     838   84107  119852    1300    1350 </span></span>
<span><span class="co">##       7       5       4       4       4       4       4       3       3       3 </span></span>
<span><span class="co">##  135621    1375  171549 1940338    2050   28050     286   39948     469     906 </span></span>
<span><span class="co">##       3       3       3       3       3       3       3       3       3       3 </span></span>
<span><span class="co">##   13687 1508657 1582879    1654    1663  168808    1743  216851    2701   33038 </span></span>
<span><span class="co">##       2       2       2       2       2       2       2       2       2       2</span></span></code></pre>
<div class="sourceCode" id="cb22"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="../reference/getMostFrequentTaxa.html">getMostFrequentTaxa</a></span><span class="op">(</span><span class="va">subset.dat</span>, direction<span class="op">=</span><span class="st">"UP"</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">##    1301     543  119852    1300    1350  135621    1578 1940338     286     469 </span></span>
<span><span class="co">##       6       4       3       3       3       3       3       3       3       3</span></span></code></pre>
<div class="sourceCode" id="cb24"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="../reference/getMostFrequentTaxa.html">getMostFrequentTaxa</a></span><span class="op">(</span><span class="va">subset.dat</span>, direction<span class="op">=</span><span class="st">"DOWN"</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">##   1380 168808 171549    194   2050  28050  33958    838    906   1033 </span></span>
<span><span class="co">##      3      2      2      2      2      2      2      2      2      1</span></span></code></pre>
</div>
<div class="section level2">
<h2 id="excluding-feces-samples">Excluding feces samples<a class="anchor" aria-label="anchor" href="#excluding-feces-samples"></a>
</h2>
<p>Summary of studies and most frequent taxa in only samples from female
reproductive tract, excluding feces samples</p>
<div class="sourceCode" id="cb26"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">subset.dat2</span> <span class="op">&lt;-</span></span>
<span>  <span class="va">dat</span><span class="op">[</span><span class="fu"><a href="https://rdrr.io/r/base/which.html" class="external-link">which</a></span><span class="op">(</span></span>
<span>    <span class="va">dat</span><span class="op">$</span><span class="va">PMID</span> <span class="op">==</span> <span class="st">"30778155"</span> <span class="op">|</span></span>
<span>      <span class="va">dat</span><span class="op">$</span><span class="va">PMID</span> <span class="op">==</span> <span class="st">"32192080"</span> <span class="op">|</span></span>
<span>      <span class="va">dat</span><span class="op">$</span><span class="va">PMID</span> <span class="op">==</span> <span class="st">"31087436"</span> <span class="op">|</span></span>
<span>      <span class="va">dat</span><span class="op">$</span><span class="va">PMID</span> <span class="op">==</span> <span class="st">"26901400"</span> <span class="op">|</span></span>
<span>      <span class="va">dat</span><span class="op">$</span><span class="va">PMID</span> <span class="op">==</span> <span class="st">"32046455"</span> <span class="op">|</span></span>
<span>      <span class="va">dat</span><span class="op">$</span><span class="va">PMID</span> <span class="op">==</span> <span class="st">"33925708"</span> <span class="op">|</span></span>
<span>      <span class="va">dat</span><span class="op">$</span><span class="va">PMID</span> <span class="op">==</span> <span class="st">"32299442"</span> <span class="op">|</span></span>
<span>      <span class="va">dat</span><span class="op">$</span><span class="va">PMID</span> <span class="op">==</span> <span class="st">"33313185"</span> <span class="op">|</span> <span class="va">dat</span><span class="op">$</span><span class="va">PMID</span> <span class="op">==</span> <span class="st">"34268384"</span></span>
<span>  <span class="op">)</span>, <span class="op">]</span></span>
<span></span>
<span><span class="va">reproductive_sigs</span> <span class="op">&lt;-</span></span>
<span>  <span class="va">subset.dat2</span><span class="op">[</span><span class="fu"><a href="https://rdrr.io/r/base/which.html" class="external-link">which</a></span><span class="op">(</span><span class="va">subset.dat2</span><span class="op">$</span><span class="va">`Body site`</span> <span class="op">!=</span> <span class="st">"feces"</span> <span class="op">|</span></span>
<span>                      <span class="fu"><a href="https://rdrr.io/r/base/NA.html" class="external-link">is.na</a></span><span class="op">(</span><span class="va">subset.dat2</span><span class="op">$</span><span class="va">`Body site`</span><span class="op">)</span><span class="op">)</span>, <span class="op">]</span></span>
<span></span>
<span><span class="fu">bugSigSimple</span><span class="fu">::</span><span class="fu"><a href="../reference/createStudyTable.html">createStudyTable</a></span><span class="op">(</span><span class="va">reproductive_sigs</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## <span style="color: #949494;"># A tibble: 9 Ã— 9</span></span></span>
<span><span class="co">##   `Study code`  MaxCases MaxControls `Study design` Condition N_signatures PMID </span></span>
<span><span class="co">##   <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>            <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span>       <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span> <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>          <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>            <span style="color: #949494; font-style: italic;">&lt;int&gt;</span> <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span></span></span>
<span><span class="co">## <span style="color: #BCBCBC;">1</span> AkiyamaK_2019       30          39 cross-sectionâ€¦ Endometrâ€¦            1 3108â€¦</span></span>
<span><span class="co">## <span style="color: #BCBCBC;">2</span> AtaB_2019           14          14 prospective câ€¦ Endometrâ€¦            2 3077â€¦</span></span>
<span><span class="co">## <span style="color: #BCBCBC;">3</span> ChaoX_2021          37          66 case-control   Endometrâ€¦            4 3426â€¦</span></span>
<span><span class="co">## <span style="color: #BCBCBC;">4</span> ChenS_2020          14         120 case-control   Adenomyoâ€¦            3 3331â€¦</span></span>
<span><span class="co">## <span style="color: #BCBCBC;">5</span> HernandesC_2â€¦       21          11 case-control   Endometrâ€¦            6 3219â€¦</span></span>
<span><span class="co">## <span style="color: #BCBCBC;">6</span> KhanKN_2016         32          32 case-control   Endometrâ€¦            5 2690â€¦</span></span>
<span><span class="co">## <span style="color: #BCBCBC;">7</span> LeeSR_2021          45          45 case-control   Endometrâ€¦            2 3392â€¦</span></span>
<span><span class="co">## <span style="color: #BCBCBC;">8</span> PerrottaAR_2â€¦        8          12 cross-sectionâ€¦ Endometrâ€¦            2 3204â€¦</span></span>
<span><span class="co">## <span style="color: #BCBCBC;">9</span> WeiW_2020           36          14 cross-sectionâ€¦ Endometrâ€¦            2 3229â€¦</span></span>
<span><span class="co">## <span style="color: #949494;"># â„¹ 2 more variables: DOI &lt;chr&gt;, URL &lt;chr&gt;</span></span></span></code></pre>
<div class="sourceCode" id="cb28"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">allfreqs</span> <span class="op">&lt;-</span> <span class="fu">bugSigSimple</span><span class="fu">::</span><span class="fu"><a href="../reference/createTaxonTable.html">createTaxonTable</a></span><span class="op">(</span><span class="va">reproductive_sigs</span>, n <span class="op">=</span> <span class="fl">20</span><span class="op">)</span> <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html" class="external-link">%&gt;%</a></span> <span class="co">#could change number</span></span>
<span>  <span class="fu"><a href="https://dplyr.tidyverse.org/reference/arrange.html" class="external-link">arrange</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/AsIs.html" class="external-link">I</a></span><span class="op">(</span><span class="va">decreased_signatures</span> <span class="op">-</span> <span class="va">increased_signatures</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="va">incfreqs</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">allfreqs</span>, <span class="fu"><a href="https://rdrr.io/r/base/AsIs.html" class="external-link">I</a></span><span class="op">(</span><span class="va">increased_signatures</span> <span class="op">-</span> <span class="va">decreased_signatures</span><span class="op">)</span> <span class="op">&gt;</span> <span class="fl">0</span><span class="op">)</span></span>
<span><span class="va">decfreqs</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">allfreqs</span>, <span class="fu"><a href="https://rdrr.io/r/base/AsIs.html" class="external-link">I</a></span><span class="op">(</span><span class="va">increased_signatures</span> <span class="op">-</span> <span class="va">decreased_signatures</span><span class="op">)</span> <span class="op">&lt;</span> <span class="fl">0</span><span class="op">)</span></span>
<span><span class="fu">kableExtra</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/kableExtra/man/kbl.html" class="external-link">kbl</a></span><span class="op">(</span><span class="va">allfreqs</span><span class="op">)</span> <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html" class="external-link">%&gt;%</a></span></span>
<span>  <span class="fu"><a href="https://rdrr.io/pkg/kableExtra/man/kable_classic.html" class="external-link">kable_paper</a></span><span class="op">(</span><span class="st">"hover"</span>, full_width <span class="op">=</span> <span class="cn">FALSE</span><span class="op">)</span></span></code></pre></div>
<table class="table  lightable-paper lightable-hover" style='font-family: "Arial Narrow", arial, helvetica, sans-serif; width: auto !important; margin-left: auto; margin-right: auto;'>
<thead><tr>
<th style="text-align:left;">
Taxon Name
</th>
<th style="text-align:left;">
Taxonomic Level
</th>
<th style="text-align:right;">
total_signatures
</th>
<th style="text-align:right;">
increased_signatures
</th>
<th style="text-align:right;">
decreased_signatures
</th>
<th style="text-align:right;">
Binomial Test pval
</th>
<th style="text-align:left;">
kingdom
</th>
<th style="text-align:left;">
phylum
</th>
<th style="text-align:left;">
class
</th>
<th style="text-align:left;">
order
</th>
<th style="text-align:left;">
family
</th>
<th style="text-align:left;">
genus
</th>
<th style="text-align:left;">
species
</th>
<th style="text-align:right;">
n_signatures
</th>
<th style="text-align:left;">
metaphlan_name
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
Streptococcus
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.22
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Bacilli
</td>
<td style="text-align:left;">
Lactobacillales
</td>
<td style="text-align:left;">
Streptococcaceae
</td>
<td style="text-align:left;">
Streptococcus
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:left;">
d__Bacteria|k__Bacillati|p__Bacillota|c__Bacilli|o__Lactobacillales|f__Streptococcaceae|g__Streptococcus
</td>
</tr>
<tr>
<td style="text-align:left;">
Coriobacteriaceae
</td>
<td style="text-align:left;">
family
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.25
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
Coriobacteriia
</td>
<td style="text-align:left;">
Coriobacteriales
</td>
<td style="text-align:left;">
Coriobacteriaceae
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:left;">
d__Bacteria|k__Bacillati|p__Actinomycetota|c__Coriobacteriia|o__Coriobacteriales|f__Coriobacteriaceae
</td>
</tr>
<tr>
<td style="text-align:left;">
Streptococcaceae
</td>
<td style="text-align:left;">
family
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.25
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Bacilli
</td>
<td style="text-align:left;">
Lactobacillales
</td>
<td style="text-align:left;">
Streptococcaceae
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:left;">
d__Bacteria|k__Bacillati|p__Bacillota|c__Bacilli|o__Lactobacillales|f__Streptococcaceae
</td>
</tr>
<tr>
<td style="text-align:left;">
Enterobacteriaceae
</td>
<td style="text-align:left;">
family
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.25
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Pseudomonadati
</td>
<td style="text-align:left;">
Pseudomonadota
</td>
<td style="text-align:left;">
Gammaproteobacteria
</td>
<td style="text-align:left;">
Enterobacterales
</td>
<td style="text-align:left;">
Enterobacteriaceae
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:left;">
d__Bacteria|k__Pseudomonadati|p__Pseudomonadota|c__Gammaproteobacteria|o__Enterobacterales|f__Enterobacteriaceae
</td>
</tr>
<tr>
<td style="text-align:left;">
Escherichia/Shigella sp.
</td>
<td style="text-align:left;">
species
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.25
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Pseudomonadati
</td>
<td style="text-align:left;">
Pseudomonadota
</td>
<td style="text-align:left;">
Gammaproteobacteria
</td>
<td style="text-align:left;">
Enterobacterales
</td>
<td style="text-align:left;">
Enterobacteriaceae
</td>
<td style="text-align:left;">
Escherichia/Shigella sp.
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:left;">
d__Bacteria|k__Pseudomonadati|p__Pseudomonadota|c__Gammaproteobacteria|o__Enterobacterales|f__Enterobacteriaceae|s__Escherichia/Shigella
sp.
</td>
</tr>
<tr>
<td style="text-align:left;">
Acinetobacter
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.25
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Pseudomonadati
</td>
<td style="text-align:left;">
Pseudomonadota
</td>
<td style="text-align:left;">
Gammaproteobacteria
</td>
<td style="text-align:left;">
Moraxellales
</td>
<td style="text-align:left;">
Moraxellaceae
</td>
<td style="text-align:left;">
Acinetobacter
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:left;">
d__Bacteria|k__Pseudomonadati|p__Pseudomonadota|c__Gammaproteobacteria|o__Moraxellales|f__Moraxellaceae|g__Acinetobacter
</td>
</tr>
<tr>
<td style="text-align:left;">
Pseudomonadaceae
</td>
<td style="text-align:left;">
family
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.25
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Pseudomonadati
</td>
<td style="text-align:left;">
Pseudomonadota
</td>
<td style="text-align:left;">
Gammaproteobacteria
</td>
<td style="text-align:left;">
Pseudomonadales
</td>
<td style="text-align:left;">
Pseudomonadaceae
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:left;">
d__Bacteria|k__Pseudomonadati|p__Pseudomonadota|c__Gammaproteobacteria|o__Pseudomonadales|f__Pseudomonadaceae
</td>
</tr>
<tr>
<td style="text-align:left;">
Pseudomonas
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.25
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Pseudomonadati
</td>
<td style="text-align:left;">
Pseudomonadota
</td>
<td style="text-align:left;">
Gammaproteobacteria
</td>
<td style="text-align:left;">
Pseudomonadales
</td>
<td style="text-align:left;">
Pseudomonadaceae
</td>
<td style="text-align:left;">
Pseudomonas
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:left;">
d__Bacteria|k__Pseudomonadati|p__Pseudomonadota|c__Gammaproteobacteria|o__Pseudomonadales|f__Pseudomonadaceae|g__Pseudomonas
</td>
</tr>
<tr>
<td style="text-align:left;">
Alloscardovia
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.50
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
Actinomycetes
</td>
<td style="text-align:left;">
Bifidobacteriales
</td>
<td style="text-align:left;">
Bifidobacteriaceae
</td>
<td style="text-align:left;">
Alloscardovia
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:left;">
d__Bacteria|k__Bacillati|p__Actinomycetota|c__Actinomycetes|o__Bifidobacteriales|f__Bifidobacteriaceae|g__Alloscardovia
</td>
</tr>
<tr>
<td style="text-align:left;">
Arthrobacter
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.50
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
Actinomycetes
</td>
<td style="text-align:left;">
Micrococcales
</td>
<td style="text-align:left;">
Micrococcaceae
</td>
<td style="text-align:left;">
Arthrobacter
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:left;">
d__Bacteria|k__Bacillati|p__Actinomycetota|c__Actinomycetes|o__Micrococcales|f__Micrococcaceae|g__Arthrobacter
</td>
</tr>
<tr>
<td style="text-align:left;">
Aerococcus
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.00
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Bacilli
</td>
<td style="text-align:left;">
Lactobacillales
</td>
<td style="text-align:left;">
Aerococcaceae
</td>
<td style="text-align:left;">
Aerococcus
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:left;">
d__Bacteria|k__Bacillati|p__Bacillota|c__Bacilli|o__Lactobacillales|f__Aerococcaceae|g__Aerococcus
</td>
</tr>
<tr>
<td style="text-align:left;">
Lactobacillus
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.00
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Bacilli
</td>
<td style="text-align:left;">
Lactobacillales
</td>
<td style="text-align:left;">
Lactobacillaceae
</td>
<td style="text-align:left;">
Lactobacillus
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:left;">
d__Bacteria|k__Bacillati|p__Bacillota|c__Bacilli|o__Lactobacillales|f__Lactobacillaceae|g__Lactobacillus
</td>
</tr>
<tr>
<td style="text-align:left;">
Dialister
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.00
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Negativicutes
</td>
<td style="text-align:left;">
Veillonellales
</td>
<td style="text-align:left;">
Veillonellaceae
</td>
<td style="text-align:left;">
Dialister
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:left;">
d__Bacteria|k__Bacillati|p__Bacillota|c__Negativicutes|o__Veillonellales|f__Veillonellaceae|g__Dialister
</td>
</tr>
<tr>
<td style="text-align:left;">
Prevotella
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1.00
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Pseudomonadati
</td>
<td style="text-align:left;">
Bacteroidota
</td>
<td style="text-align:left;">
Bacteroidia
</td>
<td style="text-align:left;">
Bacteroidales
</td>
<td style="text-align:left;">
Prevotellaceae
</td>
<td style="text-align:left;">
Prevotella
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:left;">
d__Bacteria|k__Pseudomonadati|p__Bacteroidota|c__Bacteroidia|o__Bacteroidales|f__Prevotellaceae|g__Prevotella
</td>
</tr>
<tr>
<td style="text-align:left;">
Campylobacter
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1.00
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Pseudomonadati
</td>
<td style="text-align:left;">
Campylobacterota
</td>
<td style="text-align:left;">
Epsilonproteobacteria
</td>
<td style="text-align:left;">
Campylobacterales
</td>
<td style="text-align:left;">
Campylobacteraceae
</td>
<td style="text-align:left;">
Campylobacter
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:left;">
d__Bacteria|k__Pseudomonadati|p__Campylobacterota|c__Epsilonproteobacteria|o__Campylobacterales|f__Campylobacteraceae|g__Campylobacter
</td>
</tr>
<tr>
<td style="text-align:left;">
Gardnerella
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.00
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
Actinomycetes
</td>
<td style="text-align:left;">
Bifidobacteriales
</td>
<td style="text-align:left;">
Bifidobacteriaceae
</td>
<td style="text-align:left;">
Gardnerella
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:left;">
d__Bacteria|k__Bacillati|p__Actinomycetota|c__Actinomycetes|o__Bifidobacteriales|f__Bifidobacteriaceae|g__Gardnerella
</td>
</tr>
<tr>
<td style="text-align:left;">
Propionibacterium
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.00
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
Actinomycetes
</td>
<td style="text-align:left;">
Propionibacteriales
</td>
<td style="text-align:left;">
Propionibacteriaceae
</td>
<td style="text-align:left;">
Propionibacterium
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:left;">
d__Bacteria|k__Bacillati|p__Actinomycetota|c__Actinomycetes|o__Propionibacteriales|f__Propionibacteriaceae|g__Propionibacterium
</td>
</tr>
<tr>
<td style="text-align:left;">
Atopobium
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1.00
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
Coriobacteriia
</td>
<td style="text-align:left;">
Coriobacteriales
</td>
<td style="text-align:left;">
Atopobiaceae
</td>
<td style="text-align:left;">
Atopobium
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:left;">
d__Bacteria|k__Bacillati|p__Actinomycetota|c__Coriobacteriia|o__Coriobacteriales|f__Atopobiaceae|g__Atopobium
</td>
</tr>
<tr>
<td style="text-align:left;">
Mobiluncus
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1.00
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Actinomycetota
</td>
<td style="text-align:left;">
Actinomycetes
</td>
<td style="text-align:left;">
Actinomycetales
</td>
<td style="text-align:left;">
Actinomycetaceae
</td>
<td style="text-align:left;">
Mobiluncus
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:left;">
d__Bacteria|k__Bacillati|p__Actinomycetota|c__Actinomycetes|o__Actinomycetales|f__Actinomycetaceae|g__Mobiluncus
</td>
</tr>
<tr>
<td style="text-align:left;">
Megasphaera
</td>
<td style="text-align:left;">
genus
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1.00
</td>
<td style="text-align:left;">
Bacteria
</td>
<td style="text-align:left;">
Bacillati
</td>
<td style="text-align:left;">
Bacillota
</td>
<td style="text-align:left;">
Negativicutes
</td>
<td style="text-align:left;">
Veillonellales
</td>
<td style="text-align:left;">
Veillonellaceae
</td>
<td style="text-align:left;">
Megasphaera
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:left;">
d__Bacteria|k__Bacillati|p__Bacillota|c__Negativicutes|o__Veillonellales|f__Veillonellaceae|g__Megasphaera
</td>
</tr>
</tbody>
</table>
<div class="section level3">
<h3 id="load-hmp-data-from-curatedmetagenomicdata">Load HMP data from curatedMetagenomicData<a class="anchor" aria-label="anchor" href="#load-hmp-data-from-curatedmetagenomicdata"></a>
</h3>
<div class="sourceCode" id="cb29"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/base/message.html" class="external-link">suppressPackageStartupMessages</a></span><span class="op">(</span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/waldronlab/curatedMetagenomicData" class="external-link">curatedMetagenomicData</a></span><span class="op">)</span><span class="op">)</span></span>
<span><span class="co"># hmp_vagina_metadata &lt;- filter(sampleMetadata, study_name == "HMP_2012" &amp; body_site == "vagina")</span></span>
<span><span class="va">all_healthy_adult_vagina_metadata</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">sampleMetadata</span>, <span class="va">body_site</span> <span class="op">==</span> <span class="st">"vagina"</span> <span class="op">&amp;</span> <span class="va">disease</span> <span class="op">==</span> <span class="st">"healthy"</span> <span class="op">&amp;</span> <span class="va">age_category</span> <span class="op">==</span> <span class="st">"adult"</span><span class="op">)</span></span>
<span><span class="va">se</span> <span class="op">&lt;-</span></span>
<span>  <span class="fu">curatedMetagenomicData</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/curatedMetagenomicData/man/returnSamples.html" class="external-link">returnSamples</a></span><span class="op">(</span><span class="va">all_healthy_adult_vagina_metadata</span>, dataType <span class="op">=</span> <span class="st">"relative_abundance"</span>, counts <span class="op">=</span> <span class="cn">FALSE</span><span class="op">)</span></span></code></pre></div>
<p>Get matrices of species and genus relative abundance</p>
<div class="sourceCode" id="cb30"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">allranks</span> <span class="op">&lt;-</span> <span class="fu">mia</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/mia/man/agglomerate-methods.html" class="external-link">splitByRanks</a></span><span class="op">(</span><span class="va">se</span><span class="op">)</span></span>
<span><span class="co">#species_relab &lt;- t(assay(allranks[["species"]]))</span></span>
<span><span class="va">genus_relab</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/t.html" class="external-link">t</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">assay</a></span><span class="op">(</span><span class="va">allranks</span><span class="op">[[</span><span class="st">"genus"</span><span class="op">]</span><span class="op">]</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="co">#family_relab &lt;- t(assay(allranks[["family"]]))</span></span></code></pre></div>
<p>Centered log-ratio transformation on genus level relative
abundance:</p>
<div class="sourceCode" id="cb31"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">genus_relab_clr</span> <span class="op">&lt;-</span> <span class="fu">Hotelling</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/Hotelling/man/clr.html" class="external-link">clr</a></span><span class="op">(</span><span class="va">genus_relab</span> <span class="op">+</span> <span class="fl">1</span><span class="op">)</span></span></code></pre></div>
<div class="sourceCode" id="cb32"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">df</span> <span class="op">&lt;-</span> <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/select.html" class="external-link">select</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/data.frame.html" class="external-link">data.frame</a></span><span class="op">(</span><span class="va">genus_relab_clr</span><span class="op">)</span>, <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Streptococcus"</span>, <span class="st">"Lactobacillus"</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="va">df</span><span class="op">$</span><span class="va">age</span> <span class="op">&lt;-</span> <span class="va">se</span><span class="op">$</span><span class="va">age</span></span>
<span><span class="va">df</span><span class="op">$</span><span class="va">BMI</span> <span class="op">&lt;-</span> <span class="va">se</span><span class="op">$</span><span class="va">BMI</span></span>
<span><span class="va">fit</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/stats/lm.html" class="external-link">lm</a></span><span class="op">(</span><span class="va">Streptococcus</span> <span class="op">~</span> <span class="va">Lactobacillus</span> <span class="op">+</span> <span class="va">age</span> <span class="op">+</span> <span class="va">BMI</span>, data <span class="op">=</span> <span class="va">df</span><span class="op">)</span>  </span>
<span><span class="fu"><a href="https://rdrr.io/r/base/summary.html" class="external-link">summary</a></span><span class="op">(</span><span class="va">fit</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## </span></span>
<span><span class="co">## Call:</span></span>
<span><span class="co">## lm(formula = Streptococcus ~ Lactobacillus + age + BMI, data = df)</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## Residuals:</span></span>
<span><span class="co">##     Min      1Q  Median      3Q     Max </span></span>
<span><span class="co">## -0.6728 -0.1875 -0.0264  0.0893  3.6729 </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## Coefficients:</span></span>
<span><span class="co">##                Estimate Std. Error t value Pr(&gt;|t|)    </span></span>
<span><span class="co">## (Intercept)   -0.386413   0.436694  -0.885 0.378987    </span></span>
<span><span class="co">## Lactobacillus -0.078095   0.047933  -1.629 0.107344    </span></span>
<span><span class="co">## age            0.035738   0.010073   3.548 0.000665 ***</span></span>
<span><span class="co">## BMI           -0.009094   0.014295  -0.636 0.526581    </span></span>
<span><span class="co">## ---</span></span>
<span><span class="co">## Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## Residual standard error: 0.4857 on 77 degrees of freedom</span></span>
<span><span class="co">##   (14 observations deleted due to missingness)</span></span>
<span><span class="co">## Multiple R-squared:  0.1799, Adjusted R-squared:  0.1479 </span></span>
<span><span class="co">## F-statistic: 5.629 on 3 and 77 DF,  p-value: 0.00153</span></span></code></pre>
<div class="sourceCode" id="cb34"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/base/summary.html" class="external-link">summary</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/stats/lm.html" class="external-link">lm</a></span><span class="op">(</span><span class="va">Streptococcus</span> <span class="op">~</span> <span class="va">Lactobacillus</span>, data <span class="op">=</span> <span class="va">df</span><span class="op">)</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## </span></span>
<span><span class="co">## Call:</span></span>
<span><span class="co">## lm(formula = Streptococcus ~ Lactobacillus, data = df)</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## Residuals:</span></span>
<span><span class="co">##     Min      1Q  Median      3Q     Max </span></span>
<span><span class="co">## -0.6006 -0.0609 -0.0462 -0.0440  4.1909 </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## Coefficients:</span></span>
<span><span class="co">##               Estimate Std. Error t value Pr(&gt;|t|)   </span></span>
<span><span class="co">## (Intercept)    0.55246    0.20193   2.736  0.00745 **</span></span>
<span><span class="co">## Lactobacillus -0.11409    0.04751  -2.401  0.01832 * </span></span>
<span><span class="co">## ---</span></span>
<span><span class="co">## Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## Residual standard error: 0.5642 on 93 degrees of freedom</span></span>
<span><span class="co">## Multiple R-squared:  0.05839,    Adjusted R-squared:  0.04826 </span></span>
<span><span class="co">## F-statistic: 5.767 on 1 and 93 DF,  p-value: 0.01832</span></span></code></pre>
<div class="sourceCode" id="cb36"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/stats/wilcox.test.html" class="external-link">wilcox.test</a></span><span class="op">(</span><span class="va">df</span><span class="op">$</span><span class="va">Streptococcus</span>, <span class="va">df</span><span class="op">$</span><span class="va">Lactobacillus</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## </span></span>
<span><span class="co">##  Wilcoxon rank sum test with continuity correction</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## data:  df$Streptococcus and df$Lactobacillus</span></span>
<span><span class="co">## W = 213.5, p-value &lt; 2.2e-16</span></span>
<span><span class="co">## alternative hypothesis: true location shift is not equal to 0</span></span></code></pre>
<div class="sourceCode" id="cb38"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/stats/t.test.html" class="external-link">t.test</a></span><span class="op">(</span><span class="va">df</span><span class="op">$</span><span class="va">Streptococcus</span>, <span class="va">df</span><span class="op">$</span><span class="va">Lactobacillus</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## </span></span>
<span><span class="co">##  Welch Two Sample t-test</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## data:  df$Streptococcus and df$Lactobacillus</span></span>
<span><span class="co">## t = -28.666, df = 133.93, p-value &lt; 2.2e-16</span></span>
<span><span class="co">## alternative hypothesis: true difference in means is not equal to 0</span></span>
<span><span class="co">## 95 percent confidence interval:</span></span>
<span><span class="co">##  -4.258800 -3.709054</span></span>
<span><span class="co">## sample estimates:</span></span>
<span><span class="co">## mean of x mean of y </span></span>
<span><span class="co">## 0.0879008 4.0718280</span></span></code></pre>
<div class="sourceCode" id="cb40"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">secount</span> <span class="op">&lt;-</span></span>
<span>  <span class="fu">curatedMetagenomicData</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/curatedMetagenomicData/man/returnSamples.html" class="external-link">returnSamples</a></span><span class="op">(</span><span class="va">all_healthy_adult_vagina_metadata</span>, dataType <span class="op">=</span> <span class="st">"relative_abundance"</span>, counts <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## </span></span>
<span><span class="co">## $`2021-03-31.FerrettiP_2018.relative_abundance`</span></span>
<span><span class="co">## dropping rows without rowTree matches:</span></span>
<span><span class="co">##   k__Bacteria|p__Actinobacteria|c__Coriobacteriia|o__Coriobacteriales|f__Coriobacteriaceae|g__Collinsella|s__Collinsella_stercoris</span></span>
<span><span class="co">##   k__Bacteria|p__Actinobacteria|c__Coriobacteriia|o__Coriobacteriales|f__Coriobacteriaceae|g__Enorma|s__[Collinsella]_massiliensis</span></span>
<span><span class="co">##   k__Bacteria|p__Firmicutes|c__Bacilli|o__Bacillales|f__Bacillales_unclassified|g__Gemella|s__Gemella_bergeri</span></span>
<span><span class="co">##   k__Bacteria|p__Firmicutes|c__Bacilli|o__Lactobacillales|f__Carnobacteriaceae|g__Granulicatella|s__Granulicatella_elegans</span></span>
<span><span class="co">##   k__Bacteria|p__Firmicutes|c__Clostridia|o__Clostridiales|f__Ruminococcaceae|g__Ruminococcus|s__Ruminococcus_champanellensis</span></span>
<span><span class="co">##   k__Bacteria|p__Firmicutes|c__Erysipelotrichia|o__Erysipelotrichales|f__Erysipelotrichaceae|g__Bulleidia|s__Bulleidia_extructa</span></span>
<span><span class="co">##   k__Bacteria|p__Proteobacteria|c__Betaproteobacteria|o__Burkholderiales|f__Sutterellaceae|g__Sutterella|s__Sutterella_parvirubra</span></span>
<span><span class="co">##   k__Bacteria|p__Spirochaetes|c__Spirochaetia|o__Leptospirales|f__Leptospiraceae|g__Leptospira|s__Leptospira_hartskeerlii</span></span>
<span><span class="co">##   k__Bacteria|p__Synergistetes|c__Synergistia|o__Synergistales|f__Synergistaceae|g__Cloacibacillus|s__Cloacibacillus_evryensis</span></span></code></pre>
<pre><code><span><span class="co">## $`2021-03-31.HMP_2012.relative_abundance`</span></span>
<span><span class="co">## dropping rows without rowTree matches:</span></span>
<span><span class="co">##   k__Bacteria|p__Actinobacteria|c__Coriobacteriia|o__Coriobacteriales|f__Atopobiaceae|g__Olsenella|s__Olsenella_profusa</span></span>
<span><span class="co">##   k__Bacteria|p__Actinobacteria|c__Coriobacteriia|o__Coriobacteriales|f__Coriobacteriaceae|g__Collinsella|s__Collinsella_stercoris</span></span>
<span><span class="co">##   k__Bacteria|p__Actinobacteria|c__Coriobacteriia|o__Coriobacteriales|f__Coriobacteriaceae|g__Enorma|s__[Collinsella]_massiliensis</span></span>
<span><span class="co">##   k__Bacteria|p__Firmicutes|c__Bacilli|o__Bacillales|f__Bacillales_unclassified|g__Gemella|s__Gemella_bergeri</span></span>
<span><span class="co">##   k__Bacteria|p__Firmicutes|c__Bacilli|o__Lactobacillales|f__Carnobacteriaceae|g__Granulicatella|s__Granulicatella_elegans</span></span>
<span><span class="co">##   k__Bacteria|p__Firmicutes|c__Clostridia|o__Clostridiales|f__Ruminococcaceae|g__Ruminococcus|s__Ruminococcus_champanellensis</span></span>
<span><span class="co">##   k__Bacteria|p__Firmicutes|c__Erysipelotrichia|o__Erysipelotrichales|f__Erysipelotrichaceae|g__Bulleidia|s__Bulleidia_extructa</span></span>
<span><span class="co">##   k__Bacteria|p__Proteobacteria|c__Betaproteobacteria|o__Burkholderiales|f__Sutterellaceae|g__Sutterella|s__Sutterella_parvirubra</span></span>
<span><span class="co">##   k__Bacteria|p__Synergistetes|c__Synergistia|o__Synergistales|f__Synergistaceae|g__Cloacibacillus|s__Cloacibacillus_evryensis</span></span></code></pre>
<pre><code><span><span class="co">## $`2021-03-31.WampachL_2018.relative_abundance`</span></span>
<span><span class="co">## dropping rows without rowTree matches:</span></span>
<span><span class="co">##   k__Bacteria|p__Actinobacteria|c__Coriobacteriia|o__Coriobacteriales|f__Coriobacteriaceae|g__Collinsella|s__Collinsella_stercoris</span></span>
<span><span class="co">##   k__Bacteria|p__Actinobacteria|c__Coriobacteriia|o__Coriobacteriales|f__Coriobacteriaceae|g__Enorma|s__[Collinsella]_massiliensis</span></span>
<span><span class="co">##   k__Bacteria|p__Firmicutes|c__Bacilli|o__Lactobacillales|f__Carnobacteriaceae|g__Granulicatella|s__Granulicatella_elegans</span></span>
<span><span class="co">##   k__Bacteria|p__Proteobacteria|c__Betaproteobacteria|o__Burkholderiales|f__Sutterellaceae|g__Sutterella|s__Sutterella_parvirubra</span></span>
<span><span class="co">##   k__Bacteria|p__Synergistetes|c__Synergistia|o__Synergistales|f__Synergistaceae|g__Cloacibacillus|s__Cloacibacillus_evryensis</span></span></code></pre>
<div class="sourceCode" id="cb44"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">allranks</span> <span class="op">&lt;-</span> <span class="fu">mia</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/mia/man/agglomerate-methods.html" class="external-link">splitByRanks</a></span><span class="op">(</span><span class="va">secount</span><span class="op">)</span></span>
<span><span class="va">genus_relab_count</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/t.html" class="external-link">t</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">assay</a></span><span class="op">(</span><span class="va">allranks</span><span class="op">[[</span><span class="st">"genus"</span><span class="op">]</span><span class="op">]</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="va">df</span> <span class="op">&lt;-</span> <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/select.html" class="external-link">select</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/data.frame.html" class="external-link">data.frame</a></span><span class="op">(</span><span class="va">genus_relab_count</span><span class="op">)</span>, <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Streptococcus"</span>, <span class="st">"Lactobacillus"</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="va">df</span><span class="op">$</span><span class="va">age</span> <span class="op">&lt;-</span> <span class="va">secount</span><span class="op">$</span><span class="va">age</span></span>
<span><span class="va">df</span><span class="op">$</span><span class="va">BMI</span> <span class="op">&lt;-</span> <span class="va">secount</span><span class="op">$</span><span class="va">BMI</span></span></code></pre></div>
<div class="sourceCode" id="cb45"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/atahk/pscl" class="external-link">pscl</a></span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## Classes and Methods for R originally developed in the</span></span>
<span><span class="co">## Political Science Computational Laboratory</span></span>
<span><span class="co">## Department of Political Science</span></span>
<span><span class="co">## Stanford University (2002-2015),</span></span>
<span><span class="co">## by and under the direction of Simon Jackman.</span></span>
<span><span class="co">## hurdle and zeroinfl functions by Achim Zeileis.</span></span></code></pre>
<div class="sourceCode" id="cb47"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">fit.ZInegbin</span> <span class="op">&lt;-</span> <span class="fu">pscl</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/pscl/man/zeroinfl.html" class="external-link">zeroinfl</a></span><span class="op">(</span><span class="va">Streptococcus</span> <span class="op">~</span> <span class="va">Lactobacillus</span> <span class="op">+</span> <span class="va">age</span> <span class="op">+</span> <span class="va">BMI</span> <span class="op">|</span> <span class="fl">1</span>,</span>
<span>                data <span class="op">=</span> <span class="va">df</span>,</span>
<span>                dist <span class="op">=</span> <span class="st">"negbin"</span><span class="op">)</span></span></code></pre></div>
</div>
</div>
<div class="section level2">
<h2 id="create-a-data-frame-for-regression-analysis-from-healthy-controls-for-log-linear-model">Create a data frame for regression analysis from healthy controls
for log-linear model<a class="anchor" aria-label="anchor" href="#create-a-data-frame-for-regression-analysis-from-healthy-controls-for-log-linear-model"></a>
</h2>
<div class="sourceCode" id="cb48"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/waldronlab/curatedMetagenomicData" class="external-link">curatedMetagenomicData</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://dplyr.tidyverse.org" class="external-link">dplyr</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/rstudio/DT" class="external-link">DT</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://purrr.tidyverse.org/" class="external-link">purrr</a></span><span class="op">)</span></span>
<span></span>
<span><span class="va">taxon_strep</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"1884"</span> <span class="op">=</span> <span class="st">"Streptococcus"</span><span class="op">)</span></span>
<span><span class="va">taxon_lact</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"1578"</span> <span class="op">=</span> <span class="st">"Lactobacillus"</span><span class="op">)</span></span>
<span></span>
<span><span class="va">target_sample</span> <span class="op">&lt;-</span> <span class="st">"vagina"</span></span>
<span><span class="va">sample_metadata</span> <span class="op">&lt;-</span> <span class="va">sampleMetadata</span> <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html" class="external-link">%&gt;%</a></span> </span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span></span>
<span>        <span class="fu"><a href="https://rdrr.io/r/base/grep.html" class="external-link">grepl</a></span><span class="op">(</span><span class="va">target_sample</span>, <span class="va">body_site</span><span class="op">)</span> <span class="op">|</span> <span class="fu"><a href="https://rdrr.io/r/base/grep.html" class="external-link">grepl</a></span><span class="op">(</span><span class="va">target_sample</span>, <span class="va">body_subsite</span><span class="op">)</span></span>
<span>    <span class="op">)</span> <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html" class="external-link">%&gt;%</a></span> </span>
<span>    <span class="fu"><a href="https://purrr.tidyverse.org/reference/keep.html" class="external-link">discard</a></span><span class="op">(</span><span class="op">~</span><span class="fu"><a href="https://rdrr.io/r/base/all.html" class="external-link">all</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/NA.html" class="external-link">is.na</a></span><span class="op">(</span><span class="va">.x</span><span class="op">)</span><span class="op">)</span><span class="op">)</span> <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html" class="external-link">%&gt;%</a></span> </span>
<span>    <span class="fu"><a href="https://tibble.tidyverse.org/reference/as_tibble.html" class="external-link">as_tibble</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/dim.html" class="external-link">dim</a></span><span class="op">(</span><span class="va">sample_metadata</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## [1] 96 26</span></span></code></pre>
<div class="sourceCode" id="cb50"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/base/unique.html" class="external-link">unique</a></span><span class="op">(</span><span class="va">sample_metadata</span><span class="op">$</span><span class="va">study_name</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## [1] "FerrettiP_2018" "HMP_2012"       "WampachL_2018"</span></span></code></pre>
<div class="sourceCode" id="cb52"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">tse</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/curatedMetagenomicData/man/returnSamples.html" class="external-link">returnSamples</a></span><span class="op">(</span></span>
<span>    sampleMetadata <span class="op">=</span> <span class="va">sample_metadata</span>,</span>
<span>    dataType <span class="op">=</span> <span class="st">"relative_abundance"</span>,</span>
<span>    counts <span class="op">=</span> <span class="cn">FALSE</span>,</span>
<span>    rownames <span class="op">=</span> <span class="st">"short"</span></span>
<span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## </span></span>
<span><span class="co">## $`2021-03-31.FerrettiP_2018.relative_abundance`</span></span>
<span><span class="co">## dropping rows without rowTree matches:</span></span>
<span><span class="co">##   k__Bacteria|p__Actinobacteria|c__Coriobacteriia|o__Coriobacteriales|f__Coriobacteriaceae|g__Collinsella|s__Collinsella_stercoris</span></span>
<span><span class="co">##   k__Bacteria|p__Actinobacteria|c__Coriobacteriia|o__Coriobacteriales|f__Coriobacteriaceae|g__Enorma|s__[Collinsella]_massiliensis</span></span>
<span><span class="co">##   k__Bacteria|p__Firmicutes|c__Bacilli|o__Bacillales|f__Bacillales_unclassified|g__Gemella|s__Gemella_bergeri</span></span>
<span><span class="co">##   k__Bacteria|p__Firmicutes|c__Bacilli|o__Lactobacillales|f__Carnobacteriaceae|g__Granulicatella|s__Granulicatella_elegans</span></span>
<span><span class="co">##   k__Bacteria|p__Firmicutes|c__Clostridia|o__Clostridiales|f__Ruminococcaceae|g__Ruminococcus|s__Ruminococcus_champanellensis</span></span>
<span><span class="co">##   k__Bacteria|p__Firmicutes|c__Erysipelotrichia|o__Erysipelotrichales|f__Erysipelotrichaceae|g__Bulleidia|s__Bulleidia_extructa</span></span>
<span><span class="co">##   k__Bacteria|p__Proteobacteria|c__Betaproteobacteria|o__Burkholderiales|f__Sutterellaceae|g__Sutterella|s__Sutterella_parvirubra</span></span>
<span><span class="co">##   k__Bacteria|p__Spirochaetes|c__Spirochaetia|o__Leptospirales|f__Leptospiraceae|g__Leptospira|s__Leptospira_hartskeerlii</span></span>
<span><span class="co">##   k__Bacteria|p__Synergistetes|c__Synergistia|o__Synergistales|f__Synergistaceae|g__Cloacibacillus|s__Cloacibacillus_evryensis</span></span></code></pre>
<pre><code><span><span class="co">## $`2021-03-31.HMP_2012.relative_abundance`</span></span>
<span><span class="co">## dropping rows without rowTree matches:</span></span>
<span><span class="co">##   k__Bacteria|p__Actinobacteria|c__Coriobacteriia|o__Coriobacteriales|f__Atopobiaceae|g__Olsenella|s__Olsenella_profusa</span></span>
<span><span class="co">##   k__Bacteria|p__Actinobacteria|c__Coriobacteriia|o__Coriobacteriales|f__Coriobacteriaceae|g__Collinsella|s__Collinsella_stercoris</span></span>
<span><span class="co">##   k__Bacteria|p__Actinobacteria|c__Coriobacteriia|o__Coriobacteriales|f__Coriobacteriaceae|g__Enorma|s__[Collinsella]_massiliensis</span></span>
<span><span class="co">##   k__Bacteria|p__Firmicutes|c__Bacilli|o__Bacillales|f__Bacillales_unclassified|g__Gemella|s__Gemella_bergeri</span></span>
<span><span class="co">##   k__Bacteria|p__Firmicutes|c__Bacilli|o__Lactobacillales|f__Carnobacteriaceae|g__Granulicatella|s__Granulicatella_elegans</span></span>
<span><span class="co">##   k__Bacteria|p__Firmicutes|c__Clostridia|o__Clostridiales|f__Ruminococcaceae|g__Ruminococcus|s__Ruminococcus_champanellensis</span></span>
<span><span class="co">##   k__Bacteria|p__Firmicutes|c__Erysipelotrichia|o__Erysipelotrichales|f__Erysipelotrichaceae|g__Bulleidia|s__Bulleidia_extructa</span></span>
<span><span class="co">##   k__Bacteria|p__Proteobacteria|c__Betaproteobacteria|o__Burkholderiales|f__Sutterellaceae|g__Sutterella|s__Sutterella_parvirubra</span></span>
<span><span class="co">##   k__Bacteria|p__Synergistetes|c__Synergistia|o__Synergistales|f__Synergistaceae|g__Cloacibacillus|s__Cloacibacillus_evryensis</span></span></code></pre>
<pre><code><span><span class="co">## $`2021-03-31.WampachL_2018.relative_abundance`</span></span>
<span><span class="co">## dropping rows without rowTree matches:</span></span>
<span><span class="co">##   k__Bacteria|p__Actinobacteria|c__Coriobacteriia|o__Coriobacteriales|f__Coriobacteriaceae|g__Collinsella|s__Collinsella_stercoris</span></span>
<span><span class="co">##   k__Bacteria|p__Actinobacteria|c__Coriobacteriia|o__Coriobacteriales|f__Coriobacteriaceae|g__Enorma|s__[Collinsella]_massiliensis</span></span>
<span><span class="co">##   k__Bacteria|p__Firmicutes|c__Bacilli|o__Lactobacillales|f__Carnobacteriaceae|g__Granulicatella|s__Granulicatella_elegans</span></span>
<span><span class="co">##   k__Bacteria|p__Proteobacteria|c__Betaproteobacteria|o__Burkholderiales|f__Sutterellaceae|g__Sutterella|s__Sutterella_parvirubra</span></span>
<span><span class="co">##   k__Bacteria|p__Synergistetes|c__Synergistia|o__Synergistales|f__Synergistaceae|g__Cloacibacillus|s__Cloacibacillus_evryensis</span></span></code></pre>
<pre><code><span><span class="co">## Warning in convertNode(tree = tree, node = labN, use.alias = FALSE, message =</span></span>
<span><span class="co">## FALSE): Multiple nodes are found to have the same label.</span></span></code></pre>
<div class="sourceCode" id="cb57"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">tse</span></span></code></pre></div>
<pre><code><span><span class="co">## class: TreeSummarizedExperiment </span></span>
<span><span class="co">## dim: 908 96 </span></span>
<span><span class="co">## metadata(0):</span></span>
<span><span class="co">## assays(1): relative_abundance</span></span>
<span><span class="co">## rownames(908): species:Escherichia coli species:Klebsiella</span></span>
<span><span class="co">##   michiganensis ... species:Pediococcus pentosaceus species:Rahnella</span></span>
<span><span class="co">##   aquatilis</span></span>
<span><span class="co">## rowData names(7): superkingdom phylum ... genus species</span></span>
<span><span class="co">## colnames(96): CA_C10001MS2003VA_t0M15 CA_C10003MS2031VA_t0M15 ...</span></span>
<span><span class="co">##   MV_C118 MV_C120</span></span>
<span><span class="co">## colData names(25): study_name subject_id ... family_role body_subsite</span></span>
<span><span class="co">## reducedDimNames(0):</span></span>
<span><span class="co">## mainExpName: NULL</span></span>
<span><span class="co">## altExpNames(0):</span></span>
<span><span class="co">## rowLinks: a LinkDataFrame (908 rows)</span></span>
<span><span class="co">## rowTree: 1 phylo tree(s) (10430 leaves)</span></span>
<span><span class="co">## colLinks: NULL</span></span>
<span><span class="co">## colTree: NULL</span></span></code></pre>
<div class="section level3">
<h3 id="comparing-mean-to-variance-of-the-outcome-variable">Comparing mean to variance of the outcome variable<a class="anchor" aria-label="anchor" href="#comparing-mean-to-variance-of-the-outcome-variable"></a>
</h3>
<div class="sourceCode" id="cb59"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://ggplot2.tidyverse.org" class="external-link">ggplot2</a></span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/mean.html" class="external-link">mean</a></span><span class="op">(</span><span class="va">df</span><span class="op">$</span><span class="va">Streptococcus</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## [1] 4049.558</span></span></code></pre>
<div class="sourceCode" id="cb61"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/stats/cor.html" class="external-link">var</a></span><span class="op">(</span><span class="va">df</span><span class="op">$</span><span class="va">Streptococcus</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## [1] 509068831</span></span></code></pre>
<div class="sourceCode" id="cb63"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">fraczeroes</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/nrow.html" class="external-link">nrow</a></span><span class="op">(</span><span class="va">df</span><span class="op">[</span><span class="va">df</span><span class="op">$</span><span class="va">Streptococcus</span> <span class="op">==</span> <span class="fl">0</span>,<span class="op">]</span><span class="op">)</span><span class="op">/</span><span class="fu"><a href="https://rdrr.io/r/base/nrow.html" class="external-link">nrow</a></span><span class="op">(</span><span class="va">df</span><span class="op">)</span></span>
<span><span class="va">fraczeroes</span></span></code></pre></div>
<pre><code><span><span class="co">## [1] 0.8736842</span></span></code></pre>
<p>#Fit log-linear models</p>
<div class="sourceCode" id="cb65"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">fit.pois</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/stats/glm.html" class="external-link">glm</a></span><span class="op">(</span><span class="va">Streptococcus</span> <span class="op">~</span> <span class="va">Lactobacillus</span> <span class="op">+</span> <span class="va">age</span> <span class="op">+</span> <span class="va">BMI</span>,</span>
<span>                data <span class="op">=</span> <span class="va">df</span>,</span>
<span>                family <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/stats/family.html" class="external-link">poisson</a></span><span class="op">(</span>link <span class="op">=</span> <span class="st">"log"</span><span class="op">)</span><span class="op">)</span></span>
<span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/summary.html" class="external-link">summary</a></span> <span class="op">(</span><span class="va">fit.pois</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## </span></span>
<span><span class="co">## Call:</span></span>
<span><span class="co">## glm(formula = Streptococcus ~ Lactobacillus + age + BMI, family = poisson(link = "log"), </span></span>
<span><span class="co">##     data = df)</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## Coefficients:</span></span>
<span><span class="co">##                 Estimate Std. Error z value Pr(&gt;|z|)    </span></span>
<span><span class="co">## (Intercept)   -5.313e-01  1.873e-02  -28.36   &lt;2e-16 ***</span></span>
<span><span class="co">## Lactobacillus  5.141e-08  9.828e-10   52.32   &lt;2e-16 ***</span></span>
<span><span class="co">## age            3.599e-01  4.136e-04  870.25   &lt;2e-16 ***</span></span>
<span><span class="co">## BMI           -1.573e-01  6.453e-04 -243.77   &lt;2e-16 ***</span></span>
<span><span class="co">## ---</span></span>
<span><span class="co">## Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## (Dispersion parameter for poisson family taken to be 1)</span></span>
<span><span class="co">## </span></span>
<span><span class="co">##     Null deviance: 1929049  on 80  degrees of freedom</span></span>
<span><span class="co">## Residual deviance:  632598  on 77  degrees of freedom</span></span>
<span><span class="co">##   (14 observations deleted due to missingness)</span></span>
<span><span class="co">## AIC: 632704</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## Number of Fisher Scoring iterations: 20</span></span></code></pre>
<div class="sourceCode" id="cb67"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/atahk/pscl" class="external-link">pscl</a></span><span class="op">)</span></span>
<span><span class="va">fit.ZIpois</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/pscl/man/zeroinfl.html" class="external-link">zeroinfl</a></span><span class="op">(</span><span class="va">Streptococcus</span> <span class="op">~</span> <span class="va">Lactobacillus</span> <span class="op">+</span> <span class="va">age</span> <span class="op">+</span> <span class="va">BMI</span> <span class="op">|</span> <span class="fl">1</span>,</span>
<span>                data <span class="op">=</span> <span class="va">df</span>,</span>
<span>                dist <span class="op">=</span> <span class="st">"poisson"</span><span class="op">)</span></span>
<span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/summary.html" class="external-link">summary</a></span> <span class="op">(</span><span class="va">fit.ZIpois</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## </span></span>
<span><span class="co">## Call:</span></span>
<span><span class="co">## zeroinfl(formula = Streptococcus ~ Lactobacillus + age + BMI | 1, data = df, </span></span>
<span><span class="co">##     dist = "poisson")</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## Pearson residuals:</span></span>
<span><span class="co">##     Min      1Q  Median      3Q     Max </span></span>
<span><span class="co">## -0.3753 -0.3753 -0.3752 -0.3750 27.8168 </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## Count model coefficients (poisson with log link):</span></span>
<span><span class="co">##                 Estimate Std. Error z value Pr(&gt;|z|)</span></span>
<span><span class="co">## (Intercept)    6.169e+00        NaN     NaN      NaN</span></span>
<span><span class="co">## Lactobacillus  1.321e-07        NaN     NaN      NaN</span></span>
<span><span class="co">## age            2.420e-01        NaN     NaN      NaN</span></span>
<span><span class="co">## BMI           -2.116e-01        NaN     NaN      NaN</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## Zero-inflation model coefficients (binomial with logit link):</span></span>
<span><span class="co">##             Estimate Std. Error z value Pr(&gt;|z|)</span></span>
<span><span class="co">## (Intercept)     1.96        NaN     NaN      NaN</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## Number of iterations in BFGS optimization: 20 </span></span>
<span><span class="co">## Log-likelihood: -1.02e+05 on 5 Df</span></span></code></pre>
<div class="sourceCode" id="cb69"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="http://www.stats.ox.ac.uk/pub/MASS4/" class="external-link">MASS</a></span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## </span></span>
<span><span class="co">## Attaching package: 'MASS'</span></span></code></pre>
<pre><code><span><span class="co">## The following object is masked from 'package:dplyr':</span></span>
<span><span class="co">## </span></span>
<span><span class="co">##     select</span></span></code></pre>
<div class="sourceCode" id="cb72"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">fit.negbin</span> <span class="op">&lt;-</span></span>
<span>  <span class="fu"><a href="https://rdrr.io/pkg/MASS/man/glm.nb.html" class="external-link">glm.nb</a></span><span class="op">(</span></span>
<span>    <span class="va">Streptococcus</span> <span class="op">~</span> <span class="va">Lactobacillus</span> <span class="op">+</span> <span class="va">age</span> <span class="op">+</span> <span class="va">BMI</span>,</span>
<span>    data <span class="op">=</span> <span class="va">df</span>,</span>
<span>    control <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/stats/glm.control.html" class="external-link">glm.control</a></span><span class="op">(</span></span>
<span>      epsilon <span class="op">=</span> <span class="fl">1e-8</span>,</span>
<span>      maxit <span class="op">=</span> <span class="fl">50</span>,</span>
<span>      trace <span class="op">=</span> <span class="cn">FALSE</span></span>
<span>    <span class="op">)</span></span>
<span>  <span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## Warning: glm.fit: algorithm did not converge</span></span></code></pre>
<pre><code><span><span class="co">## Warning in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace =</span></span>
<span><span class="co">## control$trace &gt; : iteration limit reached</span></span></code></pre>
<pre><code><span><span class="co">## Warning in sqrt(1/i): NaNs produced</span></span></code></pre>
<pre><code><span><span class="co">## Warning in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace =</span></span>
<span><span class="co">## control$trace &gt; : iteration limit reached</span></span></code></pre>
<pre><code><span><span class="co">## Warning in sqrt(1/i): NaNs produced</span></span></code></pre>
<pre><code><span><span class="co">## Warning: glm.fit: algorithm did not converge</span></span></code></pre>
<pre><code><span><span class="co">## Warning in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace =</span></span>
<span><span class="co">## control$trace &gt; : iteration limit reached</span></span></code></pre>
<pre><code><span><span class="co">## Warning in sqrt(1/i): NaNs produced</span></span></code></pre>
<pre><code><span><span class="co">## Warning in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace =</span></span>
<span><span class="co">## control$trace &gt; : iteration limit reached</span></span></code></pre>
<pre><code><span><span class="co">## Warning in sqrt(1/i): NaNs produced</span></span></code></pre>
<pre><code><span><span class="co">## Warning: glm.fit: algorithm did not converge</span></span></code></pre>
<pre><code><span><span class="co">## Warning in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace =</span></span>
<span><span class="co">## control$trace &gt; : iteration limit reached</span></span></code></pre>
<pre><code><span><span class="co">## Warning in sqrt(1/i): NaNs produced</span></span></code></pre>
<pre><code><span><span class="co">## Warning in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace =</span></span>
<span><span class="co">## control$trace &gt; : iteration limit reached</span></span></code></pre>
<pre><code><span><span class="co">## Warning in sqrt(1/i): NaNs produced</span></span></code></pre>
<pre><code><span><span class="co">## Warning: glm.fit: algorithm did not converge</span></span></code></pre>
<pre><code><span><span class="co">## Warning in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace =</span></span>
<span><span class="co">## control$trace &gt; : iteration limit reached</span></span></code></pre>
<pre><code><span><span class="co">## Warning in sqrt(1/i): NaNs produced</span></span></code></pre>
<pre><code><span><span class="co">## Warning in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace =</span></span>
<span><span class="co">## control$trace &gt; : iteration limit reached</span></span></code></pre>
<pre><code><span><span class="co">## Warning in sqrt(1/i): NaNs produced</span></span></code></pre>
<pre><code><span><span class="co">## Warning: glm.fit: algorithm did not converge</span></span></code></pre>
<pre><code><span><span class="co">## Warning in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace =</span></span>
<span><span class="co">## control$trace &gt; : iteration limit reached</span></span></code></pre>
<pre><code><span><span class="co">## Warning in sqrt(1/i): NaNs produced</span></span></code></pre>
<pre><code><span><span class="co">## Warning in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace =</span></span>
<span><span class="co">## control$trace &gt; : iteration limit reached</span></span></code></pre>
<pre><code><span><span class="co">## Warning in sqrt(1/i): NaNs produced</span></span></code></pre>
<pre><code><span><span class="co">## Warning: glm.fit: algorithm did not converge</span></span></code></pre>
<pre><code><span><span class="co">## Warning in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace =</span></span>
<span><span class="co">## control$trace &gt; : iteration limit reached</span></span></code></pre>
<pre><code><span><span class="co">## Warning in sqrt(1/i): NaNs produced</span></span></code></pre>
<pre><code><span><span class="co">## Warning in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace =</span></span>
<span><span class="co">## control$trace &gt; : iteration limit reached</span></span></code></pre>
<pre><code><span><span class="co">## Warning in sqrt(1/i): NaNs produced</span></span></code></pre>
<pre><code><span><span class="co">## Warning: glm.fit: algorithm did not converge</span></span></code></pre>
<pre><code><span><span class="co">## Warning in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace =</span></span>
<span><span class="co">## control$trace &gt; : iteration limit reached</span></span></code></pre>
<pre><code><span><span class="co">## Warning in sqrt(1/i): NaNs produced</span></span></code></pre>
<pre><code><span><span class="co">## Warning in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace =</span></span>
<span><span class="co">## control$trace &gt; : iteration limit reached</span></span></code></pre>
<pre><code><span><span class="co">## Warning in sqrt(1/i): NaNs produced</span></span></code></pre>
<pre><code><span><span class="co">## Warning: glm.fit: algorithm did not converge</span></span></code></pre>
<pre><code><span><span class="co">## Warning in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace =</span></span>
<span><span class="co">## control$trace &gt; : iteration limit reached</span></span></code></pre>
<pre><code><span><span class="co">## Warning in sqrt(1/i): NaNs produced</span></span></code></pre>
<pre><code><span><span class="co">## Warning in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace =</span></span>
<span><span class="co">## control$trace &gt; : iteration limit reached</span></span></code></pre>
<pre><code><span><span class="co">## Warning in sqrt(1/i): NaNs produced</span></span></code></pre>
<pre><code><span><span class="co">## Warning: glm.fit: algorithm did not converge</span></span></code></pre>
<pre><code><span><span class="co">## Warning in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace =</span></span>
<span><span class="co">## control$trace &gt; : iteration limit reached</span></span></code></pre>
<pre><code><span><span class="co">## Warning in sqrt(1/i): NaNs produced</span></span></code></pre>
<pre><code><span><span class="co">## Warning in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace =</span></span>
<span><span class="co">## control$trace &gt; : iteration limit reached</span></span></code></pre>
<pre><code><span><span class="co">## Warning in sqrt(1/i): NaNs produced</span></span></code></pre>
<pre><code><span><span class="co">## Warning: glm.fit: algorithm did not converge</span></span></code></pre>
<pre><code><span><span class="co">## Warning in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace =</span></span>
<span><span class="co">## control$trace &gt; : iteration limit reached</span></span></code></pre>
<pre><code><span><span class="co">## Warning in sqrt(1/i): NaNs produced</span></span></code></pre>
<pre><code><span><span class="co">## Warning in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace =</span></span>
<span><span class="co">## control$trace &gt; : iteration limit reached</span></span></code></pre>
<pre><code><span><span class="co">## Warning in sqrt(1/i): NaNs produced</span></span></code></pre>
<pre><code><span><span class="co">## Warning: glm.fit: algorithm did not converge</span></span></code></pre>
<pre><code><span><span class="co">## Warning in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace =</span></span>
<span><span class="co">## control$trace &gt; : iteration limit reached</span></span></code></pre>
<pre><code><span><span class="co">## Warning in sqrt(1/i): NaNs produced</span></span></code></pre>
<pre><code><span><span class="co">## Warning in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace =</span></span>
<span><span class="co">## control$trace &gt; : iteration limit reached</span></span></code></pre>
<pre><code><span><span class="co">## Warning in sqrt(1/i): NaNs produced</span></span></code></pre>
<pre><code><span><span class="co">## Warning: glm.fit: algorithm did not converge</span></span></code></pre>
<pre><code><span><span class="co">## Warning in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace =</span></span>
<span><span class="co">## control$trace &gt; : iteration limit reached</span></span></code></pre>
<pre><code><span><span class="co">## Warning in sqrt(1/i): NaNs produced</span></span></code></pre>
<pre><code><span><span class="co">## Warning in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace =</span></span>
<span><span class="co">## control$trace &gt; : iteration limit reached</span></span></code></pre>
<pre><code><span><span class="co">## Warning in sqrt(1/i): NaNs produced</span></span></code></pre>
<pre><code><span><span class="co">## Warning: glm.fit: algorithm did not converge</span></span></code></pre>
<pre><code><span><span class="co">## Warning in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace =</span></span>
<span><span class="co">## control$trace &gt; : iteration limit reached</span></span></code></pre>
<pre><code><span><span class="co">## Warning in sqrt(1/i): NaNs produced</span></span></code></pre>
<pre><code><span><span class="co">## Warning in glm.nb(Streptococcus ~ Lactobacillus + age + BMI, data = df, :</span></span>
<span><span class="co">## alternation limit reached</span></span></code></pre>
<div class="sourceCode" id="cb137"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://broom.tidymodels.org/" class="external-link">broom</a></span><span class="op">)</span></span>
<span><span class="fu"><a href="https://generics.r-lib.org/reference/tidy.html" class="external-link">tidy</a></span><span class="op">(</span><span class="va">fit.negbin</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## <span style="color: #949494;"># A tibble: 4 Ã— 5</span></span></span>
<span><span class="co">##   term               estimate   std.error statistic     p.value</span></span>
<span><span class="co">##   <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>                 <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span>       <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span>     <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span>       <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span></span></span>
<span><span class="co">## <span style="color: #BCBCBC;">1</span> (Intercept)   -<span style="color: #BB0000;">34.4</span>         8.11            -<span style="color: #BB0000;">4.24</span> 0.000<span style="text-decoration: underline;">022</span>4  </span></span>
<span><span class="co">## <span style="color: #BCBCBC;">2</span> Lactobacillus   0.000<span style="text-decoration: underline;">000</span>889 0.000<span style="text-decoration: underline;">000</span>196      4.54 0.000<span style="text-decoration: underline;">005</span>70 </span></span>
<span><span class="co">## <span style="color: #BCBCBC;">3</span> age             1.09        0.207            5.30 0.000<span style="text-decoration: underline;">000</span>119</span></span>
<span><span class="co">## <span style="color: #BCBCBC;">4</span> BMI             0.297       0.277            1.07 0.282</span></span></code></pre>
<div class="sourceCode" id="cb139"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">fit.ZInegbin</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/pscl/man/zeroinfl.html" class="external-link">zeroinfl</a></span><span class="op">(</span><span class="va">Streptococcus</span> <span class="op">~</span> <span class="va">Lactobacillus</span> <span class="op">+</span> <span class="va">age</span> <span class="op">+</span> <span class="va">BMI</span> <span class="op">|</span> <span class="fl">1</span>,</span>
<span>                data <span class="op">=</span> <span class="va">df</span>,</span>
<span>                dist <span class="op">=</span> <span class="st">"negbin"</span><span class="op">)</span></span>
<span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/summary.html" class="external-link">summary</a></span><span class="op">(</span><span class="va">fit.ZInegbin</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## </span></span>
<span><span class="co">## Call:</span></span>
<span><span class="co">## zeroinfl(formula = Streptococcus ~ Lactobacillus + age + BMI | 1, data = df, </span></span>
<span><span class="co">##     dist = "negbin")</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## Pearson residuals:</span></span>
<span><span class="co">##     Min      1Q  Median      3Q     Max </span></span>
<span><span class="co">## -0.1898 -0.1898 -0.1898 -0.1898  7.0569 </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## Count model coefficients (negbin with log link):</span></span>
<span><span class="co">##                 Estimate Std. Error z value Pr(&gt;|z|)</span></span>
<span><span class="co">## (Intercept)    1.945e+01        NaN     NaN      NaN</span></span>
<span><span class="co">## Lactobacillus -2.787e-07        NaN     NaN      NaN</span></span>
<span><span class="co">## age            2.839e-02        NaN     NaN      NaN</span></span>
<span><span class="co">## BMI           -4.084e-01        NaN     NaN      NaN</span></span>
<span><span class="co">## Log(theta)    -9.837e-01        NaN     NaN      NaN</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## Zero-inflation model coefficients (binomial with logit link):</span></span>
<span><span class="co">##             Estimate Std. Error z value Pr(&gt;|z|)</span></span>
<span><span class="co">## (Intercept)    1.921        NaN     NaN      NaN</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## Theta = 0.3739 </span></span>
<span><span class="co">## Number of iterations in BFGS optimization: 40 </span></span>
<span><span class="co">## Log-likelihood: -131.8 on 6 Df</span></span></code></pre>
</div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

      </div>

</div>



      <footer><div class="copyright">
  <p></p>
<p>Developed by Ludwig Geistlinger, Chloe Mirzayi, Fatima Zohra, Levi Waldron.</p>
</div>

<div class="pkgdown">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.1.2.</p>
</div>

      </footer>
</div>






  </body>
</html>
